
* Genomes
- A2
  #+begin_src sh
  wget https://www.cottongen.org/cottongen_downloads/Gossypium_arboreum/WHU_A2_Updated/assembly/Garboreum_Shixiya1_WHUv3.0rc.genome.standard.fa.gz
  gunzip Garboreum_Shixiya1_WHUv3.0rc.genome.standard.fa.gz

  # Only retain chrs
  ml singularity/3.5.2 samtools
  samtools faidx Garboreum_Shixiya1_WHUv3.0rc.genome.standard.fa Chr{01..13} > A2.fa
  #+end_src

- D5
  #+begin_src sh
  wget -O D5.fa.gz https://www.cottongen.org/cottongen_downloads/Gossypium_raimondii/BYU-Improved_G.raimondii/assembly/D5.v1.pred.fa.gz
  gunzip D5.fa.gz
  #+end_src

- B1
  #+begin_src sh
  cp ../b1-anno/0-ref/B1.genome.fa B1.fa
  #+end_src

#+begin_src sh :tangle joint.db.sh

ml singularity/3.5.2 bwa

cat B1.fa D5.fa A2.fa > joint.fa
bwa index joint.fa

#+end_src

* Reads

#+NAME: sra_data
| SRA         | Species              | PI       |
|-------------+----------------------+----------|
| SRR6334584  | Gossypium davidsonii | Udall    |
| SRR8136261  | Gossypium davidsonii | Peterson |
| SRR6334602  | 2(A2D1)              | Udall    |
| SRR6334601  | 2(A2D3)              | Udall    |
| SRR3560153  | Gossypium anomalum   | Peterson |
| SRR3560155  | Gossypium anomalum   | Peterson |
| SRR3560156  | Gossypium anomalum   | Peterson |
| SRR12745560 | Gossypium anomalum   | CAAS     |
| SRR8979965  | Gossypium arboreum   | Udall    |
| SRR8979944  | Gossypium arboreum   | Udall    |
| SRR8979925  | Gossypium arboreum   | Udall    |
| SRR3560162  | Gossypium somalense  | E2       |
| SRR617704   | Gossypium longicalyx | F1       |


#+begin_src sh :var sra=sra_data[,0] :tangle raw/downlaod.sh

SRA=${sra[$SLURM_ARRAY_TASK_ID]};

URL=ftp://ftp.sra.ebi.ac.uk/vol1/fastq/${SRA:0:6}/

if [ ${#SRA} -gt 9 ]; then
    DIR=00${SRA:9}
    URL+=${DIR: -3}/
fi

URL+=$SRA

wget $URL/${SRA}_1.fastq.gz && wget $URL/${SRA}_2.fastq.gz || wget $URL/${SRA}.fastq.gz

#+end_src

* Align

#+begin_src sh :var sra=sra_data[,0] :tangle align.sh

SRA=${sra[$SLURM_ARRAY_TASK_ID]};

ml singularity/3.5.2 bwa samtools

bwa mem -t $SLURM_CPUS_PER_TASK joint.fa raw/${SRA}_{1,2}.fastq.gz |
    samtools view -bS - |
    samtools sort -o $SRA.bam -T $SRA.tmp -
samtools index $SRA.bam
samtools flagstat $SRA.bam > $SRA.stats
#+end_src


#+begin_src sh :var sra=sra_data[,0] :tangle stat.sh
ml singularity/3.5.2 bwa samtools parallel

# for fa in B1 A2 D5; do
#     samtools faidx $fa.fa
#     awk '{print $1, 0, $2}' OFS="\t" $fa.fa.fai > $fa.bed
# done

SRA=${sra[$SLURM_ARRAY_TASK_ID]};

    echo $SRA
    sed -n -e '1s/ .*//p' -e '5s/ .*//p' $SRA.stats

    for fa in B1 A2 D5; do
        samtools view  -L $fa.bed -q 30 -c $SRA.bam
    done




#+end_src

| Accession   | Species              |     total |    mapped |        B1 |  B1 % |        A2 |  A2 % |        D5 |  D5 % |
|-------------+----------------------+-----------+-----------+-----------+-------+-----------+-------+-----------+-------|
| SRR6334584  | Gossypium davidsonii | 242344682 | 240324128 |   7270216 |  3.03 |  10121407 |  4.21 | 148054946 | 61.61 |
| SRR8136261  | Gossypium davidsonii | 337437239 | 334699019 |   8624842 |  2.58 |   9092561 |  2.72 | 196922555 | 58.84 |
| SRR6334602  | 2(A2D1)              | 310722422 | 295843181 |   4904740 |  1.66 | 159135266 | 53.79 |  59248960 | 20.03 |
| SRR6334601  | 2(A2D3)              | 322043617 | 316189434 |   6230117 |  1.97 | 132879559 | 42.03 |  80051268 | 25.32 |
| SRR3560153  | Gossypium anomalum   | 305977284 | 297638880 | 250268307 | 84.08 |   5231412 |  1.76 |    718280 |  0.24 |
| SRR3560155  | Gossypium anomalum   | 269209430 | 263528164 | 221044814 | 83.88 |   4620671 |  1.75 |    637811 |  0.24 |
| SRR3560156  | Gossypium anomalum   | 132066295 | 128737188 | 107684954 | 83.65 |   2347574 |  1.82 |    323087 |  0.25 |
| SRR12745560 | Gossypium anomalum   | 219435120 | 218381356 | 183823170 | 84.18 |   7061835 |  3.23 |    969963 |  0.44 |
| SRR8979965  | Gossypium arboreum   | 478364261 | 476168092 |   5455280 |  1.15 | 346423334 | 72.75 |   1719046 |  0.36 |
| SRR8979944  | Gossypium arboreum   | 424564635 | 422364742 |   4765932 |  1.13 | 308666069 | 73.08 |   1595026 |  0.38 |
| SRR8979925  | Gossypium arboreum   | 415343892 | 414200416 |   4494645 |  1.09 | 302706188 | 73.08 |   1290059 |  0.31 |
#+TBLFM: $6=100*$5/$4;%0.2f :: $8=100*$7/$4;%0.2f :: $10=100*$9/$4;%0.2f



* bedtools

#+begin_src sh :tangle bedtools.sh

sra=(SRR6334601 SRR6334602 SRR8979965 SRR8979944 SRR8979925 SRR6334584 SRR8136261)

ml singularity/3.5.2 bedtools samtools parallel

parallel -j12 --eta "singularity exec /apps/singularity-3/samtools/samtools-v1.9-4-deb_cv1.sif samtools view -q 30 -b -L B1.bed {}.bam | singularity exec /apps/singularity-3/bedtools/bedtools-2.28.0.sif bedtools bamtobed -i - > {}.B1.bed" ::: ${sra[@]}

# Remove all overlaps that may be A2, D3, or A2D1 related
sort -m -k1,1 -k2,2n {SRR6334584,SRR8136261}.B1.bed |
    bedtools merge > D3.B1.bed
sort -m -k1,1 -k2,2n {SRR8979965,SRR8979944,SRR8979925}.B1.bed |
    bedtools merge > A2.B1.bed
sort -m -k1,1 -k2,2n D3.B1.bed A2.B1.bed \
        <(bedtools merge -i SRR6334602.B1.bed) |
    bedtools merge |
    bedtools subtract -a SRR6334601.B1.bed -b - |
    sort -k1,1 -k2,2n |
    bedtools merge > A2D3.B1.specific.bed

# Get reads that are B1 specific in A2D3
pv  SRR6334601.bam |
    samtools view -q 30 -b -L A2D3.B1.specific.bed -  > A2D3.B1.specific.bam
bedtools bamtobed -i A2D3.B1.specific.bam |
    bedtools genomecov -g B1.fa.fai -i - > A2D3.B1.specific.genomecov


bedtools bamtobed -i A2D3.B1.specific.bam | bedtools merge -i - | awk '$3-$2 > 5000'
#+end_src

#+CAPTION: Regions of A2D3 B1 specific overlap > 5000
| B_02 |   1751517 |   1759641 |
| B_02 |  20496842 |  20502002 |
| B_02 |  70412653 |  70418445 |
| B_03 | 106013266 | 106019946 |
| B_04 |  50725528 |  50730720 |
| B_05 |   2823728 |   2829382 |
| B_05 |  10464363 |  10469657 |
| B_06 |  92554341 |  92562783 |
| B_06 |  92562858 |  92570652 |
| B_06 |  92641196 |  92646951 |
| B_06 |  92646956 |  92657533 |
| B_06 |  92670418 |  92675475 |
| B_06 |  92675847 |  92681845 |
| B_06 |  92681890 |  92694410 |
| B_06 |  92736613 |  92744234 |
| B_06 |  92748186 |  92760871 |
| B_06 |  92774479 |  92782790 |
| B_06 |  92785370 |  92790493 |
| B_06 |  92819750 |  92830095 |
| B_06 |  92830555 |  92841871 |
| B_07 |  76245298 |  76251283 |
| B_08 |  15137345 |  15143614 |
| B_08 |  94864034 |  94870100 |
| B_11 |  51368114 |  51373226 |
| B_11 |  86773815 |  86779971 |
| B_11 |  95280456 |  95287671 |
| B_11 |  99060757 |  99066311 |
| B_12 |  17145074 |  17150915 |



#+Caption: Overlapping genes
| B1.B_02G017200 | B_02 |   1759353 |   1759641 | NGA4: B3 domain-containing transcription factor NGA4 (Arabidopsis thaliana OX%3D3702)                            |
| B1.B_03G260300 | B_03 | 106013267 | 106019946 | Os03g0733400: Zinc finger BED domain-containing protein RICESLEEPER 2 (Oryza sativa subsp. japonica OX%3D39947)  |
| B1.B_05G053800 | B_05 |  10464364 |  10465910 | At3g47570: Probable LRR receptor-like serine/threonine-protein kinase At3g47570 (Arabidopsis thaliana OX%3D3702) |
| B1.B_06G223600 | B_06 |  92567389 |  92568340 | At4g19870: F-box/kelch-repeat protein At4g19870 (Arabidopsis thaliana OX%3D3702)                                 |
| B1.B_06G223900 | B_06 |  92741077 |  92741403 | PAP12: Fe(3+)-Zn(2+) purple acid phosphatase 12 (Arabidopsis thaliana OX%3D3702);                                |
| B1.B_08G233100 | B_08 |  94864035 |  94870100 | HMA5: Probable copper-transporting ATPase HMA5 (Arabidopsis thaliana OX%3D3702)                                  |
| B1.B_11G340700 | B_11 |  95280537 |  95281955 | Retrovirus-related Pol polyprotein from transposon TNT 1-94 (Nicotiana tabacum OX%3D4097)                        |
| B1.B_11G362400 | B_11 |  99060758 |  99066311 | At3g47200: UPF0481 protein At3g47200 (Arabidopsis thaliana OX%3D3702);                                           |



* overlapping gene
B06 has a large region of A2D3 specific reads. Only one gene is in that region,
F-box/kelch-repeat protein ( [[https://www.arabidopsis.org/servlets/TairObject?name=AT4G19870&type=locus][At4g19870]] ). A homolog of that gene if radish is
reported to interact with CMS via miRNA (miR395a). ([[https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4956657/][PMC4956657]],[[https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5713397/][PMC5713397]])


- Protein blast results for B1.B_06G223600-1 vs B1, D5, and A2
 #+Caption: B1.B_06G223600-1 -- F-box/kelch-repeat protein
 #+begin_src txt :tangle pep.of.interest.fa
>B1.B_06G223600-1  Name:"Similar to At4g19870 F-box/kelch-repeat protein At4g19870 (Arabidopsis thaliana OX=3702)"
MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPLSSTAVACGNH
VYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLVLAAEEKIYVFERIGSESFGE
VYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNANDTLYAYYFDRKSWVCLEQKFCYWSESATIV
DDVLYTVIYNYSGKFRSLEAYNLLDKKHLTVKWSSGFSGGSNNEWHSGGIHAAAEHQSAITVPYPKDVHD
IMLF
 #+end_src
 #+begin_src sh
for genome in B1 D5 A2 AD1; do
    blastp -query pep.of.interest.fa \
           -subject $genome.proteins.fa \
           -evalue 1e-50 \
           -out $genome.fbox-like.blast
done
 #+end_src
  + B1
    #+begin_example
BLASTP 2.9.0+


Reference: Stephen F. Altschul, Thomas L. Madden, Alejandro A.
Schaffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and David J.
Lipman (1997), "Gapped BLAST and PSI-BLAST: a new generation of
protein database search programs", Nucleic Acids Res. 25:3389-3402.


Reference for composition-based statistics: Alejandro A. Schaffer,
L. Aravind, Thomas L. Madden, Sergei Shavirin, John L. Spouge, Yuri
I. Wolf, Eugene V. Koonin, and Stephen F. Altschul (2001),
"Improving the accuracy of PSI-BLAST protein database searches with
composition-based statistics and other refinements", Nucleic Acids
Res. 29:2994-3005.



Database: User specified sequence set (Input: B1.proteins.fa).
           37,830 sequences; 18,135,083 total letters



Query= B1.B_06G223600-1  Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"

Length=284
                                                                      Score        E
Sequences producing significant alignments:                          (Bits)     Value

B1.B_06G223600-1 Name:"Similar to At4g19870 F-box/kelch-repeat pr...  589        0.0
B1.B_06G222300-1 Name:"Similar to At4g19870 F-box/kelch-repeat pr...  455        5e-163
B1.B_06G214200-1 Name:"Similar to At4g19870 F-box/kelch-repeat pr...  431        2e-153
B1.B_06G224100-1 Name:"Similar to At4g19870 F-box/kelch-repeat pr...  430        4e-153
B1.B_06G214100-1 Name:"Similar to At4g19870 F-box/kelch-repeat pr...  421        2e-149
B1.B_06G224000-1 Name:"Similar to At4g19870 F-box/kelch-repeat pr...  412        2e-146


> B1.B_06G223600-1 Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
Length=284

 Score = 589 bits (1518),  Expect = 0.0, Method: Compositional matrix adjust.
 Identities = 284/284 (100%), Positives = 284/284 (100%), Gaps = 0/284 (0%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL
Sbjct  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
            SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV
Sbjct  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN
Sbjct  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT
Sbjct  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240

Query  241  VKWSSGFSGGSNNEWHSGGIHAAAEHQSAITVPYPKDVHDIMLF  284
            VKWSSGFSGGSNNEWHSGGIHAAAEHQSAITVPYPKDVHDIMLF
Sbjct  241  VKWSSGFSGGSNNEWHSGGIHAAAEHQSAITVPYPKDVHDIMLF  284


> B1.B_06G222300-1 Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
Length=316

 Score = 455 bits (1171),  Expect = 5e-163, Method: Compositional matrix adjust.
 Identities = 235/316 (74%), Positives = 251/316 (79%), Gaps = 32/316 (10%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDELIR+LVE  PGTDG+  LG  F+D++ N VFIRAMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELIRALVENSPGTDGKTALGLHFIDLKSNKVFIRAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAVACGNH+YVIGGIR+NDA++ GYD+VNDVFQLDLKDLERGWRKTTSMLFPR+FPLV
Sbjct  61   GSTAVACGNHIYVIGGIRENDASLRGYDEVNDVFQLDLKDLERGWRKTTSMLFPRAFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAE KIYVFE +GSESFGEVYDISGDIWEPL  PPE I L  PV DSSRSRILVHCNAN
Sbjct  121  LAAEGKIYVFEGMGSESFGEVYDISGDIWEPLLPPPEAIDLSVPVPDSSRSRILVHCNAN  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWVCLEQKFCYWS+SATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHL
Sbjct  181  DTLYAYYYDRKSWVCLEQKFCYWSDSATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLP  240

Query  241  VKWSSGFS-------------------GGSN-----------NEW--HSGGIHAAAEHQS  268
            VKWSS FS                   G  N           N W    GGIHAAAEHQ
Sbjct  241  VKWSSEFSVNPPPNGTLYRLGNGKLILGWVNLFHERFEYIRFNIWCNEQGGIHAAAEHQF  300

Query  269  AITVPYPKDVHDIMLF  284
            AITV YP+D+  I LF
Sbjct  301  AITVSYPEDISSIWLF  316


> B1.B_06G214200-1 Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
Length=312

 Score = 431 bits (1107),  Expect = 2e-153, Method: Compositional matrix adjust.
 Identities = 222/315 (70%), Positives = 244/315 (77%), Gaps = 34/315 (11%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDELIR+LV+KWP TDG++TLG  F+D+E N VFI+AMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELIRALVDKWPDTDGKITLGLRFIDLERNKVFIKAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAV+CGNHVYVIGG+R+NDA   GYD VNDVFQLDLKDLERGWRKTTSMLFPR FPLV
Sbjct  61   CSTAVSCGNHVYVIGGMRENDANFRGYDRVNDVFQLDLKDLERGWRKTTSMLFPRVFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAE +IYVFE +GSESFGEVYDISGD WEPLS PPE I +C P+LDSSRSRILVHC+A+
Sbjct  121  LAAEGRIYVFEYLGSESFGEVYDISGDFWEPLSPPPEAIDICVPILDSSRSRILVHCHAS  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWV LEQKFCYWS++ATIVDDVLYTV+ NYS    SLEAYNLLDKKHL
Sbjct  181  DTLYAYYYDRKSWVYLEQKFCYWSDTATIVDDVLYTVLDNYSC---SLEAYNLLDKKHLP  237

Query  241  VKWSS-----------------------------GFSGGSNNEW--HSGGIHAAAEHQSA  269
            VKWSS                              F     N W    GGIH AAEHQSA
Sbjct  238  VKWSSEVPVTPPQGCTLYRLGSGKLILGWVNHLRTFEYIRFNIWCNEQGGIHVAAEHQSA  297

Query  270  ITVPYPKDVHDIMLF  284
            ITVPYPKD+ +I  F
Sbjct  298  ITVPYPKDISNIWFF  312


> B1.B_06G224100-1 Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
Length=316

 Score = 430 bits (1106),  Expect = 4e-153, Method: Compositional matrix adjust.
 Identities = 224/316 (71%), Positives = 246/316 (78%), Gaps = 32/316 (10%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDELIR+LV K  GTDG++  G  F+D+E N VFIRAMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELIRALVRKRRGTDGKIAHGLHFIDLESNKVFIRAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAVACGNHVY+IGGI +NDA   GYD VNDVFQLDLKDLERGWRKTTSMLFP++FPLV
Sbjct  61   CSTAVACGNHVYIIGGIHENDANFRGYDGVNDVFQLDLKDLERGWRKTTSMLFPQAFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LA E KIY FE  GSESFGEVYDISGDIWEPLS+PPE I LC PVLDSSRSRILVHC+A+
Sbjct  121  LAVEGKIYFFEYRGSESFGEVYDISGDIWEPLSSPPEDIDLCVPVLDSSRSRILVHCDAS  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHL-  239
            DTLYAYY+DR SWVCLEQKFCYWS+SATIVDDVLYT+IYNYS KFRSLEAYNLLDKKHL
Sbjct  181  DTLYAYYYDRNSWVCLEQKFCYWSDSATIVDDVLYTLIYNYSCKFRSLEAYNLLDKKHLP  240

Query  240  -----TVKW-------------SSGFSGGSN-----------NEW--HSGGIHAAAEHQS  268
                   +W             +S F  G N           N W    GGIHAAAEH+S
Sbjct  241  SNGHQNFQWLHHWVALCIVWAMASLFWVGVNQFRKRFEYIRFNIWCNEQGGIHAAAEHKS  300

Query  269  AITVPYPKDVHDIMLF  284
            AI+VPYP+D+ +I LF
Sbjct  301  AISVPYPEDISNIWLF  316


> B1.B_06G214100-1 Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
Length=312

 Score = 421 bits (1082),  Expect = 2e-149, Method: Compositional matrix adjust.
 Identities = 219/315 (70%), Positives = 239/315 (76%), Gaps = 34/315 (11%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDELIR+LV+KWP TDG+M LG  F+D+E N VFI+AMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELIRALVQKWPDTDGKMPLGLRFIDLERNKVFIKAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAV+CGNHVYVIGGIR+NDA   GYD +NDVFQLDL DLERGWRKTTSMLFPR+FPLV
Sbjct  61   GSTAVSCGNHVYVIGGIRENDANFRGYDGLNDVFQLDLNDLERGWRKTTSMLFPRAFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAE KIYVFE  GSESFGEVYDISGD WE LS PPE I +C PVLDSSRSRILVHC+A+
Sbjct  121  LAAEGKIYVFEYWGSESFGEVYDISGDFWELLSPPPEAIDICVPVLDSSRSRILVHCHAS  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWVCLEQKFCYWS++ATIVDDVLYTV+ NYS    SLEAYNLLDKKHL
Sbjct  181  DTLYAYYYDRKSWVCLEQKFCYWSDTATIVDDVLYTVLDNYSC---SLEAYNLLDKKHLP  237

Query  241  VKWSS-----------------------------GFSGGSNNEW--HSGGIHAAAEHQSA  269
            VKW S                              F     N W    GGIH AAE QSA
Sbjct  238  VKWLSEVPVTPPQGCTLYRLGSGKLILGWVNHLRTFEYLRFNIWCNEQGGIHVAAERQSA  297

Query  270  ITVPYPKDVHDIMLF  284
            I VPYPKD+ ++  F
Sbjct  298  IAVPYPKDICNLWFF  312


> B1.B_06G224000-1 Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
Length=306

 Score = 412 bits (1060),  Expect = 2e-146, Method: Compositional matrix adjust.
 Identities = 217/316 (69%), Positives = 238/316 (75%), Gaps = 42/316 (13%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF G++PDELIR+LV K PGTDG++  G  F D+E N VFI+AMPPDAP
Sbjct  1    MSGDTSDSSSYFDGQDPDELIRALVRKRPGTDGKIVHGLHFFDLESNKVFIKAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             ST VACGNHVYVIGGIR+NDA   GYD           DLERGWRKTTSMLFPR+FPLV
Sbjct  61   CSTVVACGNHVYVIGGIRENDANFRGYD----------ADLERGWRKTTSMLFPRAFPLV  110

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAA+ K+YVFE  GSESFGEVYDISGDIWEPLS PPE I L  PVLDSSRSRILVHC+A+
Sbjct  111  LAADGKVYVFEHWGSESFGEVYDISGDIWEPLSPPPEDIDLSVPVLDSSRSRILVHCDAS  170

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWVCLEQKFCYWS+SATIVDDVLYT+IYNYS KFRSLEAYNLLDKKHL
Sbjct  171  DTLYAYYYDRKSWVCLEQKFCYWSDSATIVDDVLYTLIYNYSCKFRSLEAYNLLDKKHLP  230

Query  241  VKWSSGFS-------------------GGSN-----------NEW--HSGGIHAAAEHQS  268
            VKWSS FS                   G  N           N W    GGIHA AEH+S
Sbjct  231  VKWSSEFSVAPPLGSTPYRLGNGKLILGWVNQFRKRFEYIRFNIWCNEQGGIHAVAEHKS  290

Query  269  AITVPYPKDVHDIMLF  284
            AI+VPYP+D+ +I LF
Sbjct  291  AISVPYPEDISNIWLF  306



Lambda      K        H        a         alpha
   0.319    0.137    0.434    0.792     4.96

Gapped
Lambda      K        H        a         alpha    sigma
   0.267   0.0410    0.140     1.90     42.6     43.6

Effective search space used: 2619508299


  Database: User specified sequence set (Input: B1.proteins.fa).
    Posted date:  Unknown
  Number of letters in database: 18,135,083
  Number of sequences in database:  37,830



Matrix: BLOSUM62
Gap Penalties: Existence: 11, Extension: 1
Neighboring words threshold: 11
Window for multiple hits: 40

    #+end_example
  + D5
    #+begin_example
BLASTP 2.9.0+


Reference: Stephen F. Altschul, Thomas L. Madden, Alejandro A.
Schaffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and David J.
Lipman (1997), "Gapped BLAST and PSI-BLAST: a new generation of
protein database search programs", Nucleic Acids Res. 25:3389-3402.


Reference for composition-based statistics: Alejandro A. Schaffer,
L. Aravind, Thomas L. Madden, Sergei Shavirin, John L. Spouge, Yuri
I. Wolf, Eugene V. Koonin, and Stephen F. Altschul (2001),
"Improving the accuracy of PSI-BLAST protein database searches with
composition-based statistics and other refinements", Nucleic Acids
Res. 29:2994-3005.



Database: User specified sequence set (Input: D5.proteins.fa).
           41,030 sequences; 16,489,195 total letters



Query= B1.B_06G223600-1  Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"

Length=284
                                                                      Score        E
Sequences producing significant alignments:                          (Bits)     Value

D5.v1.pred_00030947-RA protein Name:"Similar to At4g19870 F-box/k...  461        2e-165
D5.v1.pred_00030954-RA protein Name:"Similar to At4g19870 F-box/k...  416        6e-148
D5.v1.pred_00030938-RA protein Name:"Similar to At2g39490 F-box p...  311        7e-104
D5.v1.pred_00030950-RA protein Name:"Similar to At4g19870 F-box/k...  280        2e-95
D5.v1.pred_00030965-RA protein Name:"Similar to At4g19870 F-box/k...  253        6e-84


> D5.v1.pred_00030947-RA protein Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
AED:0.00 eAED:0.00 QI:0|-1|0|1|-1|1|1|0|314
Length=314

 Score = 461 bits (1186),  Expect = 2e-165, Method: Compositional matrix adjust.
 Identities = 233/314 (74%), Positives = 252/314 (80%), Gaps = 30/314 (10%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDELIR+LV+KWPGTDG++T+G  FLD+E N+VFI+AMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELIRALVQKWPGTDGKITVGLHFLDLESNEVFIKAMPPDAPF  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAVACG+HVYVIGGIR+NDAT+F  D VNDVFQLDLKDLERGWRKTTSMLFPR  P V
Sbjct  61   CSTAVACGDHVYVIGGIRQNDATLFDNDGVNDVFQLDLKDLERGWRKTTSMLFPRVLPRV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            +AAE KI+VFE +GSESFGEVYDISGDIWEPLS PPE I LC PVLDSSRSRILVHCNAN
Sbjct  121  VAAEGKIFVFEYMGSESFGEVYDISGDIWEPLSPPPEDIDLCVPVLDSSRSRILVHCNAN  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWVCLEQKFCYWS++ATIVDDVLYT IYNY  KFRSLEAYNLLDKKHL
Sbjct  181  DTLYAYYYDRKSWVCLEQKFCYWSDAATIVDDVLYTAIYNYYDKFRSLEAYNLLDKKHLP  240

Query  241  VKWSSGFS------------GGSNNEW------------------HSGGIHAAAEHQSAI  270
            VKWSS FS            G     W                    GGIHAAAEHQSAI
Sbjct  241  VKWSSEFSVDPHGTLYRLGNGKCIFVWFNRLDKSFEYIRFHIWCNEQGGIHAAAEHQSAI  300

Query  271  TVPYPKDVHDIMLF  284
            +VPYPKD+ DI LF
Sbjct  301  SVPYPKDISDIKLF  314


> D5.v1.pred_00030954-RA protein Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
AED:0.03 eAED:0.03 QI:0|0|0|1|1|1|2|0|297
Length=297

 Score = 416 bits (1070),  Expect = 6e-148, Method: Compositional matrix adjust.
 Identities = 218/314 (69%), Positives = 234/314 (75%), Gaps = 47/314 (15%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            M GDTSDSSSYF GREPDELIR+LV+ WPGTDG+MT+G  FLD+E N VFI+AMPPDAP
Sbjct  1    MRGDTSDSSSYFDGREPDELIRALVQNWPGTDGKMTVGLHFLDLESNKVFIKAMPPDAPF  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAVACG+HVYVIGGIR+NDAT+F YD VNDVFQLDLKDLERGWRKTTSMLFPR  P V
Sbjct  61   CSTAVACGDHVYVIGGIRQNDATLFDYDGVNDVFQLDLKDLERGWRKTTSMLFPRFLPRV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            +AAE KI+VFE +GSESFGEVYDISGDIWEPLS PPE I LC PVLDSSRSRILVHCNAN
Sbjct  121  VAAEGKIFVFEYMGSESFGEVYDISGDIWEPLSPPPEDIDLCVPVLDSSRSRILVHCNAN  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+D                  TIVDDVLYT IYNYS KFRSLEAYNLLDKKHL
Sbjct  181  DTLYAYYYD-----------------PTIVDDVLYTAIYNYSDKFRSLEAYNLLDKKHLP  223

Query  241  VKWSSGFSGGSN----------------------------NEW--HSGGIHAAAEHQSAI  270
            VKWSS FS   +                            N W    GGIHAAAEHQSAI
Sbjct  224  VKWSSEFSVDPHGTLYRLGNGKCIFVWFNHLDKSFEYIRFNIWCNEQGGIHAAAEHQSAI  283

Query  271  TVPYPKDVHDIMLF  284
            +VPYPKD+ DI LF
Sbjct  284  SVPYPKDISDIKLF  297


> D5.v1.pred_00030938-RA protein Name:"Similar to At2g39490 F-box
protein At2g39490 (Arabidopsis thaliana OX=3702)" AED:0.25
eAED:0.45 QI:0|0|0|0.37|1|1|8|0|508
Length=508

 Score = 311 bits (798),  Expect = 7e-104, Method: Compositional matrix adjust.
 Identities = 178/314 (57%), Positives = 198/314 (63%), Gaps = 59/314 (19%)

Query  3    GDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPLSS  62
            GDTSDSSSYF GREPDELIR+LV+K PGTDG+ TLG  F+D+E N VFI+AMPPDAP SS
Sbjct  91   GDTSDSSSYFDGREPDELIRALVQKRPGTDGKRTLGLHFIDLESNKVFIKAMPPDAPSSS  150

Query  63   TAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLVLA  122
            TAV+CGNHVYVIGGIRK    +       +V          G  K         FP+ L
Sbjct  151  TAVSCGNHVYVIGGIRKMMLLLLVMMMTLNV----------GGEKLLLCCSLGLFPMFLL  200

Query  123  AEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGL---CDPVLDSSRSRILVHCNA  179
             +E+          +   VYDISGDIWEPLS PPE I +   C PVLDSSRSRILVHC+A
Sbjct  201  LKERF---------TSSNVYDISGDIWEPLSPPPEAIDIRVVCVPVLDSSRSRILVHCDA  251

Query  180  NDTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHL  239
            +DTLYAYY+DRKSWVCLEQKFCYW +SA IVDDVLYT IY       SLEAYNLLDKKHL
Sbjct  252  SDTLYAYYYDRKSWVCLEQKFCYWFDSAAIVDDVLYTFIYKC-----SLEAYNLLDKKHL  306

Query  240  TVKWSS------------------------------GFSGGSNNEW--HSGGIHAAAEHQ  267
             VKWSS                              GF     N W    GGIHAAAEHQ
Sbjct  307  PVKWSSEFPVAPPLGSALYRLGNGKLILGWVNHLHRGFECIRFNIWCNEQGGIHAAAEHQ  366

Query  268  SAITVPYPKDVHDI  281
            SAITVPYP+D+  I
Sbjct  367  SAITVPYPEDISSI  380


> D5.v1.pred_00030950-RA protein Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
AED:0.31 eAED:0.75 QI:0|0|0|1|1|1|3|0|210
Length=210

 Score = 280 bits (716),  Expect = 2e-95, Method: Compositional matrix adjust.
 Identities = 146/217 (67%), Positives = 159/217 (73%), Gaps = 40/217 (18%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPD+LIR+L                      +  FI+AMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDQLIRAL----------------------HLSFIKAMPPDAPS  38

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAV+CGNHVYVIGGIRKNDAT+                      KTTSMLFPRS PLV
Sbjct  39   CSTAVSCGNHVYVIGGIRKNDATLLL------------------LIKTTSMLFPRSLPLV  80

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
             AAE KI+VFE +GSESFGEVY+ISGDIWEPLS PPE I LC PVLDSSRSRILVHC+AN
Sbjct  81   AAAEGKIFVFENMGSESFGEVYNISGDIWEPLSPPPEDIDLCVPVLDSSRSRILVHCSAN  140

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTV  217
            DTLYAYY+DRKSWVCLEQKFCYWS++ATIVDDVLYT
Sbjct  141  DTLYAYYYDRKSWVCLEQKFCYWSDAATIVDDVLYTA  177


> D5.v1.pred_00030965-RA protein Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"
AED:0.31 eAED:-0.13 QI:0|0|0|0.75|1|1|4|0|266
Length=266

 Score = 253 bits (645),  Expect = 6e-84, Method: Compositional matrix adjust.
 Identities = 151/279 (54%), Positives = 173/279 (62%), Gaps = 39/279 (14%)

Query  19   ELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPLSSTAVACGNHVYVIGGIR  78
            ELIR+ V+KWPGTDG+MTLG  FLD+E ++VFIRAMPPDAPLS   +        +G
Sbjct  14   ELIRTHVQKWPGTDGKMTLGLHFLDLESDEVFIRAMPPDAPLSPLPLPVATTFTFLGEYA  73

Query  79   KNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLVLAAEEKIYVFERIGSESF  138
            K    +           +  + L  G  K         FPLVLAAE KIYVFE +G ESF
Sbjct  74   KMMLLL-----------VIGRTLNVGGEKLLLCCSLGLFPLVLAAEGKIYVFELLGFESF  122

Query  139  GEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNANDTLYAYYFDRKSWVCLEQ  198
            G+VYDISGDIWEPLS          P+   S   +L   +     YAYY+DRKSWVCLE
Sbjct  123  GDVYDISGDIWEPLS----------PLRKLSLYVVLFWFS-----YAYYYDRKSWVCLEP  167

Query  199  KFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLTVKWSSGFS----------  248
            KFCYWS+ A+IVDDVLYTV+YNYSG+F SLEAYNLLDKKHL VKWSS FS
Sbjct  168  KFCYWSDPASIVDDVLYTVMYNYSGEFCSLEAYNLLDKKHLPVKWSSEFSVDPPRSGTLF  227

Query  249  --GGSNNEWHSGGIHAAAEHQSAITVPY-PKDVHDIMLF  284
              G        GGI AAAEHQSAITVPY  KD+ DI LF
Sbjct  228  GLGNGKCILRWGGIDAAAEHQSAITVPYRSKDICDIFLF  266



Lambda      K        H        a         alpha
   0.319    0.137    0.434    0.792     4.96

Gapped
Lambda      K        H        a         alpha    sigma
   0.267   0.0410    0.140     1.90     42.6     43.6

Effective search space used: 2279059880


  Database: User specified sequence set (Input: D5.proteins.fa).
    Posted date:  Unknown
  Number of letters in database: 16,489,195
  Number of sequences in database:  41,030



Matrix: BLOSUM62
Gap Penalties: Existence: 11, Extension: 1
Neighboring words threshold: 11
Window for multiple hits: 40

    #+end_example
  + A2
    #+begin_example
BLASTP 2.9.0+


Reference: Stephen F. Altschul, Thomas L. Madden, Alejandro A.
Schaffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and David J.
Lipman (1997), "Gapped BLAST and PSI-BLAST: a new generation of
protein database search programs", Nucleic Acids Res. 25:3389-3402.


Reference for composition-based statistics: Alejandro A. Schaffer,
L. Aravind, Thomas L. Madden, Sergei Shavirin, John L. Spouge, Yuri
I. Wolf, Eugene V. Koonin, and Stephen F. Altschul (2001),
"Improving the accuracy of PSI-BLAST protein database searches with
composition-based statistics and other refinements", Nucleic Acids
Res. 29:2994-3005.



Database: User specified sequence set (Input: A2.proteins.fa).
           43,278 sequences; 16,644,893 total letters



Query= B1.B_06G223600-1  Name:"Similar to At4g19870 F-box/kelch-repeat
protein At4g19870 (Arabidopsis thaliana OX=3702)"

Length=284
                                                                      Score        E
Sequences producing significant alignments:                          (Bits)     Value

Gar11G43950                                                           447        8e-160
Gar06G05690                                                           441        3e-157
Gar06G05390                                                           434        1e-154
Gar06G04460                                                           407        3e-145
Gar06G04480                                                           409        3e-145
Gar06G04420                                                           380        2e-134
Gar06G05370                                                           233        2e-77


> Gar11G43950
Length=319

 Score = 447 bits (1150),  Expect = 8e-160, Method: Compositional matrix adjust.
 Identities = 232/315 (74%), Positives = 250/315 (79%), Gaps = 32/315 (10%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDELIR+LVEK PGTDG++T G  ++D+E N VFIRAMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELIRALVEKSPGTDGKITHGLHYIDLESNKVFIRAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAVACGNHVYVIGGIR+NDA   GYD VNDVFQLDLKDLERGWRKTTSMLFPR+FPLV
Sbjct  61   CSTAVACGNHVYVIGGIRENDANFRGYDGVNDVFQLDLKDLERGWRKTTSMLFPRAFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAE KIYVFE  GSESFGEVYDISGDIWEPLS PPE I +C PVLD SRSRILVHC+A+
Sbjct  121  LAAEGKIYVFEYWGSESFGEVYDISGDIWEPLSPPPEDIDICVPVLDCSRSRILVHCHAS  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            D LYAYY+DRKSWVCLEQKFCYWS+SATIVDDVLYT+I+NYS KFRSLEAYNLLDKKHL
Sbjct  181  DALYAYYYDRKSWVCLEQKFCYWSDSATIVDDVLYTLIFNYSYKFRSLEAYNLLDKKHLP  240

Query  241  VKWSSGFS----GGSN--------------------------NEW--HSGGIHAAAEHQS  268
            VKWSS FS     GS                           N W    GGIHAAAEHQS
Sbjct  241  VKWSSEFSVAPPPGSTLYRLGNGKLILGWVNHFDKRFEYIRFNIWCNEQGGIHAAAEHQS  300

Query  269  AITVPYPKDVHDIML  283
             ITVPYP+D+ +I L
Sbjct  301  PITVPYPEDMENICL  315


> Gar06G05690
Length=316

 Score = 441 bits (1133),  Expect = 3e-157, Method: Compositional matrix adjust.
 Identities = 230/316 (73%), Positives = 246/316 (78%), Gaps = 32/316 (10%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDS SYF GREPDELIR+L  K+PGTDGE+T G  F+D+E N VFIRAMPPDA
Sbjct  1    MSGDTSDSFSYFDGREPDELIRALFGKYPGTDGEITHGLHFIDLESNKVFIRAMPPDARS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAVACGNHV VIGGIR+NDA   GYD VNDVFQLDLKDLERGWRKTTSMLFPR+FPLV
Sbjct  61   CSTAVACGNHVNVIGGIRENDANFRGYDGVNDVFQLDLKDLERGWRKTTSMLFPRAFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAE KIYVFE  GSESFGEVYDIS DIWEPLS PPE I LC PVLDSSRSRILVHC+A+
Sbjct  121  LAAEGKIYVFEYWGSESFGEVYDISRDIWEPLSPPPEDIDLCVPVLDSSRSRILVHCHAS  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWVCLEQKFCYWS+SATIVDDVLYT+IYNYS KFRSLEAYNL DKKHL
Sbjct  181  DTLYAYYYDRKSWVCLEQKFCYWSDSATIVDDVLYTLIYNYSCKFRSLEAYNLFDKKHLP  240

Query  241  VKWSSGFS------------------GGSNNEWH--------------SGGIHAAAEHQS  268
            VK SS FS                   G  N +H               GGIHAA EHQS
Sbjct  241  VKLSSEFSVAPPLGSTLYHLGNGKLISGWVNHFHKRFEYIRFNIWCNEQGGIHAAVEHQS  300

Query  269  AITVPYPKDVHDIMLF  284
            AITVPYP+D+ +I LF
Sbjct  301  AITVPYPEDISNIWLF  316


> Gar06G05390
Length=312

 Score = 434 bits (1115),  Expect = 1e-154, Method: Compositional matrix adjust.
 Identities = 223/315 (71%), Positives = 245/315 (78%), Gaps = 34/315 (11%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDELIR+LV+KWP TDG+MTLG  F+D+E N VFI+AMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELIRALVQKWPDTDGKMTLGLRFIDLERNKVFIKAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAV+CGNHVYVIGG+R+NDA   GYD VNDVFQLDLKDLERGWRKT SMLFPR+FPLV
Sbjct  61   GSTAVSCGNHVYVIGGMRENDANFRGYDGVNDVFQLDLKDLERGWRKTISMLFPRAFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAE KIYVFE  GSESFGEVYDISGD WEPLS PPE I +C PVLDSSRSRILVHC+A+
Sbjct  121  LAAEGKIYVFEYWGSESFGEVYDISGDFWEPLSPPPEAIDICVPVLDSSRSRILVHCHAS  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWVCLEQKFCYWS++ATIVDDVLYTV+ NYS    SLEAYNLLDKKHL
Sbjct  181  DTLYAYYYDRKSWVCLEQKFCYWSDTATIVDDVLYTVLDNYSC---SLEAYNLLDKKHLP  237

Query  241  VKW------------------SSGFSGGSNNEWHS-------------GGIHAAAEHQSA  269
            VKW                  S     G  N  H+             GGIH AAEHQSA
Sbjct  238  VKWLSEVPVTPPQGCTLYRLGSGKLILGWVNHLHTFEYIRFNIWCNEQGGIHVAAEHQSA  297

Query  270  ITVPYPKDVHDIMLF  284
            ITVPYPKD+ +++ F
Sbjct  298  ITVPYPKDICNLLFF  312


> Gar06G04460
Length=242

 Score = 407 bits (1046),  Expect = 3e-145, Method: Compositional matrix adjust.
 Identities = 195/242 (81%), Positives = 216/242 (89%), Gaps = 0/242 (0%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDEL+R+LV+KWP  DG++T G  F+D+E N VFI+AMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELVRALVQKWPDIDGKITHGLHFIDLESNKVFIKAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAV+CGNHVYVIGG+R+NDA   GYD VNDVFQLDLKDLERGWRKTTSMLFPR FPLV
Sbjct  61   CSTAVSCGNHVYVIGGMRENDANFRGYDGVNDVFQLDLKDLERGWRKTTSMLFPRVFPLV  120

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAE KIYVF+ + SESFGEVYDISGD WEPLS PPE I +C PVLDSSRSRILVHC+A+
Sbjct  121  LAAEGKIYVFQYLVSESFGEVYDISGDFWEPLSPPPEAIDICVPVLDSSRSRILVHCHAS  180

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYA ++DRKSWVCLEQKFCYWS++ATIVDDVLYTVIYNYSGKFRS EAYNLLD+KHL
Sbjct  181  DTLYANHYDRKSWVCLEQKFCYWSDTATIVDDVLYTVIYNYSGKFRSSEAYNLLDRKHLP  240

Query  241  VK  242
            VK
Sbjct  241  VK  242


> Gar06G04480
Length=294

 Score = 409 bits (1051),  Expect = 3e-145, Method: Compositional matrix adjust.
 Identities = 219/316 (69%), Positives = 234/316 (74%), Gaps = 54/316 (17%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MSGDTSDSSSYF GREPDELIR+LVEK PGTDG+  LG  F+D++ N VFIRAMPPDAP
Sbjct  1    MSGDTSDSSSYFDGREPDELIRALVEKSPGTDGKTALGLHFIDLKSNKVFIRAMPPDAPS  60

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
             STAVACGNHVYVIGGIR+NDA++ GYD+VNDVFQLDLKDLER
Sbjct  61   GSTAVACGNHVYVIGGIRENDASLRGYDEVNDVFQLDLKDLER-----------------  103

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
                 KIYVFE +GSESFGEVYDIS DIWEPLS+PPE I LC PVLDSSRSRILVHCNAN
Sbjct  104  -----KIYVFEGMGSESFGEVYDISWDIWEPLSSPPEAIDLCVPVLDSSRSRILVHCNAN  158

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWVCLEQKFCYWS+SATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHL
Sbjct  159  DTLYAYYYDRKSWVCLEQKFCYWSDSATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLP  218

Query  241  VKWSSGFS-------------------GGSN-----------NEW--HSGGIHAAAEHQS  268
            VKWSS FS                   G  N           N W    GGIHAAAEHQ
Sbjct  219  VKWSSEFSVNPPPNGTLYRLGNGKLILGWVNLFHERFEYIRFNIWCNEQGGIHAAAEHQF  278

Query  269  AITVPYPKDVHDIMLF  284
            AITV YP+D+  I LF
Sbjct  279  AITVSYPEDISSIWLF  294


> Gar06G04420
Length=258

 Score = 380 bits (977),  Expect = 2e-134, Method: Compositional matrix adjust.
 Identities = 196/248 (79%), Positives = 203/248 (82%), Gaps = 31/248 (13%)

Query  1    MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL  60
            MS D SDSS+YFG     ELIR LVEKWPGTDG+MTLG  FLD+E ++VFIRAMPPDA L
Sbjct  1    MSVDASDSSTYFG-----ELIRVLVEKWPGTDGKMTLGLHFLDLERDEVFIRAMPPDASL  55

Query  61   SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV  120
            SSTAVACGNHVYV+GGIRKNDATIFGYDDVNDVFQLDLKDLE GWRKTTSMLFPRSFPLV
Sbjct  56   SSTAVACGNHVYVLGGIRKNDATIFGYDDVNDVFQLDLKDLEHGWRKTTSMLFPRSFPLV  115

Query  121  LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN  180
            LAAE KIYVFER                          I LCDPVLDS RSRILVHCNAN
Sbjct  116  LAAEGKIYVFER--------------------------ISLCDPVLDSFRSRILVHCNAN  149

Query  181  DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT  240
            DTLYAYY+DRKSWVCL QKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHL
Sbjct  150  DTLYAYYYDRKSWVCLGQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLP  209

Query  241  VKWSSGFS  248
            VKWSS FS
Sbjct  210  VKWSSEFS  217


> Gar06G05370
Length=167

 Score = 233 bits (593),  Expect = 2e-77, Method: Compositional matrix adjust.
 Identities = 112/138 (81%), Positives = 124/138 (90%), Gaps = 0/138 (0%)

Query  111  MLFPRSFPLVLAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSR  170
            MLFPR+FPLV AAE KIYVFE++  ESFGEVYDISGDIWEPLS P E I L +PVLDSSR
Sbjct  1    MLFPRAFPLVFAAEGKIYVFEQLVFESFGEVYDISGDIWEPLSPPLEAIALSNPVLDSSR  60

Query  171  SRILVHCNANDTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEA  230
            SRILVHC+ ND+LYAYY+DRKSWVCL+QKFCYWS+ A+IVDDVLYTV+YNYSG+F SLEA
Sbjct  61   SRILVHCHVNDSLYAYYYDRKSWVCLDQKFCYWSDLASIVDDVLYTVMYNYSGEFCSLEA  120

Query  231  YNLLDKKHLTVKWSSGFS  248
            YNLLDKKHL VKWSS FS
Sbjct  121  YNLLDKKHLPVKWSSEFS  138



Lambda      K        H        a         alpha
   0.319    0.137    0.434    0.792     4.96

Gapped
Lambda      K        H        a         alpha    sigma
   0.267   0.0410    0.140     1.90     42.6     43.6

Effective search space used: 2266345112


  Database: User specified sequence set (Input: A2.proteins.fa).
    Posted date:  Unknown
  Number of letters in database: 16,644,893
  Number of sequences in database:  43,278



Matrix: BLOSUM62
Gap Penalties: Existence: 11, Extension: 1
Neighboring words threshold: 11
Window for multiple hits: 40

    #+end_example



#+Caption: protein blast for B1.B_06G223600-1 vs B1, D5, and A2 (evalue <= 10^-50)
| Genome | Gene                   | Identity | Len | Match | Gap | Q.Start | Q.End | S.Start | S.End |   E-value | Bits |
|--------+------------------------+----------+-----+-------+-----+---------+-------+---------+-------+-----------+------|
| B1     | B1.B_06G223600-1       |  100.000 | 284 |     0 |   0 |       1 |   284 |       1 |   284 |       0.0 |  589 |
| B1     | B1.B_06G222300-1       |   74.367 | 316 |    49 |   3 |       1 |   284 |       1 |   316 | 5.10e-163 |  455 |
| B1     | B1.B_06G214200-1       |   70.476 | 315 |    59 |   3 |       1 |   284 |       1 |   312 | 2.16e-153 |  431 |
| B1     | B1.B_06G224100-1       |   70.886 | 316 |    60 |   4 |       1 |   284 |       1 |   316 | 4.31e-153 |  430 |
| B1     | B1.B_06G214100-1       |   69.524 | 315 |    62 |   3 |       1 |   284 |       1 |   312 | 1.64e-149 |  421 |
| B1     | B1.B_06G224000-1       |   68.671 | 316 |    57 |   4 |       1 |   284 |       1 |   306 | 2.32e-146 |  412 |
| D5     | D5.v1.pred_00030947-RA |   74.204 | 314 |    51 |   2 |       1 |   284 |       1 |   314 | 1.87e-165 |  461 |
| D5     | D5.v1.pred_00030954-RA |   69.427 | 314 |    49 |   3 |       1 |   284 |       1 |   297 | 5.81e-148 |  416 |
| D5     | D5.v1.pred_00030938-RA |   56.688 | 314 |    77 |   6 |       3 |   281 |      91 |   380 | 7.24e-104 |  311 |
| D5     | D5.v1.pred_00030950-RA |   67.281 | 217 |    31 |   2 |       1 |   217 |       1 |   177 |  1.75e-95 |  280 |
| D5     | D5.v1.pred_00030965-RA |   54.122 | 279 |    89 |   5 |      19 |   284 |      14 |   266 |  5.69e-84 |  253 |
| A2     | Gar11G43950            |   73.651 | 315 |    51 |   3 |       1 |   283 |       1 |   315 | 8.38e-160 |  447 |
| A2     | Gar06G05690            |   72.785 | 316 |    54 |   2 |       1 |   284 |       1 |   316 | 2.98e-157 |  441 |
| A2     | Gar06G05390            |   70.794 | 315 |    58 |   3 |       1 |   284 |       1 |   312 | 1.07e-154 |  434 |
| A2     | Gar06G04460            |   80.579 | 242 |    47 |   0 |       1 |   242 |       1 |   242 | 3.25e-145 |  407 |
| A2     | Gar06G04480            |   69.304 | 316 |    43 |   4 |       1 |   284 |       1 |   294 | 3.37e-145 |  409 |
| A2     | Gar06G04420            |   79.032 | 248 |    21 |   2 |       1 |   248 |       1 |   217 | 1.66e-134 |  380 |
| A2     | Gar06G05370            |   81.159 | 138 |    26 |   0 |     111 |   248 |       1 |   138 |  1.63e-77 |  233 |


- Muscle alignment and tree of all proteins similar to B1.B_06G223600-1
  + input fasta
    #+begin_src fasta :tangle proteins.of.interest.fa
>sp|Q8GXF6|FBK85_ARATH F-box/kelch-repeat protein At4g19870 OS=Arabidopsis thaliana OX=3702 GN=At4g19870 PE=2 SV=1
MKRQAKPPEKKTKRTTNASSPTPSSSSPSLSSLPDEIVENCLARISRSYYPTLSIVSKSF
RSIISSTELYVARSHLRNTEECVYVCLSDKSFEFPKWFTLWVNPNQANSMVEKKRKKKKT
IGKLLVPIPSSNLSPVSKSAIAVGSEIYVIGGKVDGALSSAVRILDCRSNTWRDAPSMTV
ARKRPFICLYDGKIYVIGGYNKLSESEPWAEVFDIKTQTWECLSDPGTEIRNCTIYRIAE
IEGKIHFGYTQKTYAYDPKQGEWECCEGEVAFPRSQCVMESVLYTFANNYTWEDDYGCKW
WSTDGYGEKVKGLESLLEIHKRNGGSSDNTTKLVACGGKLLLLWEGYMKHNPNNRKKIWC
AVIALEKRDGGKVWGIVEWVDVVHIVPTSCKLLHCLVVSV

>B1.B_06G223600-1
MSGDTSDSSSYFGGREPDELIRSLVEKWPGTDGEMTLGCTFLDVECNDVFIRAMPPDAPL
SSTAVACGNHVYVIGGIRKNDATIFGYDDVNDVFQLDLKDLERGWRKTTSMLFPRSFPLV
LAAEEKIYVFERIGSESFGEVYDISGDIWEPLSAPPEGIGLCDPVLDSSRSRILVHCNAN
DTLYAYYFDRKSWVCLEQKFCYWSESATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLT
VKWSSGFSGGSNNEWHSGGIHAAAEHQSAITVPYPKDVHDIMLF
>B1.B_06G222300-1
MSGDTSDSSSYFDGREPDELIRALVENSPGTDGKTALGLHFIDLKSNKVFIRAMPPDAPS
GSTAVACGNHIYVIGGIRENDASLRGYDEVNDVFQLDLKDLERGWRKTTSMLFPRAFPLV
LAAEGKIYVFEGMGSESFGEVYDISGDIWEPLLPPPEAIDLSVPVPDSSRSRILVHCNAN
DTLYAYYYDRKSWVCLEQKFCYWSDSATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLP
VKWSSEFSVNPPPNGTLYRLGNGKLILGWVNLFHERFEYIRFNIWCNEQGGIHAAAEHQF
AITVSYPEDISSIWLF
>B1.B_06G214200-1
MSGDTSDSSSYFDGREPDELIRALVDKWPDTDGKITLGLRFIDLERNKVFIKAMPPDAPS
CSTAVSCGNHVYVIGGMRENDANFRGYDRVNDVFQLDLKDLERGWRKTTSMLFPRVFPLV
LAAEGRIYVFEYLGSESFGEVYDISGDFWEPLSPPPEAIDICVPILDSSRSRILVHCHAS
DTLYAYYYDRKSWVYLEQKFCYWSDTATIVDDVLYTVLDNYSCSLEAYNLLDKKHLPVKW
SSEVPVTPPQGCTLYRLGSGKLILGWVNHLRTFEYIRFNIWCNEQGGIHVAAEHQSAITV
PYPKDISNIWFF
>B1.B_06G224100-1
MSGDTSDSSSYFDGREPDELIRALVRKRRGTDGKIAHGLHFIDLESNKVFIRAMPPDAPS
CSTAVACGNHVYIIGGIHENDANFRGYDGVNDVFQLDLKDLERGWRKTTSMLFPQAFPLV
LAVEGKIYFFEYRGSESFGEVYDISGDIWEPLSSPPEDIDLCVPVLDSSRSRILVHCDAS
DTLYAYYYDRNSWVCLEQKFCYWSDSATIVDDVLYTLIYNYSCKFRSLEAYNLLDKKHLP
SNGHQNFQWLHHWVALCIVWAMASLFWVGVNQFRKRFEYIRFNIWCNEQGGIHAAAEHKS
AISVPYPEDISNIWLF
>B1.B_06G214100-1
MSGDTSDSSSYFDGREPDELIRALVQKWPDTDGKMPLGLRFIDLERNKVFIKAMPPDAPS
GSTAVSCGNHVYVIGGIRENDANFRGYDGLNDVFQLDLNDLERGWRKTTSMLFPRAFPLV
LAAEGKIYVFEYWGSESFGEVYDISGDFWELLSPPPEAIDICVPVLDSSRSRILVHCHAS
DTLYAYYYDRKSWVCLEQKFCYWSDTATIVDDVLYTVLDNYSCSLEAYNLLDKKHLPVKW
LSEVPVTPPQGCTLYRLGSGKLILGWVNHLRTFEYLRFNIWCNEQGGIHVAAERQSAIAV
PYPKDICNLWFF
>B1.B_06G224000-1
MSGDTSDSSSYFDGQDPDELIRALVRKRPGTDGKIVHGLHFFDLESNKVFIKAMPPDAPS
CSTVVACGNHVYVIGGIRENDANFRGYDADLERGWRKTTSMLFPRAFPLVLAADGKVYVF
EHWGSESFGEVYDISGDIWEPLSPPPEDIDLSVPVLDSSRSRILVHCDASDTLYAYYYDR
KSWVCLEQKFCYWSDSATIVDDVLYTLIYNYSCKFRSLEAYNLLDKKHLPVKWSSEFSVA
PPLGSTPYRLGNGKLILGWVNQFRKRFEYIRFNIWCNEQGGIHAVAEHKSAISVPYPEDI
SNIWLF
>D5.v1.pred_00030947-RA
MSGDTSDSSSYFDGREPDELIRALVQKWPGTDGKITVGLHFLDLESNEVFIKAMPPDAPF
CSTAVACGDHVYVIGGIRQNDATLFDNDGVNDVFQLDLKDLERGWRKTTSMLFPRVLPRV
VAAEGKIFVFEYMGSESFGEVYDISGDIWEPLSPPPEDIDLCVPVLDSSRSRILVHCNAN
DTLYAYYYDRKSWVCLEQKFCYWSDAATIVDDVLYTAIYNYYDKFRSLEAYNLLDKKHLP
VKWSSEFSVDPHGTLYRLGNGKCIFVWFNRLDKSFEYIRFHIWCNEQGGIHAAAEHQSAI
SVPYPKDISDIKLF
>D5.v1.pred_00030954-RA
MRGDTSDSSSYFDGREPDELIRALVQNWPGTDGKMTVGLHFLDLESNKVFIKAMPPDAPF
CSTAVACGDHVYVIGGIRQNDATLFDYDGVNDVFQLDLKDLERGWRKTTSMLFPRFLPRV
VAAEGKIFVFEYMGSESFGEVYDISGDIWEPLSPPPEDIDLCVPVLDSSRSRILVHCNAN
DTLYAYYYDPTIVDDVLYTAIYNYSDKFRSLEAYNLLDKKHLPVKWSSEFSVDPHGTLYR
LGNGKCIFVWFNHLDKSFEYIRFNIWCNEQGGIHAAAEHQSAISVPYPKDISDIKLF
>D5.v1.pred_00030938-RA
MHRQMVVHRHLNIDNSPSLRLRLPFPSCYRYGLYKRNHSGLLNELKELWWIDCSMEREPI
NALLCFLKFCPSLERFYVTLDPNAITCPALGDTSDSSSYFDGREPDELIRALVQKRPGTD
GKRTLGLHFIDLESNKVFIKAMPPDAPSSSTAVSCGNHVYVIGGIRKMMLLLLVMMMTLN
VGGEKLLLCCSLGLFPMFLLLKERFTSSNVYDISGDIWEPLSPPPEAIDIRVVCVPVLDS
SRSRILVHCDASDTLYAYYYDRKSWVCLEQKFCYWFDSAAIVDDVLYTFIYKCSLEAYNL
LDKKHLPVKWSSEFPVAPPLGSALYRLGNGKLILGWVNHLHRGFECIRFNIWCNEQGGIH
AAAEHQSAITVPYPEDISSICAPFTSLPHLNYSHPCKLTDFSSANTACRSILFALVVAKK
LIVSLELPASDLHCVKMSTLFVVLNGYEAPSTFRELELKLAFHLRIANAAAAAESDATLR
EEHGKVPHKSKGASKVKLFMINKCSACK
>D5.v1.pred_00030950-RA
MSGDTSDSSSYFDGREPDQLIRALHLSFIKAMPPDAPSCSTAVSCGNHVYVIGGIRKNDA
TLLLLIKTTSMLFPRSLPLVAAAEGKIFVFENMGSESFGEVYNISGDIWEPLSPPPEDID
LCVPVLDSSRSRILVHCSANDTLYAYYYDRKSWVCLEQKFCYWSDAATIVDDVLYTACRA
QFYDLRPNNKAQLPRPKYNTQSPPKPNPLA
>D5.v1.pred_00030965-RA
MSSDTSDSSSYIGELIRTHVQKWPGTDGKMTLGLHFLDLESDEVFIRAMPPDAPLSPLPL
PVATTFTFLGEYAKMMLLLVIGRTLNVGGEKLLLCCSLGLFPLVLAAEGKIYVFELLGFE
SFGDVYDISGDIWEPLSPLRKLSLYVVLFWFSYAYYYDRKSWVCLEPKFCYWSDPASIVD
DVLYTVMYNYSGEFCSLEAYNLLDKKHLPVKWSSEFSVDPPRSGTLFGLGNGKCILRWGG
IDAAAEHQSAITVPYRSKDICDIFLF
>Gar11G43950
MSGDTSDSSSYFDGREPDELIRALVEKSPGTDGKITHGLHYIDLESNKVFIRAMPPDAPS
CSTAVACGNHVYVIGGIRENDANFRGYDGVNDVFQLDLKDLERGWRKTTSMLFPRAFPLV
LAAEGKIYVFEYWGSESFGEVYDISGDIWEPLSPPPEDIDICVPVLDCSRSRILVHCHAS
DALYAYYYDRKSWVCLEQKFCYWSDSATIVDDVLYTLIFNYSYKFRSLEAYNLLDKKHLP
VKWSSEFSVAPPPGSTLYRLGNGKLILGWVNHFDKRFEYIRFNIWCNEQGGIHAAAEHQS
PITVPYPEDMENICLELPV
>Gar06G05690
MSGDTSDSFSYFDGREPDELIRALFGKYPGTDGEITHGLHFIDLESNKVFIRAMPPDARS
CSTAVACGNHVNVIGGIRENDANFRGYDGVNDVFQLDLKDLERGWRKTTSMLFPRAFPLV
LAAEGKIYVFEYWGSESFGEVYDISRDIWEPLSPPPEDIDLCVPVLDSSRSRILVHCHAS
DTLYAYYYDRKSWVCLEQKFCYWSDSATIVDDVLYTLIYNYSCKFRSLEAYNLFDKKHLP
VKLSSEFSVAPPLGSTLYHLGNGKLISGWVNHFHKRFEYIRFNIWCNEQGGIHAAVEHQS
AITVPYPEDISNIWLF
>Gar06G05390
MSGDTSDSSSYFDGREPDELIRALVQKWPDTDGKMTLGLRFIDLERNKVFIKAMPPDAPS
GSTAVSCGNHVYVIGGMRENDANFRGYDGVNDVFQLDLKDLERGWRKTISMLFPRAFPLV
LAAEGKIYVFEYWGSESFGEVYDISGDFWEPLSPPPEAIDICVPVLDSSRSRILVHCHAS
DTLYAYYYDRKSWVCLEQKFCYWSDTATIVDDVLYTVLDNYSCSLEAYNLLDKKHLPVKW
LSEVPVTPPQGCTLYRLGSGKLILGWVNHLHTFEYIRFNIWCNEQGGIHVAAEHQSAITV
PYPKDICNLLFF
>Gar06G04460
MSGDTSDSSSYFDGREPDELVRALVQKWPDIDGKITHGLHFIDLESNKVFIKAMPPDAPS
CSTAVSCGNHVYVIGGMRENDANFRGYDGVNDVFQLDLKDLERGWRKTTSMLFPRVFPLV
LAAEGKIYVFQYLVSESFGEVYDISGDFWEPLSPPPEAIDICVPVLDSSRSRILVHCHAS
DTLYANHYDRKSWVCLEQKFCYWSDTATIVDDVLYTVIYNYSGKFRSSEAYNLLDRKHLP
VK
>Gar06G04480
MSGDTSDSSSYFDGREPDELIRALVEKSPGTDGKTALGLHFIDLKSNKVFIRAMPPDAPS
GSTAVACGNHVYVIGGIRENDASLRGYDEVNDVFQLDLKDLERKIYVFEGMGSESFGEVY
DISWDIWEPLSSPPEAIDLCVPVLDSSRSRILVHCNANDTLYAYYYDRKSWVCLEQKFCY
WSDSATIVDDVLYTVIYNYSGKFRSLEAYNLLDKKHLPVKWSSEFSVNPPPNGTLYRLGN
GKLILGWVNLFHERFEYIRFNIWCNEQGGIHAAAEHQFAITVSYPEDISSIWLF
>Gar06G04420
MSVDASDSSTYFGELIRVLVEKWPGTDGKMTLGLHFLDLERDEVFIRAMPPDASLSSTAV
ACGNHVYVLGGIRKNDATIFGYDDVNDVFQLDLKDLEHGWRKTTSMLFPRSFPLVLAAEG
KIYVFERISLCDPVLDSFRSRILVHCNANDTLYAYYYDRKSWVCLGQKFCYWSESATIVD
DVLYTVIYNYSGKFRSLEAYNLLDKKHLPVKWSSEFSVDLTMKGILYRLGNGKCILGWVN
HKHMSFEYLRLTIWCNEQ
>Gar06G05370
MLFPRAFPLVFAAEGKIYVFEQLVFESFGEVYDISGDIWEPLSPPLEAIALSNPVLDSSR
SRILVHCHVNDSLYAYYYDRKSWVCLDQKFCYWSDLASIVDDVLYTVMYNYSGEFCSLEA
YNLLDKKHLPVKWSSEFSVDPPRSGTLFRLGNGEWILGWVNHIDMSF

  #+end_src
  + MSA alignment
    #+begin_example
sp|Q8GXF6|FBK85_ARATH       ------------------------------------------------------------
D5.v1.pred_00030938-RA      MHRQMVVHRHLNIDNSPSLRLRLPFPSCYRYGLYKRNHSGLLNELKELWWIDCSMEREPI
D5.v1.pred_00030965-RA      ------------------------------------------------------------
B1.B_06G223600-1            ------------------------------------------------------------
D5.v1.pred_00030950-RA      ------------------------------------------------------------
Gar06G05370                 ------------------------------------------------------------
Gar06G04420                 ------------------------------------------------------------
D5.v1.pred_00030947-RA      ------------------------------------------------------------
D5.v1.pred_00030954-RA      ------------------------------------------------------------
B1.B_06G224100-1            ------------------------------------------------------------
B1.B_06G222300-1            ------------------------------------------------------------
Gar06G04480                 ------------------------------------------------------------
Gar06G04460                 ------------------------------------------------------------
B1.B_06G214200-1            ------------------------------------------------------------
B1.B_06G214100-1            ------------------------------------------------------------
Gar06G05390                 ------------------------------------------------------------
B1.B_06G224000-1            ------------------------------------------------------------
Gar06G05690                 ------------------------------------------------------------
Gar11G43950                 ------------------------------------------------------------


sp|Q8GXF6|FBK85_ARATH       -----------MKRQAKPPEKKTKRTTNASSPTPSSSSPSLSSLPDEIVENCLARISRSY
D5.v1.pred_00030938-RA      NALLCFLKFCPSLERFYVTLDPNAITCPALGDTSDSSSYFDGREPDELIRALVQK-----
D5.v1.pred_00030965-RA      ----------------------------MSSDTSDSSSYI-----GELIRTHVQK-----
B1.B_06G223600-1            ----------------------------MSGDTSDSSSYFGGREPDELIRSLVEK-----
D5.v1.pred_00030950-RA      ----------------------------MSGDTSDSSSYFDGREPDQLIRALHLS-----
Gar06G05370                 ------------------------------------------------------------
Gar06G04420                 ----------------------------MSVDASDSSTYF-----GELIRVLVEK-----
D5.v1.pred_00030947-RA      ----------------------------MSGDTSDSSSYFDGREPDELIRALVQK-----
D5.v1.pred_00030954-RA      ----------------------------MRGDTSDSSSYFDGREPDELIRALVQN-----
B1.B_06G224100-1            ----------------------------MSGDTSDSSSYFDGREPDELIRALVRK-----
B1.B_06G222300-1            ----------------------------MSGDTSDSSSYFDGREPDELIRALVEN-----
Gar06G04480                 ----------------------------MSGDTSDSSSYFDGREPDELIRALVEK-----
Gar06G04460                 ----------------------------MSGDTSDSSSYFDGREPDELVRALVQK-----
B1.B_06G214200-1            ----------------------------MSGDTSDSSSYFDGREPDELIRALVDK-----
B1.B_06G214100-1            ----------------------------MSGDTSDSSSYFDGREPDELIRALVQK-----
Gar06G05390                 ----------------------------MSGDTSDSSSYFDGREPDELIRALVQK-----
B1.B_06G224000-1            ----------------------------MSGDTSDSSSYFDGQDPDELIRALVRK-----
Gar06G05690                 ----------------------------MSGDTSDSFSYFDGREPDELIRALFGK-----
Gar11G43950                 ----------------------------MSGDTSDSSSYFDGREPDELIRALVEK-----


sp|Q8GXF6|FBK85_ARATH       YPTLSIVSKSFRSIISSTELYVARSHLRNTEECVYVCLSDKSFEFPKWFTLWVNPNQANS
D5.v1.pred_00030938-RA      --------------------------RPGTDGKRTLGLHFIDLESNKVFIKAMPPD----
D5.v1.pred_00030965-RA      --------------------------WPGTDGKMTLGLHFLDLESDEVFIRAMPPD----
B1.B_06G223600-1            --------------------------WPGTDGEMTLGCTFLDVECNDVFIRAMPPD----
D5.v1.pred_00030950-RA      ------------------------------------------------FIKAMPPD----
Gar06G05370                 ------------------------------------------------------------
Gar06G04420                 --------------------------WPGTDGKMTLGLHFLDLERDEVFIRAMPPD----
D5.v1.pred_00030947-RA      --------------------------WPGTDGKITVGLHFLDLESNEVFIKAMPPD----
D5.v1.pred_00030954-RA      --------------------------WPGTDGKMTVGLHFLDLESNKVFIKAMPPD----
B1.B_06G224100-1            --------------------------RRGTDGKIAHGLHFIDLESNKVFIRAMPPD----
B1.B_06G222300-1            --------------------------SPGTDGKTALGLHFIDLKSNKVFIRAMPPD----
Gar06G04480                 --------------------------SPGTDGKTALGLHFIDLKSNKVFIRAMPPD----
Gar06G04460                 --------------------------WPDIDGKITHGLHFIDLESNKVFIKAMPPD----
B1.B_06G214200-1            --------------------------WPDTDGKITLGLRFIDLERNKVFIKAMPPD----
B1.B_06G214100-1            --------------------------WPDTDGKMPLGLRFIDLERNKVFIKAMPPD----
Gar06G05390                 --------------------------WPDTDGKMTLGLRFIDLERNKVFIKAMPPD----
B1.B_06G224000-1            --------------------------RPGTDGKIVHGLHFFDLESNKVFIKAMPPD----
Gar06G05690                 --------------------------YPGTDGEITHGLHFIDLESNKVFIRAMPPD----
Gar11G43950                 --------------------------SPGTDGKITHGLHYIDLESNKVFIRAMPPD----


sp|Q8GXF6|FBK85_ARATH       MVEKKRKKKKTIGKLLVPIPSSNLSPVSKSAIAVGSEIYVIGG--KVDGALSSAVRILDC
D5.v1.pred_00030938-RA      ------------------------APSSSTAVSCGNHVYVIGGIRK----------MMLL
D5.v1.pred_00030965-RA      ------------------------APLSPLPLPVATTFTFLG----------EYAKMMLL
B1.B_06G223600-1            ------------------------APLSSTAVACGNHVYVIGGIRKNDATIFGYDDVNDV
D5.v1.pred_00030950-RA      ------------------------APSCSTAVSCGNHVYVIGGIRKNDATLLLLI-----
Gar06G05370                 ------------------------------------------------------------
Gar06G04420                 ------------------------ASLSSTAVACGNHVYVLGGIRKNDATIFGYDDVNDV
D5.v1.pred_00030947-RA      ------------------------APFCSTAVACGDHVYVIGGIRQNDATLFDNDGVNDV
D5.v1.pred_00030954-RA      ------------------------APFCSTAVACGDHVYVIGGIRQNDATLFDYDGVNDV
B1.B_06G224100-1            ------------------------APSCSTAVACGNHVYIIGGIHENDANFRGYDGVNDV
B1.B_06G222300-1            ------------------------APSGSTAVACGNHIYVIGGIRENDASLRGYDEVNDV
Gar06G04480                 ------------------------APSGSTAVACGNHVYVIGGIRENDASLRGYDEVNDV
Gar06G04460                 ------------------------APSCSTAVSCGNHVYVIGGMRENDANFRGYDGVNDV
B1.B_06G214200-1            ------------------------APSCSTAVSCGNHVYVIGGMRENDANFRGYDRVNDV
B1.B_06G214100-1            ------------------------APSGSTAVSCGNHVYVIGGIRENDANFRGYDGLNDV
Gar06G05390                 ------------------------APSGSTAVSCGNHVYVIGGMRENDANFRGYDGVNDV
B1.B_06G224000-1            ------------------------APSCSTVVACGNHVYVIGGIRENDANFRGYDA----
Gar06G05690                 ------------------------ARSCSTAVACGNHVNVIGGIRENDANFRGYDGVNDV
Gar11G43950                 ------------------------APSCSTAVACGNHVYVIGGIRENDANFRGYDGVNDV


sp|Q8GXF6|FBK85_ARATH       -------RSNTWRDAPSMTVARKRPFICLYDGKIYVIGGYNKLSESEPWAEVFDIKTQTW
D5.v1.pred_00030938-RA      LLVMMMTLNVGGEKLLLCCSLGLFPMFLLLKERF-------------TSSNVYDISGDIW
D5.v1.pred_00030965-RA      LVIG-RTLNVGGEKLLLCCSLGLFPLVLAAEGKIYVFELLG----FESFGDVYDISGDIW
B1.B_06G223600-1            FQLDLKDLERGWRKTTSMLFPRSFPLVLAAEEKIYVFERIG----SESFGEVYDISGDIW
D5.v1.pred_00030950-RA      -------------KTTSMLFPRSLPLVAAAEGKIFVFENMG----SESFGEVYNISGDIW
Gar06G05370                 -----------------MLFPRAFPLVFAAEGKIYVFEQLV----FESFGEVYDISGDIW
Gar06G04420                 FQLDLKDLEHGWRKTTSMLFPRSFPLVLAAEGKIYVF-----------------------
D5.v1.pred_00030947-RA      FQLDLKDLERGWRKTTSMLFPRVLPRVVAAEGKIFVFEYMG----SESFGEVYDISGDIW
D5.v1.pred_00030954-RA      FQLDLKDLERGWRKTTSMLFPRFLPRVVAAEGKIFVFEYMG----SESFGEVYDISGDIW
B1.B_06G224100-1            FQLDLKDLERGWRKTTSMLFPQAFPLVLAVEGKIYFFEYRG----SESFGEVYDISGDIW
B1.B_06G222300-1            FQLDLKDLERGWRKTTSMLFPRAFPLVLAAEGKIYVFEGMG----SESFGEVYDISGDIW
Gar06G04480                 FQLDLKDLER----------------------KIYVFEGMG----SESFGEVYDISWDIW
Gar06G04460                 FQLDLKDLERGWRKTTSMLFPRVFPLVLAAEGKIYVFQYLV----SESFGEVYDISGDFW
B1.B_06G214200-1            FQLDLKDLERGWRKTTSMLFPRVFPLVLAAEGRIYVFEYLG----SESFGEVYDISGDFW
B1.B_06G214100-1            FQLDLNDLERGWRKTTSMLFPRAFPLVLAAEGKIYVFEYWG----SESFGEVYDISGDFW
Gar06G05390                 FQLDLKDLERGWRKTISMLFPRAFPLVLAAEGKIYVFEYWG----SESFGEVYDISGDFW
B1.B_06G224000-1            ------DLERGWRKTTSMLFPRAFPLVLAADGKVYVFEHWG----SESFGEVYDISGDIW
Gar06G05690                 FQLDLKDLERGWRKTTSMLFPRAFPLVLAAEGKIYVFEYWG----SESFGEVYDISRDIW
Gar11G43950                 FQLDLKDLERGWRKTTSMLFPRAFPLVLAAEGKIYVFEYWG----SESFGEVYDISGDIW
                                                            ..

sp|Q8GXF6|FBK85_ARATH       ECLSDPGTEIRN---CT-IYRIAEIEGKIHFGYTQKTYAYDPKQGEWECCEGEVAF-PRS
D5.v1.pred_00030938-RA      EPLSPPPEAIDIRVVCVPVLDSSRSRILVHCDASDTLYAYYYDRKSWVCLEQKFCYWFDS
D5.v1.pred_00030965-RA      EPLS-PLRKLSL---YVVLFWFS--------------YAYYYDRKSWVCLEPKFCYWSDP
B1.B_06G223600-1            EPLSAPPEGIGL---CDPVLDSSRSRILVHCNANDTLYAYYFDRKSWVCLEQKFCYWSES
D5.v1.pred_00030950-RA      EPLSPPPEDIDL---CVPVLDSSRSRILVHCSANDTLYAYYYDRKSWVCLEQKFCYWSDA
Gar06G05370                 EPLSPPLEAIAL---SNPVLDSSRSRILVHCHVNDSLYAYYYDRKSWVCLDQKFCYWSDL
Gar06G04420                 -------ERISL---CDPVLDSFRSRILVHCNANDTLYAYYYDRKSWVCLGQKFCYWSES
D5.v1.pred_00030947-RA      EPLSPPPEDIDL---CVPVLDSSRSRILVHCNANDTLYAYYYDRKSWVCLEQKFCYWSDA
D5.v1.pred_00030954-RA      EPLSPPPEDIDL---CVPVLDSSRSRILVHCNANDTLYAYYYD-----------------
B1.B_06G224100-1            EPLSSPPEDIDL---CVPVLDSSRSRILVHCDASDTLYAYYYDRNSWVCLEQKFCYWSDS
B1.B_06G222300-1            EPLLPPPEAIDL---SVPVPDSSRSRILVHCNANDTLYAYYYDRKSWVCLEQKFCYWSDS
Gar06G04480                 EPLSSPPEAIDL---CVPVLDSSRSRILVHCNANDTLYAYYYDRKSWVCLEQKFCYWSDS
Gar06G04460                 EPLSPPPEAIDI---CVPVLDSSRSRILVHCHASDTLYANHYDRKSWVCLEQKFCYWSDT
B1.B_06G214200-1            EPLSPPPEAIDI---CVPILDSSRSRILVHCHASDTLYAYYYDRKSWVYLEQKFCYWSDT
B1.B_06G214100-1            ELLSPPPEAIDI---CVPVLDSSRSRILVHCHASDTLYAYYYDRKSWVCLEQKFCYWSDT
Gar06G05390                 EPLSPPPEAIDI---CVPVLDSSRSRILVHCHASDTLYAYYYDRKSWVCLEQKFCYWSDT
B1.B_06G224000-1            EPLSPPPEDIDL---SVPVLDSSRSRILVHCDASDTLYAYYYDRKSWVCLEQKFCYWSDS
Gar06G05690                 EPLSPPPEDIDL---CVPVLDSSRSRILVHCHASDTLYAYYYDRKSWVCLEQKFCYWSDS
Gar11G43950                 EPLSPPPEDIDI---CVPVLDCSRSRILVHCHASDALYAYYYDRKSWVCLEQKFCYWSDS
                                     :        :                  **   .

sp|Q8GXF6|FBK85_ARATH       QCVMESVLYTFANNYTWEDDYGCKWWSTDGYGEKVKGLESLLEIHKRNGGSSDNTTKLVA
D5.v1.pred_00030938-RA      AAIVDDVLYTFIYKCS-----------LEAYNLLDKKHLP-VKWSSEFPVAPPLGSALYR
D5.v1.pred_00030965-RA      ASIVDDVLYTVMYNYS------GEFCSLEAYNLLDKKHLP-VKWSSEFSVDPPRSGTLFG
B1.B_06G223600-1            ATIVDDVLYTVIYNYS------GKFRSLEAYNLLDKKHLT-VKWSSGFSGGSNNE-----
D5.v1.pred_00030950-RA      ATIVDDVLYTACR---------AQFYDLRPNN---KAQLPRPKYNTQSPPKPNPLA----
Gar06G05370                 ASIVDDVLYTVMYNYS------GEFCSLEAYNLLDKKHLP-VKWSSEFSVDPPRSGTLFR
Gar06G04420                 ATIVDDVLYTVIYNYS------GKFRSLEAYNLLDKKHLP-VKWSSEFSVDLTMKGILYR
D5.v1.pred_00030947-RA      ATIVDDVLYTAIYNYY------DKFRSLEAYNLLDKKHLP-VKWSSEFSVDPH--GTLYR
D5.v1.pred_00030954-RA      PTIVDDVLYTAIYNYS------DKFRSLEAYNLLDKKHLP-VKWSSEFSVDPH--GTLYR
B1.B_06G224100-1            ATIVDDVLYTLIYNYS------CKFRSLEAYNLLDKKHLP-SNGHQNFQWLHHWVALCIV
B1.B_06G222300-1            ATIVDDVLYTVIYNYS------GKFRSLEAYNLLDKKHLP-VKWSSEFSVNPPPNGTLYR
Gar06G04480                 ATIVDDVLYTVIYNYS------GKFRSLEAYNLLDKKHLP-VKWSSEFSVNPPPNGTLYR
Gar06G04460                 ATIVDDVLYTVIYNYS------GKFRSSEAYNLLDRKHLP-VK-----------------
B1.B_06G214200-1            ATIVDDVLYTVLDNYS------CSL---EAYNLLDKKHLP-VKWSSEVPVTPPQGCTLYR
B1.B_06G214100-1            ATIVDDVLYTVLDNYS------CSL---EAYNLLDKKHLP-VKWLSEVPVTPPQGCTLYR
Gar06G05390                 ATIVDDVLYTVLDNYS------CSL---EAYNLLDKKHLP-VKWLSEVPVTPPQGCTLYR
B1.B_06G224000-1            ATIVDDVLYTLIYNYS------CKFRSLEAYNLLDKKHLP-VKWSSEFSVAPPLGSTPYR
Gar06G05690                 ATIVDDVLYTLIYNYS------CKFRSLEAYNLFDKKHLP-VKLSSEFSVAPPLGSTLYH
Gar11G43950                 ATIVDDVLYTLIFNYS------YKFRSLEAYNLLDKKHLP-VKWSSEFSVAPPPGSTLYR
                              :::.****                     .   .   .  :

sp|Q8GXF6|FBK85_ARATH       CG-GKLLLLWEGYMKHNPNN-RKKIWCAVIALEKRDGGKVWGIVEWVDVVHIVPTSCKLL
D5.v1.pred_00030938-RA      LGNGKLILGWVNHLHRGFECIRFNIWC-------NEQGGIHAAAEHQSAITVPY-PEDIS
D5.v1.pred_00030965-RA      LGNGKCILRW---------------------------GGIDAAAEHQSAITVPYRSKDIC
B1.B_06G223600-1            -------------------------WH---------SGGIHAAAEHQSAITVPY-PKDVH
D5.v1.pred_00030950-RA      ------------------------------------------------------------
Gar06G05370                 LGNGEWILGWVNHIDMSF------------------------------------------
Gar06G04420                 LGNGKCILGWVNHKHMSFEYLRLTIWC-------NEQ-----------------------
D5.v1.pred_00030947-RA      LGNGKCIFVWFNRLDKSFEYIRFHIWC-------NEQGGIHAAAEHQSAISVPY-PKDIS
D5.v1.pred_00030954-RA      LGNGKCIFVWFNHLDKSFEYIRFNIWC-------NEQGGIHAAAEHQSAISVPY-PKDIS
B1.B_06G224100-1            WAMASLFWVGVNQFRKRFEYIRFNIWC-------NEQGGIHAAAEHKSAISVPY-PEDIS
B1.B_06G222300-1            LGNGKLILGWVNLFHERFEYIRFNIWC-------NEQGGIHAAAEHQFAITVSY-PEDIS
Gar06G04480                 LGNGKLILGWVNLFHERFEYIRFNIWC-------NEQGGIHAAAEHQFAITVSY-PEDIS
Gar06G04460                 ------------------------------------------------------------
B1.B_06G214200-1            LGSGKLILGWVNHLR-TFEYIRFNIWC-------NEQGGIHVAAEHQSAITVPY-PKDIS
B1.B_06G214100-1            LGSGKLILGWVNHLR-TFEYLRFNIWC-------NEQGGIHVAAERQSAIAVPY-PKDIC
Gar06G05390                 LGSGKLILGWVNHLH-TFEYIRFNIWC-------NEQGGIHVAAEHQSAITVPY-PKDIC
B1.B_06G224000-1            LGNGKLILGWVNQFRKRFEYIRFNIWC-------NEQGGIHAVAEHKSAISVPY-PEDIS
Gar06G05690                 LGNGKLISGWVNHFHKRFEYIRFNIWC-------NEQGGIHAAVEHQSAITVPY-PEDIS
Gar11G43950                 LGNGKLILGWVNHFDKRFEYIRFNIWC-------NEQGGIHAAAEHQSPITVPY-PEDME


sp|Q8GXF6|FBK85_ARATH       HC------------------------------LVVSV-----------------------
D5.v1.pred_00030938-RA      SICAPFTSLPHLNYSHPCKLTDFSSANTACRSILFALVVAKKLIVSLELPASDLHCVKMS
D5.v1.pred_00030965-RA      DI------------------------------FLF-------------------------
B1.B_06G223600-1            DI------------------------------MLF-------------------------
D5.v1.pred_00030950-RA      ------------------------------------------------------------
Gar06G05370                 ------------------------------------------------------------
Gar06G04420                 ------------------------------------------------------------
D5.v1.pred_00030947-RA      DI------------------------------KLF-------------------------
D5.v1.pred_00030954-RA      DI------------------------------KLF-------------------------
B1.B_06G224100-1            NI------------------------------WLF-------------------------
B1.B_06G222300-1            SI------------------------------WLF-------------------------
Gar06G04480                 SI------------------------------WLF-------------------------
Gar06G04460                 ------------------------------------------------------------
B1.B_06G214200-1            NI------------------------------WFF-------------------------
B1.B_06G214100-1            NL------------------------------WFF-------------------------
Gar06G05390                 NL------------------------------LFF-------------------------
B1.B_06G224000-1            NI------------------------------WLF-------------------------
Gar06G05690                 NI------------------------------WLF-------------------------
Gar11G43950                 NIC-----------------------------LELPV-----------------------


sp|Q8GXF6|FBK85_ARATH       ------------------------------------------------------------
D5.v1.pred_00030938-RA      TLFVVLNGYEAPSTFRELELKLAFHLRIANAAAAAESDATLREEHGKVPHKSKGASKVKL
D5.v1.pred_00030965-RA      ------------------------------------------------------------
B1.B_06G223600-1            ------------------------------------------------------------
D5.v1.pred_00030950-RA      ------------------------------------------------------------
Gar06G05370                 ------------------------------------------------------------
Gar06G04420                 ------------------------------------------------------------
D5.v1.pred_00030947-RA      ------------------------------------------------------------
D5.v1.pred_00030954-RA      ------------------------------------------------------------
B1.B_06G224100-1            ------------------------------------------------------------
B1.B_06G222300-1            ------------------------------------------------------------
Gar06G04480                 ------------------------------------------------------------
Gar06G04460                 ------------------------------------------------------------
B1.B_06G214200-1            ------------------------------------------------------------
B1.B_06G214100-1            ------------------------------------------------------------
Gar06G05390                 ------------------------------------------------------------
B1.B_06G224000-1            ------------------------------------------------------------
Gar06G05690                 ------------------------------------------------------------
Gar11G43950                 ------------------------------------------------------------


sp|Q8GXF6|FBK85_ARATH       ----------
D5.v1.pred_00030938-RA      FMINKCSACK
D5.v1.pred_00030965-RA      ----------
B1.B_06G223600-1            ----------
D5.v1.pred_00030950-RA      ----------
Gar06G05370                 ----------
Gar06G04420                 ----------
D5.v1.pred_00030947-RA      ----------
D5.v1.pred_00030954-RA      ----------
B1.B_06G224100-1            ----------
B1.B_06G222300-1            ----------
Gar06G04480                 ----------
Gar06G04460                 ----------
B1.B_06G214200-1            ----------
B1.B_06G214100-1            ----------
Gar06G05390                 ----------
B1.B_06G224000-1            ----------
Gar06G05690                 ----------
Gar11G43950                 ----------
    #+end_example
  + Tree
    #+begin_src newick
(
(
(
sp|Q8GXF6|FBK85_ARATH:0.60500,
(
D5.v1.pred_00030965-RA:0.16222,
Gar06G05370:0.03223)
:0.07187)
:0.01538,
D5.v1.pred_00030938-RA:0.14854)
:0.02136,
(
(
B1.B_06G223600-1:0.05963,
Gar06G04420:0.07753)
:0.05057,
(
D5.v1.pred_00030950-RA:0.13941,
(
D5.v1.pred_00030947-RA:0.01448,
D5.v1.pred_00030954-RA:0.01919)
:0.05090)
:0.01724)
:0.00232,
(
(
(
(
B1.B_06G224100-1:0.10491,
B1.B_06G224000-1:0.05195)
:0.00740,
(
Gar06G05690:0.04399,
Gar11G43950:0.04145)
:0.00918)
:0.00814,
(
Gar06G04460:0.02896,
(
B1.B_06G214200-1:0.02651,
(
B1.B_06G214100-1:0.02282,
Gar06G05390:0.01243)
:0.01195)
:0.03086)
:0.02826)
:0.01505,
(
B1.B_06G222300-1:0.01534,
Gar06G04480:0.00847)
:0.05133)
:0.00419);

    #+end_src


* Galactose Oxidase
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6830320/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5713210/
https://www.annualreviews.org/doi/pdf/10.1146/annurev-arplant-050213-040119
https://lib.dr.iastate.edu/cgi/viewcontent.cgi?article=2850&context=etd

- galactose metabolism ::  https://www.genome.jp/kegg-bin/show_pathway?map00052
  Galactose oxidase is EC 1.1.3.9. Does not have a ath ortholog yet.



* validate gene model

#+begin_src sh :results raw
cat << EOF | grep -h -f - {B1,A2,D5}.gff3 | awk '$3 == "gene"' | sed -e 's/ID=//' -e 's/;.*//'
B1.B_06G223600
B1.B_06G222300
B1.B_06G214200
B1.B_06G224100
B1.B_06G214100
B1.B_06G224000
D5.v1.pred_00030947
D5.v1.pred_00030954
D5.v1.pred_00030938
D5.v1.pred_00030950
D5.v1.pred_00030965
Gar11G43950
Gar06G05690
Gar06G05390
Gar06G04460
Gar06G04480
Gar06G04420
Gar06G05370
EOF

#+end_src

#+RESULTS:
| B_06  | maker  | B1.B_06G214200      |  90250525 |  90251463 | . | - | . |
| B_06  | maker  | B1.B_06G214100      |  90241766 |  90242704 | . | - | . |
| B_06  | maker  | B1.B_06G223600      |  92567389 |  92568340 | . | + | . |
| B_06  | maker  | B1.B_06G224100      |  92796959 |  92797909 | . | + | . |
| B_06  | maker  | B1.B_06G222300      |  92220772 |  92224175 | . | + | . |
| B_06  | maker  | B1.B_06G224000      |  92795094 |  92796090 | . | - | . |
| Chr06 | EVM    | Gar06G04420.gene    |   5064228 |   5065082 | . | + | . |
| Chr06 | EVM    | Gar06G04460.gene    |   5176244 |   5176972 | . | - | . |
| Chr06 | EVM    | Gar06G04480.gene    |   5226692 |   5227641 | . | - | . |
| Chr06 | EVM    | Gar06G05370.gene    |   7376477 |   7376980 | . | + | . |
| Chr06 | EVM    | Gar06G05390.gene    |   7413975 |   7414913 | . | + | . |
| Chr06 | EVM    | Gar06G05690.gene    |   8087626 |   8088576 | . | - | . |
| Chr11 | GeMoMa | Gar11G43950.gene    | 130941024 | 130942221 | . | + | . |
| D5_06 | maker  | D5.v1.pred_00030965 |   4093147 |   4094065 | . | + | . |
| D5_06 | maker  | D5.v1.pred_00030947 |   3912221 |   3913165 | . | - | . |
| D5_06 | maker  | D5.v1.pred_00030954 |   3968218 |   3969162 | . | - | . |
| D5_06 | maker  | D5.v1.pred_00030938 |   3872262 |   3874716 | . | - | . |
| D5_06 | maker  | D5.v1.pred_00030950 |   3936828 |   3937575 | . | - | . |

#+begin_src sh
bedtools getfasta -fi joint.fa -fo genes.of.interest.fa -bed genes.of.interest.gff -name
blastn -subject genes.of.interest.fa -query main.gene.of.interest.fa -out genes.of.interest.blast
#+end_src

#+begin_example
BLASTN 2.9.0+


Reference: Zheng Zhang, Scott Schwartz, Lukas Wagner, and Webb
Miller (2000), "A greedy algorithm for aligning DNA sequences", J
Comput Biol 2000; 7(1-2):203-14.



Database: User specified sequence set (Input: genes.of.interest.fa).
           17 sequences; 19,351 total letters



Query= B1.B_06G223600

Length=951
                                                                      Score        E
Sequences producing significant alignments:                          (Bits)     Value

D5.v1.pred_00030954                                                   1349       0.0
D5.v1.pred_00030947                                                   1343       0.0
Gar06G04480.gene                                                      1332       0.0
B1.B_06G222300                                                        1317       0.0
Gar11G43950.gene                                                      1312       0.0
Gar06G05690.gene                                                      1310       0.0
B1.B_06G224100                                                        1262       0.0
Gar06G04420.gene                                                      1249       0.0
D5.v1.pred_00030938                                                   1199       0.0
Gar06G05390.gene                                                      1190       0.0
B1.B_06G214200                                                        1177       0.0
B1.B_06G214100                                                        1146       0.0
D5.v1.pred_00030965                                                   1129       0.0
Gar06G04460.gene                                                      1003       0.0
B1.B_06G224000                                                        922        0.0
D5.v1.pred_00030950                                                   909        0.0
Gar06G05370.gene                                                      686        0.0


> D5.v1.pred_00030954
Length=944

 Score = 1349 bits (730),  Expect = 0.0
 Identities = 879/951 (92%), Gaps = 9/951 (1%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            |||| ||||||||||||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  943  TGAGGGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  884

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| |||||||| |||| |||||||||||||||||| |||||||| |||||| ||||
Sbjct  883  TCAGGGCCCTCGTCCAGAATTGGCCGGGTACCGATGGTAAAATGACGGTCGGTTTGCAC-  825

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            ||||||||| ||||| ||||| | |||||||||| ||||||||||||||||||||||||
Sbjct  824  TTTTTGGATCTGGAAAGCAATAAAGTCTTTATTAAGGCGATGCCACCTGATGCTCCTTTT  765

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | ||||||||||||||| |||||| ||||||||||||||||||||||||||||| |||||
Sbjct  764  TGTTCCACTGCCGTTGCCTGTGGCGACCACGTTTACGTTATTGGGGGAATACGCCAAAAT  705

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||||| |||||| |||||||| ||||||||||||||||||||||||||||||||||
Sbjct  704  GATGCTACTCTTTTTGATTATGATGGTGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  645

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            |||||||||||||||||||||||||||||||||||||||||||||| ||| |||| ||||
Sbjct  644  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTTTTTGCCCCGGGTT  585

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
             |||||||||||| |||||||| ||||||||||  ||| |||||||||||||||||||||
Sbjct  584  GTTGCTGCTGAAGGAAAGATTTTCGTCTTCGAATATATGGGTTCTGAATCTTTTGGCGAG  525

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            ||||||||||||||||||||||||||||||||||||||| |||||||||| | |||||
Sbjct  524  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGCCCCCTCCGGAAGATATCGAT  465

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
            ||||||| |||||||||||||||||||||||||||||||||||||||||||||||||||
Sbjct  464  CTATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAC  405

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            ||||| ||||||||||||||||  ||||||||||||| ||||||||||||||||||||||
Sbjct  404  GATACCCTTTACGCTTATTACTACGATCGTAAATCATAGGTTTGCCTCGAACAAAAATTT  345

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
            ||||||||||||||  ||||||||||||||||||||||||||||||||| ||||||||||
Sbjct  344  TGTTACTGGTCTGACGCAGCGACAATCGTGGACGATGTGCTGTATACTGCCATTTATAAT  285

Query  660  TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
            ||||||| ||||||||||||||||||||| ||||||||||||||||||||||||||| ||
Sbjct  284  TATTCTGATAAGTTCCGTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTGCCC  225

Query  720  GTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTTT  779
            |||||||||||||||| |||||||| |||||||| |     |||||||||||||| ||||
Sbjct  224  GTCAAATGGTCATCAGAATTTTCAG-TGGATCCA-C-----ATGGCACTCTGTATCGTTT  172

Query  780  GGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTACAT  839
            ||||||||||||||||||||| | |||| ||||||||    ||| |||||||||||||||
Sbjct  171  GGGCAATGGCAAGTGTATTTTCGTTTGGTTCAACCATCTCGATAAGAGTTTTGAGTACAT  112

Query  840  AAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  899
            |||||| |||||||||| ||||||||||||||||||||||||||||||||||||||||||
Sbjct  111  AAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  52

Query  900  TGCCATCACTGTTCCATATCCCAAAGATGTTCATGATATAATGTTGTTCTG  950
            |||||||| ||||||||||||||||||| ||  |||||||| |||||||||
Sbjct  51   TGCCATCAGTGTTCCATATCCCAAAGATATTTCTGATATAAAGTTGTTCTG  1


> D5.v1.pred_00030947
Length=944

 Score = 1343 bits (727),  Expect = 0.0
 Identities = 878/951 (92%), Gaps = 9/951 (1%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            |||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  943  TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  884

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| |||||||| ||||||||||||||||||||||| |||| ||| |||||| ||||
Sbjct  883  TCAGGGCCCTCGTCCAGAAATGGCCGGGTACCGATGGTAAAATTACGGTCGGTTTGCAC-  825

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            ||||||||| ||||| ||||||| |||||||||| ||||||||||||||||||||||||
Sbjct  824  TTTTTGGATCTGGAAAGCAATGAAGTCTTTATTAAGGCGATGCCACCTGATGCTCCTTTT  765

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | ||||||||||||||| |||||| ||||||||||||||||||||||||||||| |||||
Sbjct  764  TGTTCCACTGCCGTTGCCTGTGGCGACCACGTTTACGTTATTGGGGGAATACGCCAAAAT  705

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||||| |||||| | |||||| ||||||||||||||||||||||||||||||||||
Sbjct  704  GATGCTACTCTTTTTGATAATGATGGTGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  645

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            || ||||||||||||||||||||||||||||||||||||||||||  ||| |||| ||||
Sbjct  644  CTCGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGGTTTTGCCCCGGGTT  585

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
             |||||||||||| |||||||| ||||||||||  ||| ||||||||||||||||| |||
Sbjct  584  GTTGCTGCTGAAGGAAAGATTTTCGTCTTCGAATATATGGGTTCTGAATCTTTTGGTGAG  525

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            ||||||||||||||||||||||||||||||||||||||| |||||||||| | |||||
Sbjct  524  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGCCCCCTCCGGAAGATATCGAT  465

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
            ||||||| ||||||||||||||||||||||||||||||||||||||||||||||||||||
Sbjct  464  CTATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  405

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            |||||||||||||||||||||| |||||||||||||||||||||||||||||||||||||
Sbjct  404  GATACTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  345

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
            ||||||||||||||  ||||||||||||||||||||||||||||||||| ||||||||||
Sbjct  344  TGTTACTGGTCTGACGCAGCGACAATCGTGGACGATGTGCTGTATACTGCCATTTATAAT  285

Query  660  TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
            |||| || ||||||||||||||||||||| ||||||||||||||||||||||||||| ||
Sbjct  284  TATTATGATAAGTTCCGTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTGCCC  225

Query  720  GTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTTT  779
            |||||||||||||||| |||||||| |||||||| |     |||||||||||||| ||||
Sbjct  224  GTCAAATGGTCATCAGAATTTTCAG-TGGATCCA-C-----ATGGCACTCTGTATCGTTT  172

Query  780  GGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTACAT  839
            ||||||||||||||||||||| | |||| |||||| |    ||| |||||||||||||||
Sbjct  171  GGGCAATGGCAAGTGTATTTTCGTTTGGTTCAACCGTCTCGATAAGAGTTTTGAGTACAT  112

Query  840  AAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  899
            ||||||  ||||||||| ||||||||||||||||||||||||||||||||||||||||||
Sbjct  111  AAGATTTCATATTTGGTGCAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  52

Query  900  TGCCATCACTGTTCCATATCCCAAAGATGTTCATGATATAATGTTGTTCTG  950
            |||||||| ||||||||||||||||||| ||  |||||||| |||||||||
Sbjct  51   TGCCATCAGTGTTCCATATCCCAAAGATATTTCTGATATAAAGTTGTTCTG  1


> Gar06G04480.gene
Length=949

 Score = 1332 bits (721),  Expect = 0.0
 Identities = 876/952 (92%), Gaps = 6/952 (1%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            |||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  948  TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  889

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGG-TTGCACT  119
            ||||| |||||||||||||||  || |||||||||||| |||   ||||||| |||||||
Sbjct  888  TCAGGGCCCTCGTCGAGAAATCCCCAGGTACCGATGGTAAAACAGCGCTCGGTTTGCACT  829

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            || ||| || || || ||||| | |||||||||||||||||||||||||||||||||| |
Sbjct  828  TTATTG-ATCTGAAAAGCAATAAAGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTCA  770

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
              ||||||||||||||| |||||||||||||||||||||||||||||||||||| |||||
Sbjct  769  GGTTCCACTGCCGTTGCCTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCGAAAAT  710

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||| | ||  |||||||||||| |||||||||||||||||||||||||||||||||
Sbjct  709  GATGCTAGTCTTCGTGGTTATGATGAAGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  650

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            |||||||||||||||||||||| |||||||||||||||||||||| ||||||||||||||
Sbjct  649  CTTGAACGTGGGTGGAGAAAAA-TACTTCTATGTTGTTCCCTCGGGCTTTTCCCCTGGTT  591

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
            ||||||||||||| ||||||||||||||||||| |||| |||||||||||||||||||||
Sbjct  590  CTTGCTGCTGAAGGAAAGATTTACGTCTTCGAAGGTATGGGTTCTGAATCTTTTGGCGAG  531

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            ||||||||||||||| |||| |||||||||||||||||| |||||||||| | ||| |
Sbjct  530  GTTTACGATATAAGTTGGGACATTTGGGAACCATTGTCGTCCCCTCCGGAAGCTATTGAT  471

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
            ||||||| ||||||||||||||||||||||||||||||||||||||||||||||||||||
Sbjct  470  CTATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  411

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            |||||||||||||||||||||| |||||||||||||||||||||||||||||||||||||
Sbjct  410  GATACTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  351

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
            |||||||||||||| ||||||||||||||||||||||| |||||||||||||||||||||
Sbjct  350  TGTTACTGGTCTGACTCAGCGACAATCGTGGACGATGTCCTGTATACTGTCATTTATAAT  291

Query  660  TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
            ||||||||||||||||||||||||||||| ||||||||||||||||||||||||||| ||
Sbjct  290  TATTCTGGTAAGTTCCGTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTGCCC  231

Query  720  GTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTTT  779
            |||||||||||||||| |||||||| || ||||| ||  |||||||||||||||| | ||
Sbjct  230  GTCAAATGGTCATCAGAATTTTCAG-TGAATCCACCACCGAATGGCACTCTGTATCGGTT  172

Query  780  GGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTACAT  839
            |||||||||||||  |||||||||||||||||||| |   |||  | |||||||||||||
Sbjct  171  GGGCAATGGCAAGCTTATTTTGGGTTGGGTCAACCTTTTCCATGAGCGTTTTGAGTACAT  112

Query  840  AAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  899
            |||||| |||||||||| |||||||||||||||||||||||||||||||||||||||||
Sbjct  111  AAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATT  52

Query  900  TGCCATCACTGTTCCATATCCCAAAGATGTT-CATGATATAATGTTGTTCTG  950
            ||| ||||||||| |||||||| ||||| || |  | ||||  |||||||||
Sbjct  51   TGCGATCACTGTTTCATATCCCGAAGATATTTCTAG-TATATGGTTGTTCTG  1


> B1.B_06G222300
Length=3403

 Score = 1317 bits (713),  Expect = 0.0
 Identities = 874/953 (92%), Gaps = 5/953 (1%)
 Strand=Plus/Plus

Query  1     TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
             |||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  95    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  154

Query  61    TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGG-TTGCACT  119
             ||||| ||||||||||||| |  || |||||||||||| |||   ||||||| |||||||
Sbjct  155   TCAGGGCCCTCGTCGAGAACTCCCCAGGTACCGATGGTAAAACAGCGCTCGGTTTGCACT  214

Query  120   TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
             || || ||| || || ||||| | |||||||||||||||||||||||||||||||||| |
Sbjct  215   TTATT-GATCTGAAAAGCAATAAAGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTCA  273

Query  180   TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
               ||||||||||||||| |||||||||||| ||||||||||||||||||||||| |||||
Sbjct  274   GGTTCCACTGCCGTTGCCTGTGGCAACCACATTTACGTTATTGGGGGAATACGCGAAAAT  333

Query  240   GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
             ||||||| | ||  |||||||||||| |||||||||||||||||| ||||||||||||||
Sbjct  334   GATGCTAGTCTTCGTGGTTATGATGAAGTAAACGATGTTTTCCAACTAGATTTGAAGGAC  393

Query  300   CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
             ||||||||||||||||||||||||||||||||||||||||||||| ||||||||||||||
Sbjct  394   CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGGCTTTTCCCCTGGTT  453

Query  360   CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
             ||||||||||||| ||||||||||||||||||| |||| |||||||||||||||||||||
Sbjct  454   CTTGCTGCTGAAGGAAAGATTTACGTCTTCGAAGGTATGGGTTCTGAATCTTTTGGCGAG  513

Query  420   GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
             ||||||||||||||||||||||||||||||||||||| | |||||||||| | |||||
Sbjct  514   GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTTGCCCCCTCCGGAAGCTATCGAT  573

Query  480   CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
             |||| || |||||||| |||||||||||||||||||||||||||||||||||||||||||
Sbjct  574   CTATCTGTTCCTGTTCCAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  633

Query  540   GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
             |||||||||||||||||||||| |||||||||||||||||||||||||||||||||||||
Sbjct  634   GATACTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  693

Query  600   TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
             |||||||||||||| ||||||||||||||||||||||| |||||||||||||||||||||
Sbjct  694   TGTTACTGGTCTGACTCAGCGACAATCGTGGACGATGTCCTGTATACTGTCATTTATAAT  753

Query  660   TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
             ||||||||||||||||||||||||||||| ||||||||||||||||||||||||||| ||
Sbjct  754   TATTCTGGTAAGTTCCGTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTGCCC  813

Query  720   GTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTTT  779
             |||||||||||||||| ||||||| ||| ||||| ||  |||||||||||||||| | ||
Sbjct  814   GTCAAATGGTCATCAGAATTTTCA-GTGAATCCACCACCGAATGGCACTCTGTATCGGTT  872

Query  780   GGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTACAT  839
             |||||||||||||  |||||||||||||||||| | |   |||  | |||||||||||||
Sbjct  873   GGGCAATGGCAAGCTTATTTTGGGTTGGGTCAATCTTTTCCATGAGCGTTTTGAGTACAT  932

Query  840   AAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  899
             |||||| |||||||||| |||||||||||||||||||||||||||||||||||||||||
Sbjct  933   AAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATT  992

Query  900   TGCCATCACTGTTCCATATCCCAAAGATGTT-CATGATATAATGTTGTTCTGA  951
             ||| ||||||||| |||||||| ||||| || |  | ||||  ||||||||||
Sbjct  993   TGCGATCACTGTTTCATATCCCGAAGATATTTCTAG-TATATGGTTGTTCTGA  1044


> Gar11G43950.gene
Length=1197

 Score = 1312 bits (710),  Expect = 0.0
 Identities = 872/952 (92%), Gaps = 3/952 (0%)
 Strand=Plus/Plus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            |||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  60

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| ||||||||||||||| |||||||||||||||| |||| |||| ||||| |||||
Sbjct  61   TCAGGGCCCTCGTCGAGAAATCGCCGGGTACCGATGGTAAAATAACGCACGGTTTGCACT  120

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
             | || ||| ||||| ||||| | |||||||||||||||||||||||||||||||||| |
Sbjct  121  ATATT-GATCTGGAAAGCAATAAAGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTCA  179

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | ||||||||||||||| |||||||||||||||||||||||||||||||||||| |||||
Sbjct  180  TGTTCCACTGCCGTTGCCTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCGAAAAT  239

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||| | ||  |||||||||||  |||||||||||||||||||||||||||||||||
Sbjct  240  GATGCTAATTTTCGTGGTTATGATGGAGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            ||||||||||||||||||||||||||||||||||||||||||||| ||||||||||||||
Sbjct  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGGCTTTTCCCCTGGTT  359

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
            ||||||||||||| |||||||||||||||||||  |   |||||||||||||||||||||
Sbjct  360  CTTGCTGCTGAAGGAAAGATTTACGTCTTCGAATATTGGGGTTCTGAATCTTTTGGCGAG  419

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            ||||||||||||||||||||||||||||||||||||||| |||||||||| | |||||
Sbjct  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGCCCCCTCCGGAAGATATCGAT  479

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
             |||||| |||||||||||||| |||||||||||||||||||||||||||| |||||| |
Sbjct  480  ATATGTGTTCCTGTTCTAGATTGTTCCCGATCTCGGATTTTGGTGCACTGCCATGCCAGT  539

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            ||| |||||||||||||||||| |||||||||||||||||||||||||||||||||||||
Sbjct  540  GATGCTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
            |||||||||||||| ||||||||||||||||||||||||||||||||| | |||| ||||
Sbjct  600  TGTTACTGGTCTGACTCAGCGACAATCGTGGACGATGTGCTGTATACTTTGATTTTTAAT  659

Query  660  TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
            ||||||  ||||||||||||||||||||| ||||||||||||||||||||||||||| ||
Sbjct  660  TATTCTTATAAGTTCCGTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTGCCC  719

Query  720  GTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTTT  779
            |||||||||||||||| ||||||| |||| |||| ||  |  | ||||||||||| ||||
Sbjct  720  GTCAAATGGTCATCAGAATTTTCA-GTGGCTCCACCACCGGGTAGCACTCTGTATCGTTT  778

Query  780  GGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTACAT  839
            |||||||||||||  ||||||||||||||||||||||    ||| | |||||||||||||
Sbjct  779  GGGCAATGGCAAGCTTATTTTGGGTTGGGTCAACCATTTCGATAAGCGTTTTGAGTACAT  838

Query  840  AAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  899
            |||||| |||||||||| ||||||||||||||||||||||||||||||||||||||||||
Sbjct  839  AAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  898

Query  900  TGCCATCACTGTTCCATATCCCAAAGATGTTCATGATATAATGTTGTTCTGA  951
            | |||||||||||||||||||| ||||| |  |  |||||  ||||||||||
Sbjct  899  TCCCATCACTGTTCCATATCCCGAAGATATGGAAAATATATGGTTGTTCTGA  950


> Gar06G05690.gene
Length=950

 Score = 1310 bits (709),  Expect = 0.0
 Identities = 871/951 (92%), Gaps = 3/951 (0%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            |||||||||||||||||||||||| ||||||||||| |||||||||||||||||||||||
Sbjct  949  TGAGCGGCGATACTAGTGATTCCTTCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  890

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| ||||| | | |||||  |||||||||||||||||||| |||| ||||| |||||
Sbjct  889  TCAGGGCCCTCTTTGGGAAATATCCGGGTACCGATGGTGAAATAACGCACGGTTTGCACT  830

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            || ||| || ||||| ||||| | ||||||||||||||||||||||||||||||| || |
Sbjct  829  TTATTG-ATCTGGAAAGCAATAAAGTCTTTATTAGGGCGATGCCACCTGATGCTCGTTCA  771

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | ||||||||||||||||||||||||||||||| |||||||||||||||||||| |||||
Sbjct  770  TGTTCCACTGCCGTTGCTTGTGGCAACCACGTTAACGTTATTGGGGGAATACGCGAAAAT  711

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||| | ||  |||||||||||  |||||||||||||||||||||||||||||||||
Sbjct  710  GATGCTAATTTTCGTGGTTATGATGGAGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  651

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            ||||||||||||||||||||||||||||||||||||||||||||| ||||||||||||||
Sbjct  650  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGGCTTTTCCCCTGGTT  591

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
            ||||||||||||| |||||||||||||||||||  |   |||||||||||||||||||||
Sbjct  590  CTTGCTGCTGAAGGAAAGATTTACGTCTTCGAATATTGGGGTTCTGAATCTTTTGGCGAG  531

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            ||||||||||||||| ||||||||||||||||||||||| |||||||||| | |||||
Sbjct  530  GTTTACGATATAAGTAGGGATATTTGGGAACCATTGTCGCCCCCTCCGGAAGATATCGAT  471

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
            ||||||| ||||||||||||||||||||||||||||||||||||||||||| |||||| |
Sbjct  470  CTATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCCATGCCAGT  411

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            |||||||||||||||||||||| |||||||||||||||||||||||||||||||||||||
Sbjct  410  GATACTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  351

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
            |||||||||||||| ||||||||||||||||||||||||||||||||| | |||||||||
Sbjct  350  TGTTACTGGTCTGACTCAGCGACAATCGTGGACGATGTGCTGTATACTTTGATTTATAAT  291

Query  660  TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
            |||||| |||||||||||||||||||||||||||||||| ||||||||||||||||| ||
Sbjct  290  TATTCTTGTAAGTTCCGTTCTTTGGAGGCATATAATTTGTTTGATAAGAAACATTTGCCC  231

Query  720  GTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTTT  779
            ||||||| |||||||| |||||||| ||| |||| || ||  | |||||||||||  |||
Sbjct  230  GTCAAATTGTCATCAGAATTTTCAG-TGGCTCCACCACTGGGTAGCACTCTGTATCATTT  172

Query  780  GGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTACAT  839
            |||||||||||||  ||||| ||||||||||||||||   |||| | |||||||||||||
Sbjct  171  GGGCAATGGCAAGCTTATTTCGGGTTGGGTCAACCATTTCCATAAGCGTTTTGAGTACAT  112

Query  840  AAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  899
            |||||| |||||||||| ||||||||||||||||||||||||||||| ||||||||||||
Sbjct  111  AAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGCAGCTGTAGAGCATCAATC  52

Query  900  TGCCATCACTGTTCCATATCCCAAAGATGTTCATGATATAATGTTGTTCTG  950
            |||||||||||||||||||||| ||||| ||  | |||||  |||||||||
Sbjct  51   TGCCATCACTGTTCCATATCCCGAAGATATTTCTAATATATGGTTGTTCTG  1


> B1.B_06G224100
Length=950

 Score = 1262 bits (683),  Expect = 0.0
 Identities = 864/953 (91%), Gaps = 5/953 (1%)
 Strand=Plus/Plus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            ||||||||||||| |||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  1    TGAGCGGCGATACAAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  60

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGG-TTGCACT  119
            ||||| ||||||||  |||| ||| ||||||||||||| ||||  ||| ||| |||||||
Sbjct  61   TCAGGGCCCTCGTCCGGAAACGGCGGGGTACCGATGGTAAAATAGCGCACGGTTTGCACT  120

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            || || ||| ||||| ||||| | |||||||||||||||||||||||||||||||||| |
Sbjct  121  TTATT-GATCTGGAAAGCAATAAAGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTCA  179

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | ||||||||||||||| |||||||||||||||||| ||||||| ||||||| | |||||
Sbjct  180  TGTTCCACTGCCGTTGCCTGTGGCAACCACGTTTACATTATTGGAGGAATACACGAAAAT  239

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||| | ||  |||||||||||  |||||||||||||||||||||||||||||||||
Sbjct  240  GATGCTAATTTTCGTGGTTATGATGGAGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            ||||||||||||||||||||||||||||||||||||||||||| | ||||||||||||||
Sbjct  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCAGGCTTTTCCCCTGGTT  359

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
            ||||||| ||||| |||||||||| ||||||||  |   |||||||||||||||||||||
Sbjct  360  CTTGCTGTTGAAGGAAAGATTTACTTCTTCGAATATCGGGGTTCTGAATCTTTTGGCGAG  419

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            ||||||||||||||||||||||||||||||||||||||| |||||||||| | |||||
Sbjct  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGTCCCCTCCGGAAGATATCGAT  479

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
            ||||||| ||||||||||||||||||||||||||||||| ||||||||||| |||||| |
Sbjct  480  CTATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTCTGGTGCACTGCGATGCCAGT  539

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            |||||||||||||||||||||| ||||||||| |||||||||||||| ||||||||||||
Sbjct  540  GATACTCTTTACGCTTATTACTATGATCGTAACTCATGGGTTTGCCTTGAACAAAAATTT  599

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
            |||||||||||||| ||||||||||||||||||||||||||||||||| | |||||||||
Sbjct  600  TGTTACTGGTCTGACTCAGCGACAATCGTGGACGATGTGCTGTATACTTTGATTTATAAT  659

Query  660  TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
            |||||| |||||||||||||||||||||| ||||||||||||||||||||||||||| ||
Sbjct  660  TATTCTTGTAAGTTCCGTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTG-CC  718

Query  720  GTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTTT  779
            |||||||||||||||| ||||||| |||| |||| || ||  | ||||||||||| ||||
Sbjct  719  GTCAAATGGTCATCAGAATTTTCA-GTGGCTCCACCACTGGGTAGCACTCTGTATCGTTT  777

Query  780  GGGCAATGGCAAGTGTATTTTGGGTTGGG-TCAACCATAAACATATGAGTTTTGAGTACA  838
            |||||||||||||  |||||||||||||| |||||||    | || | ||||||||||||
Sbjct  778  GGGCAATGGCAAGCTTATTTTGGGTTGGGGTCAACCAGTTCCGTAAGCGTTTTGAGTACA  837

Query  839  TAAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAAT  898
            ||||||| |||||||||| ||||||||||||||||||||||||||||||||||||| |||
Sbjct  838  TAAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATAAAT  897

Query  899  CTGCCATCACTGTTCCATATCCCAAAGATGTTCATGATATAATGTTGTTCTGA  951
            ||||||||| ||||||||||||| ||||| ||  | |||||  ||||||||||
Sbjct  898  CTGCCATCAGTGTTCCATATCCCGAAGATATTTCTAATATATGGTTGTTCTGA  950


> Gar06G04420.gene
Length=854

 Score = 1249 bits (676),  Expect = 0.0
 Identities = 770/816 (94%), Gaps = 3/816 (0%)
 Strand=Plus/Plus

Query  53   TGAACTGATCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGG  112
            |||||||||||||  ||| ||||||||||||||||||||||||||| |||||||||||||
Sbjct  38   TGAACTGATCAGGGTCCTTGTCGAGAAATGGCCGGGTACCGATGGTAAAATGACGCTCGG  97

Query  113  TT-GCACTTTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATG  171
            || |||| ||||||||| ||||| |  |||| ||||||||||| ||||||||||||||||
Sbjct  98   TTTGCAC-TTTTTGGATCTGGAAAGGGATGAAGTCTTTATTAGAGCGATGCCACCTGATG  156

Query  172  CTCCTTTATCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATAC  231
            || |||||||||||||||||||||| ||||||||||||||||||||| ||||||||||||
Sbjct  157  CTTCTTTATCTTCCACTGCCGTTGCCTGTGGCAACCACGTTTACGTTCTTGGGGGAATAC  216

Query  232  GCAAAAATGATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATT  291
            ||||||||||||||||||||||||||||||||||||||||||| ||||||||||||||||
Sbjct  217  GCAAAAATGATGCTACTATTTTTGGTTATGATGATGTAAACGACGTTTTCCAATTAGATT  276

Query  292  TGAAGGACCTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTC  351
            ||||||||||||||| ||||||||||||||||||||||||||||||||||||||||||||
Sbjct  277  TGAAGGACCTTGAACATGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTC  336

Query  352  CCCTGGTTCTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTT  411
            ||||||||||||||||||||| |||||||||||||||||||||||| |||||||||||||
Sbjct  337  CCCTGGTTCTTGCTGCTGAAGGAAAGATTTACGTCTTCGAACGTATGGGTTCTGAATCTT  396

Query  412  TTGGCGAGGTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGG  471
            ||||| ||||||||||||||||||||||||||||||||||||||| | ||| |||||| |
Sbjct  397  TTGGCTAGGTTTACGATATAAGTGGGGATATTTGGGAACCATTGTTGCCCCGTCCGGAAG  456

Query  472  GTATCGGCCTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCA  531
            ||||| |  |||| ||||||||||||||||||| ||||||||||||||||||||||||||
Sbjct  457  GTATCAGTTTATGCGATCCTGTTCTAGATTCTTTCCGATCTCGGATTTTGGTGCACTGCA  516

Query  532  ATGCCAATGATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAAC  591
            |||||||||||||||||||||||||||||| |||| | ||||||||||||||||||| ||
Sbjct  517  ATGCCAATGATACTCTTTACGCTTATTACTATGATAGAAAATCATGGGTTTGCCTCGGAC  576

Query  592  AAAAATTTTGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCA  651
            ||||||||||||||||||||||||| ||||||||||||||||||||||||||||||||||
Sbjct  577  AAAAATTTTGTTACTGGTCTGAATCGGCGACAATCGTGGACGATGTGCTGTATACTGTCA  636

Query  652  TTTATAATTATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAAC  711
            ||||||||||||||||||||||||||||||||||||| ||||||||||||||||||||||
Sbjct  637  TTTATAATTATTCTGGTAAGTTCCGTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAAC  696

Query  712  ATTTGACCGTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTG  771
            ||||| |||| ||||||||||||| ||||||| ||||||| ||||||||| |||| ||||
Sbjct  697  ATTTGCCCGTTAAATGGTCATCAGAATTTTCA-GTGGATCTAACAATGAAAGGCATTCTG  755

Query  772  TATAGTTTGGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTT  831
            ||| ||||||||||||||||||||||||||||||||||||||||||||||||||||||||
Sbjct  756  TATCGTTTGGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTT  815

Query  832  GAGTACATAAGATTAAATATTTGGTACAACGAGCAG  867
            |||||| ||||||||| |||||||| ||||||||||
Sbjct  816  GAGTACTTAAGATTAACTATTTGGTGCAACGAGCAG  851


> D5.v1.pred_00030938
Length=2454

 Score = 1199 bits (649),  Expect = 0.0
 Identities = 863/962 (90%), Gaps = 31/962 (3%)
 Strand=Plus/Minus

Query  1     TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
             |||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  1771  TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  1712

Query  61    TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGG-TTGCACT  119
             ||||| |||||||| ||||| ||||||||||||||||| |||  |||||||| |||||||
Sbjct  1711  TCAGGGCCCTCGTCCAGAAACGGCCGGGTACCGATGGTAAAAGAACGCTCGGTTTGCACT  1652

Query  120   TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
             || ||| || ||||| ||||| | |||||||||| ||||||||||||||||||||||| |
Sbjct  1651  TTATTG-ATCTGGAAAGCAATAAAGTCTTTATTAAGGCGATGCCACCTGATGCTCCTTCA  1593

Query  180   TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
               ||||||||||||| | ||||||||||||||||| |||||||||||||||||||||| |
Sbjct  1592  AGTTCCACTGCCGTTTCCTGTGGCAACCACGTTTATGTTATTGGGGGAATACGCAAAA-T  1534

Query  240   GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
             |||||||||  |||||||||||||||||||||||||||||||||||||||||||||||||
Sbjct  1533  GATGCTACTCCTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  1474

Query  300   CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
             |||||||||||||||||||||||||||||||||||||||||||||||||||||||  |||
Sbjct  1473  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCATGTT  1414

Query  360   CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
             ||||||||||||| ||||||||||||||||||| |||| |||||||||| ||||||||||
Sbjct  1413  CTTGCTGCTGAAGGAAAGATTTACGTCTTCGAATGTATGGGTTCTGAAT-TTTTGGCGAG  1355

Query  420   GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGG-----G-T  473
             ||||||||||||||||||||||||||||||||||||||| |||||||||| |     | |
Sbjct  1354  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGCCCCCTCCGGAAGCTATCGAT  1295

Query  474   AT-CG-GCC-TATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGC  530
             || || |   |||||| |||||||||||||||||||||||||||||||||||||||||||
Sbjct  1294  ATACGTGTTGTATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGC  1235

Query  531   AATGCCAATGATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAA  590
              |||||| ||||||||||||||||||||||| ||||||||||||||||||||||||||||
Sbjct  1234  GATGCCAGTGATACTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAA  1175

Query  591   CAAAAATTTTGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTC  650
             ||||||||||||||||||| ||| |||||| |||||||||||||||||||||||||| ||
Sbjct  1174  CAAAAATTTTGTTACTGGTTTGACTCAGCGGCAATCGTGGACGATGTGCTGTATACTTTC  1115

Query  651   ATTTATAATTATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAA  710
             ||||||||  ||             ||||||||||||| |||||||||||||||||||||
Sbjct  1114  ATTTATAA--AT-------------GTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAA  1070

Query  711   CATTTGACCGTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCT  770
             |||||| |||||||||||||||||| |||| ||| ||| |||| || ||  | || ||||
Sbjct  1069  CATTTGCCCGTCAAATGGTCATCAGAATTTCCAG-TGGCTCCACCACTGGGTAGCGCTCT  1011

Query  771   GTATAGTTTGGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTT  830
             |||| || ||||||||||||||  ||||||||||||||||||||||   |||| | ||||
Sbjct  1010  GTATCGTCTGGGCAATGGCAAGCTTATTTTGGGTTGGGTCAACCATCTCCATAGGGGTTT  951

Query  831   TGAGTACATAAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGA  890
             ||||| ||||||||| |||||||||| |||||||||||||||||||||||||||||||||
Sbjct  950   TGAGTGCATAAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGCAGCTGCAGA  891

Query  891   GCATCAATCTGCCATCACTGTTCCATATCCCAAAGATGTT-CATGATATAATGTTGTTCT  949
             ||||||||||||||||||||||||||||||| ||||| || |  | ||||  ||| ||||
Sbjct  890   GCATCAATCTGCCATCACTGTTCCATATCCCGAAGATATTTCTAG-TATATGGTTTTTCT  832

Query  950   GA  951
             ||
Sbjct  831   GA  830


> Gar06G05390.gene
Length=938

 Score = 1190 bits (644),  Expect = 0.0
 Identities = 852/952 (89%), Gaps = 15/952 (2%)
 Strand=Plus/Plus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            ||||||||||||||||||||||||| |||||||||| |||||||||||||||||||||||
Sbjct  1    TGAGCGGCGATACTAGTGATTCCTCTAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  60

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| |||||||| |||||||||||| |||||||||| ||||||||||||||| || ||
Sbjct  61   TCAGGGCCCTCGTCCAGAAATGGCCGGATACCGATGGTAAAATGACGCTCGGTTTGCGCT  120

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            || || ||| ||||| | ||| | |||||||||| ||||||||||||||||||||||| |
Sbjct  121  TTATT-GATCTGGAAAGGAATAAAGTCTTTATTAAGGCGATGCCACCTGATGCTCCTTCA  179

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
              ||||||||||||| | |||||||||||||||||||||||||||||||| ||| |||||
Sbjct  180  GGTTCCACTGCCGTTTCCTGTGGCAACCACGTTTACGTTATTGGGGGAATGCGCGAAAAT  239

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||| | ||  |||||||||||  |||||||||||||||||||||||||||||||||
Sbjct  240  GATGCTAATTTTCGTGGTTATGATGGAGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            ||||||||||||||||||||||||| ||||||||||||||||||| ||||||||||||||
Sbjct  300  CTTGAACGTGGGTGGAGAAAAACTATTTCTATGTTGTTCCCTCGGGCTTTTCCCCTGGTT  359

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
            ||||||||||||| |||||||||||||||||||  |   |||||||||||||||||||||
Sbjct  360  CTTGCTGCTGAAGGAAAGATTTACGTCTTCGAATATTGGGGTTCTGAATCTTTTGGCGAG  419

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            ||||||||||||||||||||| ||||||||||||||||| |||||||||| | |||||
Sbjct  420  GTTTACGATATAAGTGGGGATTTTTGGGAACCATTGTCGCCCCCTCCGGAAGCTATCGAT  479

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
             |||||| ||||||||||||||||||||||||||||||||||||||||||| |||||| |
Sbjct  480  ATATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCCATGCCAGT  539

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            |||||||||||||||||||||| |||||||||||||||||||||||||||||||||||||
Sbjct  540  GATACTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
            ||||||||||||||  ||||||||||||||||||||||| ||||||||||| || |||||
Sbjct  600  TGTTACTGGTCTGACACAGCGACAATCGTGGACGATGTGTTGTATACTGTCCTTGATAAT  659

Query  660  TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
            ||||||  |       ||||||||||||| ||||||||||||||||||||||||||| ||
Sbjct  660  TATTCT--T-------GTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTGCCC  710

Query  720  GTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTTT  779
            |||||||||| ||| | | || ||| ||  |||| ||  |  | ||||||||||| ||||
Sbjct  711  GTCAAATGGTTATCGGAAGTTCCAG-TGACTCCACCACAGGGTTGCACTCTGTATCGTTT  769

Query  780  GGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTACAT  839
            ||||| |||||||  ||||||||||||||||||||||   ||||    ||||||||| ||
Sbjct  770  GGGCAGTGGCAAGCTTATTTTGGGTTGGGTCAACCATCTCCATAC---TTTTGAGTATAT  826

Query  840  AAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAATC  899
            |||||| |||||||||| ||||||||||||||||||||||| ||||||||||||||||||
Sbjct  827  AAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGTAGCTGCAGAGCATCAATC  886

Query  900  TGCCATCACTGTTCCATATCCCAAAGATGTTCATGATATAATGTTGTTCTGA  951
            |||||||||||||||||||||||||||| ||  | || || |||| ||||||
Sbjct  887  TGCCATCACTGTTCCATATCCCAAAGATATTTGTAATTTATTGTTCTTCTGA  938


> B1.B_06G214200
Length=938

 Score = 1177 bits (637),  Expect = 0.0
 Identities = 850/952 (89%), Gaps = 17/952 (2%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            |||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  937  TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  878

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| |||||||||| |||||||||| |||||||||| |||| |||||||||| || ||
Sbjct  877  TCAGGGCCCTCGTCGATAAATGGCCGGATACCGATGGTAAAATAACGCTCGGTTTGCGCT  818

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            || ||| || ||||| | ||| | |||||||||| ||||||||||||||||||||||| |
Sbjct  817  TTATTG-ATCTGGAAAGGAATAAAGTCTTTATTAAGGCGATGCCACCTGATGCTCCTTCA  759

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | ||||||||||||| | |||||||||||||||||||||||||||||||| ||| |||||
Sbjct  758  TGTTCCACTGCCGTTTCCTGTGGCAACCACGTTTACGTTATTGGGGGAATGCGCGAAAAT  699

Query  240  GATGCTACTATTTTTGGTTATGAT-GATGTAAACGATGTTTTCCAATTAGATTTGAAGGA  298
            ||||||| | ||  |||||||||| || ||||||||||||||||||||||||||||||||
Sbjct  698  GATGCTAATTTTCGTGGTTATGATAGA-GTAAACGATGTTTTCCAATTAGATTTGAAGGA  640

Query  299  CCTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGT  358
            ||||||||||||||||||||||||||||||||||||||||||||||  ||||||||||||
Sbjct  639  CCTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGGTTTTTCCCCTGGT  580

Query  359  TCTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGA  418
            |||||||||||||| || ||||||||||||||||  | | ||||||||||||||||||||
Sbjct  579  TCTTGCTGCTGAAGGAAGGATTTACGTCTTCGAATATTTGGGTTCTGAATCTTTTGGCGA  520

Query  419  GGTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGG  478
            |||||||||||||||||||||| ||||||||||||||||| |||||||||| | |||||
Sbjct  519  GGTTTACGATATAAGTGGGGATTTTTGGGAACCATTGTCGCCCCCTCCGGAAGCTATCGA  460

Query  479  CCTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAA  538
              |||||| |||| |||||||||||||||||||||||||||||||||||||| ||||||
Sbjct  459  TATATGTGTTCCTATTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCCATGCCAG  400

Query  539  TGATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATT  598
            |||||| |||||||||||||||| |||||||||||||||||||| ||| |||||||||||
Sbjct  399  TGATACACTTTACGCTTATTACTATGATCGTAAATCATGGGTTTACCTTGAACAAAAATT  340

Query  599  TTGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAA  658
            |||||||||||||||  ||||||||||||||||||||||||||||||||||| || ||||
Sbjct  339  TTGTTACTGGTCTGACACAGCGACAATCGTGGACGATGTGCTGTATACTGTCCTTGATAA  280

Query  659  TTATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGAC  718
            |||||||  |       ||||||||||||| ||||||||||||||||||||||||||| |
Sbjct  279  TTATTCT--T-------GTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTGCC  229

Query  719  CGTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAGTT  778
            ||||||||||||||| |||  |||  |||  |||| ||  |  | ||||||||||| |||
Sbjct  228  CGTCAAATGGTCATC-GGAAGTTCCAGTGACTCCACCACAGGGTTGCACTCTGTATCGTT  170

Query  779  TGGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTACA  838
            |||||| |||||||  ||||||||||||||||||||||   | ||    ||||||||| |
Sbjct  169  TGGGCAGTGGCAAGCTTATTTTGGGTTGGGTCAACCATCTCCGTAC---TTTTGAGTATA  113

Query  839  TAAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCAAT  898
            ||||||| |||||||||| ||||||||||||||||||||||| |||||||||||||||||
Sbjct  112  TAAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGTAGCTGCAGAGCATCAAT  53

Query  899  CTGCCATCACTGTTCCATATCCCAAAGATGTTCATGATATAATGTTGTTCTG  950
            ||||||||||||||||||||||||||||| ||  | |||||  ||| |||||
Sbjct  52   CTGCCATCACTGTTCCATATCCCAAAGATATTTCTAATATATGGTTCTTCTG  1


> B1.B_06G214100
Length=938

 Score = 1146 bits (620),  Expect = 0.0
 Identities = 833/934 (89%), Gaps = 21/934 (2%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            |||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||
Sbjct  937  TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATGAACTGA  878

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| |||||||| |||||||||||| |||||||||| ||||| ||||||||| || ||
Sbjct  877  TCAGGGCCCTCGTCCAGAAATGGCCGGATACCGATGGTAAAATGCCGCTCGGTTTGCGCT  818

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            || ||| || ||||| | ||| | |||||||||| ||||||||||||||||||||||| |
Sbjct  817  TTATTG-ATCTGGAAAGGAATAAAGTCTTTATTAAGGCGATGCCACCTGATGCTCCTTCA  759

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
              ||||||||||||| | |||||||||||||||||||||||||||||||||||| |||||
Sbjct  758  GGTTCCACTGCCGTTTCCTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCGAAAAT  699

Query  240  GATGCTACTATTTTTGGTTATGAT-GATGTAAACGATGTTTTCCAATTAGATTTGAAGGA  298
            ||||||| | ||  |||||||||| ||| |||||||||||||||||||||||||||| ||
Sbjct  698  GATGCTAATTTTCGTGGTTATGATGGAT-TAAACGATGTTTTCCAATTAGATTTGAATGA  640

Query  299  CCTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGT  358
            |||||||||||||||||||||||||||||||||||||||||||||| |||||||||||||
Sbjct  639  CCTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGGCTTTTCCCCTGGT  580

Query  359  TCTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTA-T-AGGTTCTGAATCTTTTGGC  416
            |||||||||||||| |||||||||||||||||||  || |  ||||||||||||||||||
Sbjct  579  TCTTGCTGCTGAAGGAAAGATTTACGTCTTCGAA--TATTGGGGTTCTGAATCTTTTGGC  522

Query  417  GAGGTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATC  476
            |||||||| ||||||||||||||| ||||||||  ||||||| |||||||||| | ||||
Sbjct  521  GAGGTTTATGATATAAGTGGGGATTTTTGGGAATTATTGTCGCCCCCTCCGGAAGCTATC  462

Query  477  GGCCTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCC  536
            |   |||||| |||||||||||||||||||||||||||||||||||| |||||| |||||
Sbjct  461  GATATATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTACACTGCCATGCC  402

Query  537  AATGATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAA  596
            | ||||||||||||||||||||||| ||||||||||||||||||||||||||||||||||
Sbjct  401  AGTGATACTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAA  342

Query  597  TTTTGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTAT  656
            |||||||||||||||||  ||||||||||||||||||||||||||||||||||| || ||
Sbjct  341  TTTTGTTACTGGTCTGACACAGCGACAATCGTGGACGATGTGCTGTATACTGTCCTTGAT  282

Query  657  AATTATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTG  716
            |||||||||  |       ||||||||||||| |||||||||||||||||||||||||||
Sbjct  281  AATTATTCT--T-------GTTCTTTGGAGGCGTATAATTTGCTTGATAAGAAACATTTG  231

Query  717  ACCGTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCACTCTGTATAG  776
             |||||||||||| ||| |||  |||  |||  |||| ||  |  | ||||||||||| |
Sbjct  230  CCCGTCAAATGGTTATC-GGAAGTTCCAGTGACTCCACCACAGGGTTGCACTCTGTATCG  172

Query  777  TTTGGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGAGTTTTGAGTA  836
            |||||||| |||||||  ||||||||||||||||||||||   | ||    |||||||||
Sbjct  171  TTTGGGCAGTGGCAAGCTTATTTTGGGTTGGGTCAACCATCTCCGTAC---TTTTGAGTA  115

Query  837  CATAAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTGCAGAGCATCA  896
              ||||||| |||||||||| ||||||||||||||||||||||| |||||| |||| |||
Sbjct  114  TTTAAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATTCATGTAGCTGCCGAGCGTCA  55

Query  897  ATCTGCCATCACTGTTCCATATCCCAAAGATGTT  930
            |||||||||| |||||||||||||||||||| ||
Sbjct  54   ATCTGCCATCGCTGTTCCATATCCCAAAGATATT  21


> D5.v1.pred_00030965
Length=918

 Score = 1129 bits (611),  Expect = 0.0
 Identities = 815/908 (90%), Gaps = 36/908 (4%)
 Strand=Plus/Plus

Query  53   TGAACTGATCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGG  112
            ||||||||||||| ||| |||| ||||||||||||||||||||||| |||||||||||||
Sbjct  38   TGAACTGATCAGGACCCACGTCCAGAAATGGCCGGGTACCGATGGTAAAATGACGCTCGG  97

Query  113  TT-GCACTTTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATG  171
            || |||| ||||||||| ||||| || |||| ||||||||||||||||||||||||||||
Sbjct  98   TTTGCAC-TTTTTGGATCTGGAAAGCGATGAAGTCTTTATTAGGGCGATGCCACCTGATG  156

Query  172  CTCCTTTATCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATAC  231
            |||||||||| |||||||||||||| ||||||||||||||||||||| ||||||||||||
Sbjct  157  CTCCTTTATC-TCCACTGCCGTTGCCTGTGGCAACCACGTTTACGTTCTTGGGGGAATAC  215

Query  232  GCAAAAATGATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATT  291
            ||||||||||||||||| ||  ||||||||||||||||||||||||||||||||||||||
Sbjct  216  GCAAAAATGATGCTACTTTTGGTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATT  275

Query  292  TGAAGGACCTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTC-TTTT  350
             ||||||||||||||||||||||||||||||| ||||||||||||||||| || | ||||
Sbjct  276  GGAAGGACCTTGAACGTGGGTGGAGAAAAACTGCTTCTATGTTGTTCCCTAGGGCTTTTT  335

Query  351  CCCCTGGTTCTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATA-GGTTCTGAATC  409
            |||||||||||||||||||||| |||||||||||||||||||| |||  |||| ||||||
Sbjct  336  CCCCTGGTTCTTGCTGCTGAAGGAAAGATTTACGTCTTCGAAC-TATTGGGTTTTGAATC  394

Query  410  TTTTGGCGAGGTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGA  469
            ||||||||| |||||||||||||||||||||||||||||||||||||| |||||||||||
Sbjct  395  TTTTGGCGACGTTTACGATATAAGTGGGGATATTTGGGAACCATTGTC-GCCCCTCCGGA  453

Query  470  GGGTATCGGCCTATGTGATCCTGTT---CTAGATTCTTCCCGATCTCGGATTTTGGTGCA  526
             | |||||  ||||||| |||||||   ||||||||| ||||||||| |||||||||
Sbjct  454  AGCTATCGCTCTATGTGGTCCTGTTCTCCTAGATTCTCCCCGATCTCAGATTTTGGT---  510

Query  527  CTGCAATGCCAATGATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCT  586
             | |  | |            |||||||||||||| ||||||||||||||||||||||||
Sbjct  511  -T-C--T-C------------TTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCT  553

Query  587  CGAACAAAAATTTTGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATAC  646
            ||||| |||||||||||||||||||||  ||||| |||||||||||||||||||||||||
Sbjct  554  CGAACCAAAATTTTGTTACTGGTCTGACCCAGCGTCAATCGTGGACGATGTGCTGTATAC  613

Query  647  TGTCATTTATAATTATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAA  706
            |||||| ||||||||||||||| ||||| |||||||||||||||||||||||||||||||
Sbjct  614  TGTCATGTATAATTATTCTGGTGAGTTCTGTTCTTTGGAGGCATATAATTTGCTTGATAA  673

Query  707  GAAACATTTGACCGTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACAATGAATGGCA  766
            |||||||||| | |||||||||||||||| ||||||| ||||||||| ||| || |||||
Sbjct  674  GAAACATTTGCCGGTCAAATGGTCATCAGAATTTTCA-GTGGATCCACCAAGGAGTGGCA  732

Query  767  CTCTGTATAGTTTGGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCATAAACATATGA  826
            |||||| | |||||||||||||||||||||||||| ||||||||||| |||| |  ||||
Sbjct  733  CTCTGTTTGGTTTGGGCAATGGCAAGTGTATTTTGCGTTGGGTCAACTATAA-CG-ATGA  790

Query  827  GTTTTGAGTACATAAGATTAAATATTTGGTACAACGAGCAGGGAGGGATTCATGCAGCTG  886
            ||||||||||||||||||| ||||| |||| ||||||||||||||||||| |||||||||
Sbjct  791  GTTTTGAGTACATAAGATTTAATATGTGGTGCAACGAGCAGGGAGGGATTGATGCAGCTG  850

Query  887  CAGAGCATCAATCTGCCATCACTGTTCCATATC---CCAAAGATGTTCATGATATAATGT  943
            |||||||||||||||||||||||||||| ||||   |||||||| ||  ||||||| | |
Sbjct  851  CAGAGCATCAATCTGCCATCACTGTTCCTTATCGATCCAAAGATATTTGTGATATATTTT  910

Query  944  TGTTCTGA  951
            ||||||||
Sbjct  911  TGTTCTGA  918


> Gar06G04460.gene
Length=728

 Score = 1003 bits (543),  Expect = 0.0
 Identities = 666/727 (92%), Gaps = 2/727 (0%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            ||||||| |||||||||||||||||||||||||||| |||| |||||||||||||||||
Sbjct  727  TGAGCGGTGATACTAGTGATTCCTCCAGTTACTTTGATGGGAGAGAGCCAGATGAACTGG  668

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| |||||||| |||||||||||| || ||||||| |||| |||| ||||| |||||
Sbjct  667  TCAGGGCCCTCGTCCAGAAATGGCCGGATATCGATGGTAAAATAACGCACGGTTTGCACT  608

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            || ||| || ||||| ||||| | |||||||||| ||||||||||||||||||||||| |
Sbjct  607  TTATTG-ATCTGGAAAGCAATAAAGTCTTTATTAAGGCGATGCCACCTGATGCTCCTTCA  549

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | ||||||||||||| | |||||||| ||||||||||||||||||||||| ||| |||||
Sbjct  548  TGTTCCACTGCCGTTTCCTGTGGCAATCACGTTTACGTTATTGGGGGAATGCGCGAAAAT  489

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||| | ||  |||||||||||  |||||||||||||||||||||||||||||||||
Sbjct  488  GATGCTAATTTTCGTGGTTATGATGGAGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  429

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            |||||||||||||||||||||||||||||||||||||||||||||  |||||||||||||
Sbjct  428  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGGTTTTTCCCCTGGTT  369

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
            ||||||||||||| |||||||||||||||| ||  | | | |||||||||||||||||||
Sbjct  368  CTTGCTGCTGAAGGAAAGATTTACGTCTTCCAATATTTGGTTTCTGAATCTTTTGGCGAG  309

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            ||||||||||||||||||||| ||||||||||||||||| |||||||||| | |||||
Sbjct  308  GTTTACGATATAAGTGGGGATTTTTGGGAACCATTGTCGCCCCCTCCGGAAGCTATCGAT  249

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
             |||||| ||||||||||||||||||||||||||||||||||||||||||| |||||| |
Sbjct  248  ATATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCCATGCCAGT  189

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            ||||||||||||||| || ||| |||||||||||||||||||||||||||||||||||||
Sbjct  188  GATACTCTTTACGCTAATCACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  129

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTGTCATTTATAAT  659
            ||||||||||||||  |||||||||||||||||||||| |||||||||||||||||||||
Sbjct  128  TGTTACTGGTCTGACACAGCGACAATCGTGGACGATGTCCTGTATACTGTCATTTATAAT  69

Query  660  TATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATTTGCTTGATAAGAAACATTTGACC  719
            |||||||||||||||||||||| |||||| |||||||||||||||| |||||||||| ||
Sbjct  68   TATTCTGGTAAGTTCCGTTCTTCGGAGGCGTATAATTTGCTTGATAGGAAACATTTGCCC  9

Query  720  GTCAAAT  726
            |||||||
Sbjct  8    GTCAAAT  2


> B1.B_06G224000
Length=996

 Score = 922 bits (499),  Expect = 0.0
 Identities = 616/674 (91%), Gaps = 1/674 (0%)
 Strand=Plus/Minus

Query  277  TTTTCCAATTAGATTTGAAGGACCTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGT  336
            ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
Sbjct  673  TTTTCCAATTAGATTTGAAGGACCTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGT  614

Query  337  TCCCTCGGTCTTTTCCCCTGGTTCTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTA  396
            |||||||| ||||||||||||||||||||||||| | |||| ||||||||||||||| |
Sbjct  613  TCCCTCGGGCTTTTCCCCTGGTTCTTGCTGCTGACGGAAAGGTTTACGTCTTCGAACATT  554

Query  397  TAGGTTCTGAATCTTTTGGCGAGGTTTACGATATAAGTGGGGATATTTGGGAACCATTGT  456
              ||||||||||||||||| ||||||||||||||||||||||||||||||||||||||||
Sbjct  553  GGGGTTCTGAATCTTTTGGGGAGGTTTACGATATAAGTGGGGATATTTGGGAACCATTGT  494

Query  457  CGGCCCCTCCGGAGGGTATCGGCCTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGA  516
            || |||||||||| | |||||  ||| ||| |||||||||||||||||||||||||||||
Sbjct  493  CGCCCCCTCCGGAAGATATCGATCTAAGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGA  434

Query  517  TTTTGGTGCACTGCAATGCCAATGATACTCTTTACGCTTATTACTTTGATCGTAAATCAT  576
            || ||||||||||| |||||| ||||||||||||||||||||||| ||||||||||||||
Sbjct  433  TTCTGGTGCACTGCGATGCCAGTGATACTCTTTACGCTTATTACTATGATCGTAAATCAT  374

Query  577  GGGTTTGCCTCGAACAAAAATTTTGTTACTGGTCTGAATCAGCGACAATCGTGGACGATG  636
            ||||||||||||||||||||||||||||||||||||| ||||||||||||||||||||||
Sbjct  373  GGGTTTGCCTCGAACAAAAATTTTGTTACTGGTCTGACTCAGCGACAATCGTGGACGATG  314

Query  637  TGCTGTATACTGTCATTTATAATTATTCTGGTAAGTTCCGTTCTTTGGAGGCATATAATT  696
            ||||||||||| | ||||||||||||||| |||||||||||||||||||||| |||||||
Sbjct  313  TGCTGTATACTTTGATTTATAATTATTCTTGTAAGTTCCGTTCTTTGGAGGCGTATAATT  254

Query  697  TGCTTGATAAGAAACATTTGACCGTCAAATGGTCATCAGGATTTTCAGGTGGATCCAACA  756
            |||||||||||||||||||| | |||||||||||||||| |||||||| ||| |||| ||
Sbjct  253  TGCTTGATAAGAAACATTTGCCTGTCAAATGGTCATCAGAATTTTCAG-TGGCTCCACCA  195

Query  757  ATGAATGGCACTCTGTATAGTTTGGGCAATGGCAAGTGTATTTTGGGTTGGGTCAACCAT  816
             ||  | |||||| |||| |||||||||||||||||  |||||||||||||||||||||
Sbjct  194  CTGGGTAGCACTCCGTATCGTTTGGGCAATGGCAAGCTTATTTTGGGTTGGGTCAACCAG  135

Query  817  AAACATATGAGTTTTGAGTACATAAGATTAAATATTTGGTACAACGAGCAGGGAGGGATT  876
               | || | ||||||||||||||||||| |||||||||| |||||||||||||||||||
Sbjct  134  TTCCGTAAGCGTTTTGAGTACATAAGATTTAATATTTGGTGCAACGAGCAGGGAGGGATT  75

Query  877  CATGCAGCTGCAGAGCATCAATCTGCCATCACTGTTCCATATCCCAAAGATGTTCATGAT  936
            ||||||| |||||||||| |||||||||||| ||||||||||||| ||||| ||  | ||
Sbjct  74   CATGCAGTTGCAGAGCATAAATCTGCCATCAGTGTTCCATATCCCGAAGATATTTCTAAT  15

Query  937  ATAATGTTGTTCTG  950
            |||  |||||||||
Sbjct  14   ATATGGTTGTTCTG  1


 Score = 409 bits (221),  Expect = 1e-116
 Identities = 290/324 (90%), Gaps = 2/324 (1%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            ||||||||||||| |||||||||||||||||||||| ||||| ||| |||||||||||||
Sbjct  995  TGAGCGGCGATACGAGTGATTCCTCCAGTTACTTTGATGGGCAAGACCCAGATGAACTGA  936

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGG-TTGCACT  119
            ||||| |||| |||  |||| ||||||||||||||||| ||||   || ||| |||||||
Sbjct  935  TCAGGGCCCTTGTCAGGAAACGGCCGGGTACCGATGGTAAAATAGTGCACGGTTTGCACT  876

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            |||||| || ||||| ||||| | |||||||||| ||||||||||||||||||||||| |
Sbjct  875  TTTTTG-ATCTGGAAAGCAATAAAGTCTTTATTAAGGCGATGCCACCTGATGCTCCTTCA  817

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | |||||||| |||||| |||||||||||||||||||||||||||||||||||| |||||
Sbjct  816  TGTTCCACTGTCGTTGCCTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCGAAAAT  757

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            ||||||| | ||  |||||||||||  |||||  ||||||||||||||||||||||||||
Sbjct  756  GATGCTAATTTTCGTGGTTATGATGCAGTAAATAATGTTTTCCAATTAGATTTGAAGGAC  697

Query  300  CTTGAACGTGGGTGGAGAAAAACT  323
            ||||||||||||||||||||||||
Sbjct  696  CTTGAACGTGGGTGGAGAAAAACT  673


> D5.v1.pred_00030950
Length=747

 Score = 909 bits (492),  Expect = 0.0
 Identities = 598/649 (92%), Gaps = 7/649 (1%)
 Strand=Plus/Minus

Query  1    TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGGTGGGCGAGAGCCAGATGAACTGA  60
            |||||||||||||||||||||||||||||||||||| |||||||||||||||| ||||||
Sbjct  746  TGAGCGGCGATACTAGTGATTCCTCCAGTTACTTTGATGGGCGAGAGCCAGATCAACTGA  687

Query  61   TCAGGTCCCTCGTCGAGAAATGGCCGGGTACCGATGGTGAAATGACGCTCGGTT-GCACT  119
            ||||| |||||||||||||| |||||||| |||||||| |||| |||||||||| | |||
Sbjct  686  TCAGGGCCCTCGTCGAGAAACGGCCGGGTGCCGATGGTAAAATAACGCTCGGTTTGGACT  627

Query  120  TTTTTGGATGTGGAATGCAATGATGTCTTTATTAGGGCGATGCCACCTGATGCTCCTTTA  179
            |  ||| || ||||| ||| | | || ||||||| ||||||||||||||||||||||| |
Sbjct  626  TCATTG-ATCTGGAAAGCATTTAAGT-TTTATTAAGGCGATGCCACCTGATGCTCCTTCA  569

Query  180  TCTTCCACTGCCGTTGCTTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  239
            | ||||||||||||| | ||||||||||||||||||||||||||||||||||||||||||
Sbjct  568  TGTTCCACTGCCGTTTCCTGTGGCAACCACGTTTACGTTATTGGGGGAATACGCAAAAAT  509

Query  240  GATGCTACTATTTTTGGTTATGATGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  299
            |||||||||  |  |  || |  |||||||||||||||||||||||||||||||||||||
Sbjct  508  GATGCTACT-CTACTCCTT-T--TGATGTAAACGATGTTTTCCAATTAGATTTGAAGGAC  453

Query  300  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTTCCCCTGGTT  359
            |||||||||||||||||||||||||||||||||||||||||||||||||| |||||||||
Sbjct  452  CTTGAACGTGGGTGGAGAAAAACTACTTCTATGTTGTTCCCTCGGTCTTTGCCCCTGGTT  393

Query  360  CTTGCTGCTGAAGAAAAGATTTACGTCTTCGAACGTATAGGTTCTGAATCTTTTGGCGAG  419
              ||||||||||| |||||||| ||||||||||  ||| |||||||||||||||||||||
Sbjct  392  GCTGCTGCTGAAGGAAAGATTTTCGTCTTCGAAAATATGGGTTCTGAATCTTTTGGCGAG  333

Query  420  GTTTACGATATAAGTGGGGATATTTGGGAACCATTGTCGGCCCCTCCGGAGGGTATCGGC  479
            |||||| |||||||||||||||||||||||||||||||| |||||||||| | |||||
Sbjct  332  GTTTACAATATAAGTGGGGATATTTGGGAACCATTGTCGCCCCCTCCGGAAGATATCGAT  273

Query  480  CTATGTGATCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAATGCCAAT  539
            ||||||| |||||||||||||||||||||||||||||||||||||||||||| |||||||
Sbjct  272  CTATGTGTTCCTGTTCTAGATTCTTCCCGATCTCGGATTTTGGTGCACTGCAGTGCCAAT  213

Query  540  GATACTCTTTACGCTTATTACTTTGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  599
            |||||||||||||||||||||| |||||||||||||||||||||||||||||||||||||
Sbjct  212  GATACTCTTTACGCTTATTACTATGATCGTAAATCATGGGTTTGCCTCGAACAAAAATTT  153

Query  600  TGTTACTGGTCTGAATCAGCGACAATCGTGGACGATGTGCTGTATACTG  648
            ||||||||||||||  |||||||||||||||||||||||||||||||||
Sbjct  152  TGTTACTGGTCTGACGCAGCGACAATCGTGGACGATGTGCTGTATACTG  104


> Gar06G05370.gene
Length=503

 Score = 686 bits (371),  Expect = 0.0
 Identities = 459/502 (91%), Gaps = 3/502 (1%)
 Strand=Plus/Plus

Query  331  TGTTGTTCCCTCGGTCTTTTCCCCTGGTTCTTGCTGCTGAAGAAAAGATTTACGTCTTCG  390
            |||||||||||||| |||||||||||||| ||||||| |||| |||||||||||||||||
Sbjct  1    TGTTGTTCCCTCGGGCTTTTCCCCTGGTTTTTGCTGCAGAAGGAAAGATTTACGTCTTCG  60

Query  391  AACGTATAGG-TTCTGAATCTTTTGGCGAGGTTTACGATATAAGTGGGGATATTTGGGAA  449
            |||  || || || ||||||||||||||||||||||||||||||||||||||||||||||
Sbjct  61   AAC-AATTGGTTTTTGAATCTTTTGGCGAGGTTTACGATATAAGTGGGGATATTTGGGAA  119

Query  450  CCATTGTCGGCCCCTCCGGAGGGTATCGGCCTATGTGATCCTGTTCTAGATTCTTCCCGA  509
            ||||||||| |||||| ||| | ||| |  ||| || |||||||||||||||||||||||
Sbjct  120  CCATTGTCGCCCCCTCTGGAAGCTATTGCTCTAAGTAATCCTGTTCTAGATTCTTCCCGA  179

Query  510  TCTCGGATTTTGGTGCACTGCAATGCCAATGATACTCTTTACGCTTATTACTTTGATCGT  569
            ||||||||||||||||||||| ||| ||||||| |||||||||||||||||| |||||||
Sbjct  180  TCTCGGATTTTGGTGCACTGCCATGTCAATGATTCTCTTTACGCTTATTACTATGATCGT  239

Query  570  AAATCATGGGTTTGCCTCGAACAAAAATTTTGTTACTGGTCTGAATCAGCGACAATCGTG  629
            |||||||||||||||||||| ||||||||||||||||||||||| | |||| ||||||||
Sbjct  240  AAATCATGGGTTTGCCTCGATCAAAAATTTTGTTACTGGTCTGACTTAGCGTCAATCGTG  299

Query  630  GACGATGTGCTGTATACTGTCATTTATAATTATTCTGGTAAGTTCCGTTCTTTGGAGGCA  689
            ||||||||||||||||||||||| ||||||||||||||| ||||| |||||||||||||
Sbjct  300  GACGATGTGCTGTATACTGTCATGTATAATTATTCTGGTGAGTTCTGTTCTTTGGAGGCG  359

Query  690  TATAATTTGCTTGATAAGAAACATTTGACCGTCAAATGGTCATCAGGATTTTCAGGTGGA  749
            ||||||||||||||||||||||||||| |||||||||||||||||| ||||||| |||||
Sbjct  360  TATAATTTGCTTGATAAGAAACATTTGCCCGTCAAATGGTCATCAGAATTTTCA-GTGGA  418

Query  750  TCCAACAATGAATGGCACTCTGTATAGTTTGGGCAATGGCAAGTGTATTTTGGGTTGGGT  809
            |||| ||| || ||||||||||| | |||||||||||||| |||| ||||||||||||||
Sbjct  419  TCCACCAAGGAGTGGCACTCTGTTTCGTTTGGGCAATGGCGAGTGGATTTTGGGTTGGGT  478

Query  810  CAACCATAAACATATGAGTTTT  831
            ||||||||   |||||||||||
Sbjct  479  CAACCATATCGATATGAGTTTT  500



Lambda      K        H
    1.33    0.621     1.12

Gapped
Lambda      K        H
    1.28    0.460    0.850

Effective search space used: 17838865


  Database: User specified sequence set (Input: genes.of.interest.fa).
    Posted date:  Unknown
  Number of letters in database: 19,351
  Number of sequences in database:  17



Matrix: blastn matrix 1 -2
Gap Penalties: Existence: 0, Extension: 2.5

#+end_example

non synonymous sub at 735?
frame shift insertion at 745?


* verify unique region

Align to only B genome
#+begin_src sh :var sra=sra_data[,0] :tangle b-only/align.sh

SRA=${sra[$SLURM_ARRAY_TASK_ID]};

ml singularity/3.5.2 bwa samtools

bwa mem -t $SLURM_CPUS_PER_TASK B1.fa raw/${SRA}_{1,2}.fastq.gz |
    samtools view -bS - |
    samtools sort -o b-only/$SRA.bam -T b-only/$SRA.tmp -
samtools index b-only/$SRA.bam
samtools flagstat b-only/$SRA.bam > b-only/$SRA.stats
#+end_src



 | SRR6334601 | 2(A2D3)              |
 | SRR6334602 | 2(A2D1)              |
 | SRR8979965 | Gossypium arboreum   |
 | SRR8979944 | Gossypium arboreum   |
 | SRR8979925 | Gossypium arboreum   |
 | SRR6334584 | Gossypium davidsonii |
 | SRR8136261 | Gossypium davidsonii |

Get snps and pileup
#+begin_src sh

ml singularity/3.5.2 samtools bcftools

sra=(SRR6334601 SRR6334602 SRR8979965 SRR8979944 SRR8979925 SRR6334584 SRR8136261)

bams=$(printf "b-only/%s.bam " ${sra[@]})

declare -A genes
genes["B06G223600"]="B_06:92567389-92568340"
genes["B06G223900"]="B_06:92741077-92741403"

for gene in "${!genes[@]}"; do
    bcftools mpileup -r ${genes[$gene]} -f B1.fa $bams | bcftools call -c > b-only/$gene.vcf
    samtools mpileup -r ${genes[$gene]} -f B1.fa -a $bams > b-only/$gene.ref.mpileup
    samtools mpileup -r ${genes[$gene]} -a $bams > b-only/$gene.mpileup
    mkdir b-only/$gene
    for sra in "${sra[@]}"; do
        samtools view -b b-only/$sra.bam ${genes[$gene]} > b-only/$gene/$sra.bam
        samtools index b-only/$gene/$sra.bam
    done
done


#+end_src


** B06G223600
#+begin_src sh
awk '!/^##/ {$8 = ""; c=0; for(i = 10; i < NF; i++) c += ($i !="0/0:0"); if(c) print}' OFS="\t" b-only/B06G223600.vcf
#+end_src

#+RESULTS:
| #CHROM |      POS | ID | REF | ALT |    QUAL | FILTER |   | FORMAT | b-only/SRR6334601.bam     | b-only/SRR6334602.bam   | b-only/SRR8979965.bam   | b-only/SRR8979944.bam   | b-only/SRR8979925.bam   | b-only/SRR6334584.bam     | b-only/SRR8136261.bam   |
| B_06   | 92567395 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:116,0,255             | 0/1:30,0,255            | 0/0:0,90,255            | 0/0:0,96,255            | 0/0:0,96,255            | 0/1:163,0,145             | 0/0:0,32,85             |
| B_06   | 92567396 | .  | G   | T   |     999 | .      |   | GT:PL  | 0/1:127,0,255             | 0/1:255,0,76            | 1/1:255,90,0            | 1/1:255,96,0            | 1/1:255,87,0            | 0/0:0,132,207             | 0/0:0,45,88             |
| B_06   | 92567401 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/1:123,0,255             | 0/1:255,0,79            | 1/1:255,87,0            | 1/1:255,96,0            | 1/1:255,93,0            | 0/0:0,138,206             | 0/0:0,36,69             |
| B_06   | 92567415 | .  | C   | A   |     999 | .      |   | GT:PL  | 0/1:122,0,255             | 0/1:255,0,81            | 1/1:255,72,0            | 1/1:255,75,0            | 1/1:255,90,0            | 0/0:0,138,208             | 0/0:0,18,33             |
| B_06   | 92567417 | .  | G   | C   |     999 | .      |   | GT:PL  | 0/1:120,0,255             | 0/1:255,0,79            | 1/1:255,72,0            | 1/1:255,75,0            | 1/1:255,87,0            | 0/0:0,141,207             | 0/0:0,9,27              |
| B_06   | 92567422 | .  | T   | A   | 74.4676 | .      |   | GT:PL  | 0/1:53,0,255              | 0/1:17,0,255            | 0/0:0,72,255            | 0/0:0,75,255            | 0/0:0,90,255            | 0/1:55,0,137              | 0/0:0,6,19              |
| B_06   | 92567426 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:92,0,255              | 0/1:12,0,191            | 0/0:0,72,255            | 0/0:0,78,251            | 0/0:0,87,255            | 0/1:136,0,39              | 0/1:19,6,0              |
| B_06   | 92567431 | .  | C   | T   | 75.4651 | .      |   | GT:PL  | 0/1:111,0,255             | 0/0:0,9,61              | 0/0:0,0,0               | 0/0:0,0,0               | 0/0:0,0,0               | 0/0:0,78,167              | 0/0:0,6,19              |
| B_06   | 92567432 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:255,0,120             | 0/0:0,9,61              | 0/0:0,0,0               | 0/0:0,0,0               | 0/0:0,0,0               | 0/0:0,84,169              | 0/0:0,6,19              |
| B_06   | 92567453 | .  | G   | A   | 129.908 | .      |   | GT:PL  | 0/0:0,117,255             | 0/0:0,54,255            | 0/0:0,75,255            | 0/0:0,78,255            | 0/0:0,72,255            | 0/1:168,0,140             | 0/0:0,3,4               |
| B_06   | 92567454 | .  | G   | A   | 7.71226 | .      |   | GT:PL  | 0/1:45,0,255              | 0/0:0,54,255            | 0/0:0,75,255            | 0/0:0,78,255            | 0/0:0,72,255            | 0/0:0,117,206             | 0/0:0,3,4               |
| B_06   | 92567455 | .  | T   | G,A |     999 | .      |   | GT:PL  | 0/1:241,0,249,114,130,255 | 1/1:255,51,6,255,0,255  | 1/1:255,75,0,255,75,255 | 1/1:255,78,0,255,78,255 | 1/1:255,53,0,255,72,255 | 1/1:255,238,171,188,0,136 | 1/1:4,3,0,4,3,4         |
| B_06   | 92567456 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:123,0,255             | 0/1:255,0,44            | 1/1:255,75,0            | 1/1:255,78,0            | 1/1:255,44,0            | 0/0:0,120,207             | 0/1:0,3,4               |
| B_06   | 92567459 | .  | T   | A   |     999 | .      |   | GT:PL  | 0/1:103,0,255             | 0/1:6,0,255             | 0/0:0,75,255            | 0/0:0,78,255            | 0/0:0,69,255            | 0/1:170,0,134             | 0/0:0,3,4               |
| B_06   | 92567460 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:129,0,255             | 0/1:255,0,44            | 1/1:255,75,0            | 1/1:255,78,0            | 1/1:255,69,0            | 0/0:0,120,208             | 0/1:0,3,4               |
| B_06   | 92567464 | .  | G   | C   |     999 | .      |   | GT:PL  | 0/1:106,0,255             | 0/1:30,0,255            | 0/0:0,81,255            | 0/0:0,87,255            | 0/0:0,81,255            | 0/1:168,0,111             | 0/0:0,3,4               |
| B_06   | 92567474 | .  | C   | A   | 74.9538 | .      |   | GT:PL  | 0/1:185,170,255           | 0/0:0,35,255            | 0/0:0,90,255            | 0/0:0,87,255            | 0/0:0,93,255            | 0/1:104,0,173             | 0/0:4,3,0               |
| B_06   | 92567475 | .  | G   | A   | 68.9079 | .      |   | GT:PL  | 0/1:107,0,255             | 0/0:0,54,255            | 0/0:0,90,255            | 0/0:0,90,255            | 0/0:0,96,255            | 0/0:0,30,172              | 0/0:0,3,4               |
| B_06   | 92567480 | .  | C   | T   | 166.908 | .      |   | GT:PL  | 0/0:0,111,255             | 0/0:0,54,255            | 0/0:0,87,255            | 0/0:0,93,255            | 0/0:0,93,255            | 0/1:205,0,141             | 0/0:0,3,4               |
| B_06   | 92567488 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 1/1:255,32,0            | 1/1:255,78,0            | 1/1:255,87,0            | 1/1:255,87,0            | 0/1:204,0,105             | 1/1:0,3,4               |
| B_06   | 92567492 | .  | T   | .   | 12.3609 | .      |   | GT:PL  | 0/0:16                    | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:16                    | 0/0:0                   |
| B_06   | 92567493 | .  | G   | A   | 37.5267 | .      |   | GT:PL  | 0/1:15,0,255              | 0/0:0,54,255            | 0/0:0,84,255            | 0/0:0,87,255            | 0/0:0,93,255            | 0/1:67,0,205              | 0/0:0,0,0               |
| B_06   | 92567496 | .  | G   | A   | 123.234 | .      |   | GT:PL  | 0/1:161,0,255             | 0/0:0,54,255            | 0/0:0,87,255            | 0/0:0,84,255            | 0/0:0,90,255            | 0/0:0,46,216              | 0/0:0,0,0               |
| B_06   | 92567508 | .  | T   | C   | 46.9362 | .      |   | GT:PL  | 0/0:0,24,255              | 0/0:0,0,0               | 0/1:41,3,0              | 0/1:51,0,98             | 0/0:0,12,82             | 0/0:0,9,76                | 0/0:0,0,0               |
| B_06   | 92567518 | .  | G   | C   |     999 | .      |   | GT:PL  | 1/1:255,75,0              | 1/1:255,30,0            | 1/1:255,66,0            | 1/1:255,75,0            | 1/1:255,78,0            | 1/1:232,48,0              | 1/1:11,3,0              |
| B_06   | 92567524 | .  | T   | A   |     999 | .      |   | GT:PL  | 1/1:255,72,0              | 1/1:251,30,0            | 1/1:255,66,0            | 1/1:255,75,0            | 1/1:255,78,0            | 1/1:239,45,0              | 1/1:11,3,0              |
| B_06   | 92567526 | .  | C   | G   |     999 | .      |   | GT:PL  | 0/1:84,0,255              | 0/1:208,0,29            | 1/1:255,66,0            | 1/1:255,72,0            | 1/1:255,81,0            | 0/0:0,48,238              | 0/0:0,9,59              |
| B_06   | 92567527 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 1/1:238,10,0            | 1/1:255,66,0            | 1/1:255,75,0            | 1/1:255,81,0            | 0/1:198,0,25              | 1/1:44,0,2              |
| B_06   | 92567532 | .  | T   | A   |     999 | .      |   | GT:PL  | 0/1:242,0,243             | 1/1:255,36,0            | 1/1:255,66,0            | 1/1:255,75,0            | 1/1:255,87,0            | 1/1:198,7,0               | 1/1:61,9,0              |
| B_06   | 92567543 | .  | G   | A   | 18.8385 | .      |   | GT:PL  | 0/0:0,16,255              | 0/1:56,0,255            | 0/0:0,78,255            | 0/0:0,81,255            | 0/0:0,90,255            | 0/0:0,14,255              | 0/0:3,0,72              |
| B_06   | 92567544 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:187,0,255             | 0/1:255,0,105           | 1/1:255,78,0            | 1/1:255,78,0            | 1/1:255,81,0            | 0/0:0,54,255              | 0/0:0,15,92             |
| B_06   | 92567548 | .  | A   | T   | 146.568 | .      |   | GT:PL  | 0/1:185,0,255             | 0/0:0,54,255            | 0/0:0,78,255            | 0/0:0,78,255            | 0/0:0,81,255            | 0/0:0,60,255              | 0/0:0,24,101            |
| B_06   | 92567563 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:203,0,255             | 0/1:249,0,148           | 1/1:255,69,0            | 1/1:255,75,0            | 1/1:255,87,0            | 0/0:0,75,255              | 0/0:0,27,106            |
| B_06   | 92567566 | .  | T   | G   |     999 | .      |   | GT:PL  | 0/1:218,0,255             | 0/0:0,54,255            | 0/0:0,72,255            | 0/0:0,75,255            | 0/0:0,93,255            | 0/0:2,0,255               | 0/0:0,27,106            |
| B_06   | 92567567 | .  | T   | .   | 21.4667 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:17                    | 0/0:0                   |
| B_06   | 92567568 | .  | A   | T   |     999 | .      |   | GT:PL  | 0/1:24,0,255              | 0/1:98,0,255            | 0/0:0,72,255            | 0/0:0,72,255            | 0/0:0,81,255            | 0/1:108,0,255             | 0/1:31,0,60             |
| B_06   | 92567570 | .  | C   | G   |     999 | .      |   | GT:PL  | 0/1:34,0,255              | 0/1:100,0,255           | 0/0:0,72,255            | 0/0:0,72,255            | 0/0:0,96,255            | 0/1:150,0,75              | 1/1:50,18,0             |
| B_06   | 92567584 | .  | G   | T   | 76.9277 | .      |   | GT:PL  | 0/0:0,60,255              | 0/0:0,41,255            | 0/0:0,60,255            | 0/0:0,78,255            | 0/0:0,93,255            | 0/1:110,0,255             | 0/1:12,0,112            |
| B_06   | 92567585 | .  | C   | .   | 35.4385 | .      |   | GT:PL  | 0/0:3                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                     | 0/0:0                   |
| B_06   | 92567586 | .  | T   | C   |     999 | .      |   | GT:PL  | 0/1:255,0,185             | 1/1:255,60,0            | 1/1:255,57,0            | 1/1:255,75,0            | 1/1:255,87,0            | 1/1:255,42,0              | 1/1:156,69,0            |
| B_06   | 92567593 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/1:18,0,255              | 0/1:111,0,255           | 0/0:0,60,255            | 0/0:0,75,255            | 0/0:0,84,255            | 0/1:117,0,255             | 0/1:102,0,149           |
| B_06   | 92567604 | .  | C   | T   | 241.687 | .      |   | GT:PL  | 0/0:0,0,255               | 0/0:0,33,255            | 0/0:0,63,255            | 0/0:0,72,255            | 0/0:0,90,255            | 0/1:159,0,255             | 0/1:128,0,173           |
| B_06   | 92567608 | .  | A   | C   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/1:255,0,167           | 1/1:255,63,0            | 1/1:255,72,0            | 1/1:255,87,0            | 0/1:255,0,255             | 0/1:0,0,100             |
| B_06   | 92567618 | .  | T   | G   |     999 | .      |   | GT:PL  | 0/0:0,151,255             | 0/0:0,87,255            | 0/0:0,72,255            | 0/0:0,75,255            | 0/0:0,75,255            | 0/1:255,0,255             | 0/1:14,0,106            |
| B_06   | 92567620 | .  | C   | T   | 65.5375 | .      |   | GT:PL  | 0/0:0,56,255              | 0/0:0,15,255            | 0/0:0,72,255            | 0/0:0,75,255            | 0/0:0,78,255            | 0/1:103,0,255             | 0/0:2,0,163             |
| B_06   | 92567621 | .  | G   | A   | 134.565 | .      |   | GT:PL  | 0/1:173,0,255             | 0/0:0,84,255            | 0/0:0,72,255            | 0/0:0,75,255            | 0/0:0,78,255            | 0/0:0,104,255             | 0/0:51,193,255          |
| B_06   | 92567623 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/0:26,75,255             | 0/0:141,140,255         | 0/0:0,72,255            | 0/0:0,72,255            | 0/0:0,72,255            | 0/1:255,0,255             | 0/1:215,65,139          |
| B_06   | 92567629 | .  | G   | T   | 51.5639 | .      |   | GT:PL  | 0/0:0,154,255             | 0/1:90,0,255            | 0/0:0,75,255            | 0/0:0,66,255            | 0/0:0,87,255            | 0/0:0,175,255             | 0/0:0,255,255           |
| B_06   | 92567636 | .  | C   | A   | 56.6476 | .      |   | GT:PL  | 0/0:0,65,255              | 0/0:0,32,255            | 0/0:0,78,255            | 0/0:0,48,255            | 0/0:0,81,255            | 0/1:95,0,255              | 0/0:0,9,193             |
| B_06   | 92567638 | .  | A   | C   |     999 | .      |   | GT:PL  | 0/1:186,0,255             | 0/1:172,0,255           | 0/0:0,78,255            | 0/0:0,45,255            | 0/0:0,78,255            | 0/0:255,255,255           | 1/1:255,171,2           |
| B_06   | 92567639 | .  | T   | C   |     999 | .      |   | GT:PL  | 0/1:39,0,255              | 0/0:0,32,255            | 0/0:0,78,255            | 0/0:0,48,255            | 0/0:0,84,255            | 0/1:255,0,255             | 0/1:218,0,221           |
| B_06   | 92567641 | .  | T   | G   |     999 | .      |   | GT:PL  | 0/1:210,0,255             | 0/0:0,69,255            | 0/0:0,78,255            | 0/0:0,48,255            | 0/0:0,87,255            | 0/1:255,0,255             | 0/0:0,51,180            |
| B_06   | 92567642 | .  | T   | G   |     999 | .      |   | GT:PL  | 0/1:217,0,255             | 0/0:0,55,255            | 0/0:0,81,255            | 0/0:0,48,255            | 0/0:0,81,255            | 0/1:255,0,255             | 0/0:0,13,150            |
| B_06   | 92567644 | .  | G   | T   | 60.5639 | .      |   | GT:PL  | 0/1:99,0,255              | 0/0:0,126,255           | 0/0:0,78,255            | 0/0:0,51,255            | 0/0:0,84,255            | 0/0:0,179,255             | 0/0:0,255,255           |
| B_06   | 92567645 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/0:0,29,255              | 0/1:125,0,255           | 0/0:0,78,255            | 0/0:0,51,255            | 0/0:0,81,255            | 0/1:132,0,255             | 0/1:40,0,214            |
| B_06   | 92567647 | .  | T   | A   |     999 | .      |   | GT:PL  | 0/1:66,0,255              | 0/0:0,32,255            | 0/0:0,75,255            | 0/0:0,51,255            | 0/0:0,87,255            | 0/1:163,0,255             | 0/0:0,28,251            |
| B_06   | 92567649 | .  | T   | A   | 22.5875 | .      |   | GT:PL  | 0/0:0,118,255             | 0/0:0,49,255            | 0/0:0,75,255            | 0/0:0,51,255            | 0/0:0,84,255            | 0/1:61,0,255              | 0/0:0,170,255           |
| B_06   | 92567654 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/0:0,28,255              | 0/1:109,0,255           | 0/0:0,78,255            | 0/0:0,51,255            | 0/0:0,87,255            | 0/1:110,0,255             | 0/0:0,93,255            |
| B_06   | 92567664 | .  | T   | C   |     999 | .      |   | GT:PL  | 0/1:90,0,255              | 0/1:255,0,222           | 1/1:255,81,0            | 1/1:255,51,0            | 1/1:255,59,0            | 0/0:0,255,255             | 0/0:0,255,255           |
| B_06   | 92567665 | .  | G   | A   | 72.5641 | .      |   | GT:PL  | 0/1:111,0,255             | 0/0:0,144,255           | 0/0:0,81,255            | 0/0:0,54,255            | 0/0:0,84,255            | 0/0:0,255,255             | 0/0:0,255,255           |
| B_06   | 92567681 | .  | T   | G   |     999 | .      |   | GT:PL  | 0/1:149,0,255             | 0/1:32,0,255            | 0/0:0,87,255            | 0/0:0,57,255            | 0/0:0,78,255            | 0/1:250,0,255             | 0/1:3,0,248             |
| B_06   | 92567696 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:158,0,255             | 0/1:255,0,184           | 1/1:255,87,0            | 1/1:255,57,0            | 1/1:255,87,0            | 0/0:0,253,255             | 0/0:0,255,255           |
| B_06   | 92567721 | .  | G   | T   | 54.5639 | .      |   | GT:PL  | 0/1:93,0,255              | 0/0:0,89,255            | 0/0:0,81,255            | 0/0:0,60,255            | 0/0:0,75,255            | 0/0:0,217,255             | 0/0:0,255,255           |
| B_06   | 92567729 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/0:0,120,255           | 0/0:0,69,255            | 0/0:0,57,255            | 0/0:0,75,255            | 0/0:0,124,255             | 0/0:0,255,255           |
| B_06   | 92567735 | .  | C   | .   | 7.32591 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:32                    | 0/0:0                   |
| B_06   | 92567739 | .  | T   | G   | 12.7976 | .      |   | GT:PL  | 0/0:0,132,255             | 0/0:0,51,255            | 0/0:0,72,255            | 0/0:0,48,255            | 0/0:0,72,255            | 0/1:51,0,255              | 0/0:0,138,255           |
| B_06   | 92567744 | .  | T   | A   |     999 | .      |   | GT:PL  | 0/1:73,0,255              | 0/0:0,12,255            | 0/0:0,63,255            | 0/0:0,48,255            | 0/0:0,66,255            | 0/1:246,0,165             | 1/1:214,72,59           |
| B_06   | 92567745 | .  | G   | T   |     999 | .      |   | GT:PL  | 0/1:83,0,255              | 0/0:0,3,255             | 0/0:0,69,255            | 0/0:0,51,255            | 0/0:0,69,255            | 0/1:255,0,245             | 0/1:159,0,6             |
| B_06   | 92567749 | .  | C   | .   | 17.5144 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:21                    | 0/0:0                   |
| B_06   | 92567752 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/0:0,90,255            | 0/0:0,69,255            | 0/0:0,51,255            | 0/0:0,69,255            | 0/0:0,61,255              | 0/0:0,18,195            |
| B_06   | 92567755 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/0:0,81,255            | 0/0:0,63,255            | 0/0:0,42,255            | 0/0:0,63,255            | 0/1:60,0,255              | 0/0:0,35,196            |
| B_06   | 92567762 | .  | A   | G   |     999 | .      |   | GT:PL  | 1/1:255,108,0             | 1/1:255,78,0            | 1/1:255,54,0            | 1/1:255,48,0            | 1/1:255,60,0            | 1/1:255,96,0              | 1/1:53,9,0              |
| B_06   | 92567764 | .  | A   | G   | 4.68768 | .      |   | GT:PL  | 0/0:0,108,255             | 0/0:0,75,255            | 0/0:0,51,255            | 0/0:0,48,255            | 0/0:0,57,255            | 0/0:0,93,255              | 0/1:44,3,0              |
| B_06   | 92567774 | .  | T   | G   |     999 | .      |   | GT:PL  | 0/1:246,0,255             | 0/0:0,66,255            | 0/0:0,63,255            | 0/0:0,48,255            | 0/0:0,57,255            | 0/0:0,84,255              | 0/0:0,9,53              |
| B_06   | 92567782 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/0:0,99,255              | 0/0:0,37,255            | 0/0:0,66,255            | 0/0:0,51,255            | 0/0:0,57,255            | 0/1:219,0,54              | 0/0:0,3,44              |
| B_06   | 92567787 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/1:246,0,255             | 1/1:255,57,0            | 1/1:255,66,0            | 1/1:255,54,0            | 1/1:255,57,0            | 0/1:199,0,20              | 1/1:4,3,0               |
| B_06   | 92567806 | .  | G   | T   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 1/1:255,24,0            | 1/1:255,63,0            | 1/1:255,54,0            | 1/1:255,60,0            | 0/0:0,45,176              | 0/1:0,6,20              |
| B_06   | 92567808 | .  | G   | C   | 5.98862 | .      |   | GT:PL  | 0/0:0,61,255              | 0/0:0,57,255            | 0/0:0,66,255            | 0/0:0,63,255            | 0/0:0,60,255            | 0/1:43,0,133              | 0/0:0,6,20              |
| B_06   | 92567824 | .  | G   | C   | 5.98866 | .      |   | GT:PL  | 0/0:0,87,255              | 0/0:0,42,255            | 0/0:0,69,255            | 0/0:0,66,255            | 0/0:0,60,255            | 0/1:43,0,132              | 0/0:0,6,20              |
| B_06   | 92567840 | .  | C   | .   | 25.9985 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:12                    | 0/0:0                   |
| B_06   | 92567841 | .  | A   | G   | 6.10799 | .      |   | GT:PL  | 0/0:0,90,255              | 0/0:0,33,248            | 0/0:0,66,255            | 0/0:0,63,255            | 0/0:0,51,255            | 0/1:43,0,99               | 0/0:0,3,17              |
| B_06   | 92567846 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:193,0,255             | 1/1:222,27,0            | 1/1:255,54,0            | 1/1:255,57,0            | 1/1:255,45,0            | 0/0:0,30,130              | 0/1:0,3,17              |
| B_06   | 92567848 | .  | G   | C,T |     999 | .      |   | GT:PL  | 1/1:255,84,0,255,84,255   | 1/1:217,27,0,217,27,217 | 1/1:255,54,0,255,54,255 | 1/1:255,57,0,255,57,255 | 1/1:255,45,0,255,45,255 | 1/1:102,18,0,101,3,95     | 1/1:0,0,0,0,0,0         |
| B_06   | 92567852 | .  | C   | G   |     999 | .      |   | GT:PL  | 0/1:200,0,255             | 1/1:222,27,0            | 1/1:255,54,0            | 1/1:255,57,0            | 1/1:255,45,0            | 0/0:0,24,112              | 1/1:0,0,0               |
| B_06   | 92567859 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:177,0,255             | 1/1:220,27,0            | 1/1:255,60,0            | 1/1:255,54,0            | 1/1:255,45,0            | 1/1:81,12,0               | 1/1:0,0,0               |
| B_06   | 92567861 | .  | G   | C   | 24.5889 | .      |   | GT:PL  | 0/0:0,58,255              | 0/0:0,27,214            | 0/0:0,60,255            | 0/0:0,57,255            | 0/0:0,48,255            | 0/1:71,10,1               | 0/0:0,0,0               |
| B_06   | 92567866 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:163,0,255             | 1/1:217,27,0            | 1/1:255,60,0            | 1/1:255,57,0            | 1/1:255,45,0            | 0/0:0,12,81               | 1/1:0,0,0               |
| B_06   | 92567867 | .  | G   | C   | 24.5889 | .      |   | GT:PL  | 0/0:0,67,255              | 0/0:0,27,216            | 0/0:0,60,255            | 0/0:0,57,255            | 0/0:0,48,255            | 0/1:71,10,1               | 0/0:0,0,0               |
| B_06   | 92567868 | .  | C   | T   |     999 | .      |   | GT:PL  | 1/1:255,72,0              | 1/1:180,24,0            | 1/1:255,60,0            | 1/1:255,54,0            | 1/1:255,48,0            | 1/1:81,12,0               | 1/1:0,0,0               |
| B_06   | 92567869 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/0:0,78,255              | 1/1:179,11,0            | 1/1:255,57,0            | 1/1:255,57,0            | 1/1:255,48,0            | 0/0:0,12,81               | 1/1:0,0,0               |
| B_06   | 92567874 | .  | T   | C   |     999 | .      |   | GT:PL  | 0/1:180,0,255             | 0/1:195,0,10            | 1/1:255,60,0            | 1/1:255,57,0            | 1/1:255,48,0            | 0/0:59,68,107             | 0/1:0,3,37              |
| B_06   | 92567876 | .  | A   | T   | 45.7999 | .      |   | GT:PL  | 0/1:27,0,255              | 0/0:0,25,255            | 0/0:0,63,255            | 0/0:0,60,255            | 0/0:0,63,255            | 0/1:78,43,54              | 0/1:37,3,0              |
| B_06   | 92567893 | .  | T   | .   | 24.4461 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:14                    | 0/0:0                   |
| B_06   | 92567894 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:206,0,246             | 0/1:255,0,62            | 1/1:255,54,0            | 1/1:255,69,0            | 1/1:255,63,0            | 0/0:0,45,172              | 0/0:0,21,141            |
| B_06   | 92567895 | .  | C   | T   |  139.57 | .      |   | GT:PL  | 0/1:178,0,239             | 0/0:0,72,255            | 0/0:0,54,255            | 0/0:0,69,255            | 0/0:0,60,255            | 0/0:0,63,173              | 0/0:0,21,138            |
| B_06   | 92567903 | .  | G   | .   | 9.90516 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:29                    | 0/0:0                   |
| B_06   | 92567913 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:228,0,229             | 0/0:0,108,255           | 0/0:0,45,255            | 0/0:0,63,255            | 0/0:0,63,255            | 0/0:0,63,223              | 0/0:0,51,235            |
| B_06   | 92567920 | .  | A   | T   | 19.6112 | .      |   | GT:PL  | 0/0:0,87,255              | 0/0:0,58,255            | 0/0:0,51,255            | 0/0:0,63,255            | 0/0:0,54,255            | 0/1:58,0,122              | 0/0:0,54,249            |
| B_06   | 92567927 | .  | A   | G   | 29.5684 | .      |   | GT:PL  | 0/0:0,67,255              | 0/0:0,74,255            | 0/0:0,51,255            | 0/0:0,57,255            | 0/0:0,60,255            | 0/1:68,0,168              | 0/0:0,47,247            |
| B_06   | 92567930 | .  | A   | G   | 29.5684 | .      |   | GT:PL  | 0/0:0,93,255              | 0/0:0,74,255            | 0/0:0,48,255            | 0/0:0,66,255            | 0/0:0,54,255            | 0/1:68,0,168              | 0/0:0,47,242            |
| B_06   | 92567931 | .  | T   | .   | 34.4387 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:4                     | 0/0:0                   |
| B_06   | 92567951 | .  | T   | A   |     999 | .      |   | GT:PL  | 0/1:255,0,208             | 1/1:255,129,0           | 1/1:252,36,0            | 1/1:255,63,0            | 1/1:255,54,0            | 1/1:255,90,0              | 1/1:255,144,0           |
| B_06   | 92567956 | .  | C   | A   |     999 | .      |   | GT:PL  | 0/1:211,0,255             | 0/1:255,0,137           | 1/1:250,28,0            | 1/1:255,20,0            | 1/1:255,27,0            | 0/0:0,87,255              | 0/0:0,196,255           |
| B_06   | 92567958 | .  | T   | A   |     999 | .      |   | GT:PL  | 0/1:233,0,255             | 0/1:255,0,178           | 1/1:239,28,0            | 1/1:255,20,0            | 1/1:239,20,0            | 0/0:0,93,255              | 0/0:0,235,255           |
| B_06   | 92567966 | .  | G   | A   | 94.4352 | .      |   | GT:PL  | 0/0:0,144,255             | 0/0:0,157,255           | 0/0:0,51,255            | 0/0:0,93,255            | 0/0:0,66,255            | 0/1:65,0,182              | 0/1:76,0,163            |
| B_06   | 92567977 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:253,0,255             | 0/0:0,144,255           | 0/0:0,57,255            | 0/0:0,99,255            | 0/0:0,75,255            | 0/0:0,87,255              | 0/0:0,255,255           |
| B_06   | 92567978 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/1:255,0,165           | 1/1:255,27,0            | 1/1:255,38,0            | 1/1:255,44,0            | 0/0:0,99,255              | 0/0:0,255,255           |
| B_06   | 92567981 | .  | A   | C   | 27.5712 | .      |   | GT:PL  | 0/0:0,38,255              | 0/0:0,106,255           | 0/0:0,57,255            | 0/0:0,96,255            | 0/0:0,78,255            | 0/1:66,0,204              | 0/0:0,115,255           |
| B_06   | 92567993 | .  | A   | C   | 178.436 | .      |   | GT:PL  | 0/0:0,128,255             | 0/0:0,82,255            | 0/0:0,60,255            | 0/0:0,99,255            | 0/0:0,72,255            | 0/1:78,0,241              | 0/1:147,0,255           |
| B_06   | 92567994 | .  | C   | .   | 17.5147 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:21                    | 0/0:0                   |
| B_06   | 92567995 | .  | T   | .   | 13.6295 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:25                    | 0/0:0                   |
| B_06   | 92567999 | .  | C   | T   | 196.437 | .      |   | GT:PL  | 0/0:0,125,255             | 0/0:0,82,255            | 0/0:0,57,255            | 0/0:0,99,255            | 0/0:0,75,255            | 0/1:83,0,236              | 0/1:160,0,255           |
| B_06   | 92568001 | .  | G   | .   | 3.23425 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:0                   | 0/0:38                    | 0/0:0                   |
| B_06   | 92568003 | .  | A   | C   |     999 | .      |   | GT:PL  | 0/1:84,0,255              | 0/1:174,0,255           | 0/0:0,50,255            | 0/0:0,33,255            | 0/0:0,37,255            | 1/1:255,84,12             | 1/1:255,255,0           |
| B_06   | 92568004 | .  | T   | G   |     999 | .      |   | GT:PL  | 0/1:12,0,255              | 0/1:135,0,255           | 0/0:0,63,255            | 0/0:0,93,255            | 0/0:0,69,255            | 0/1:168,0,87              | 0/1:153,0,132           |
| B_06   | 92568006 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/1:255,0,179           | 1/1:255,61,0            | 1/1:255,33,0            | 1/1:255,34,0            | 0/0:0,73,255              | 0/0:0,255,255           |
| B_06   | 92568010 | .  | A   | G   | 187.436 | .      |   | GT:PL  | 0/0:0,125,255             | 0/0:0,53,255            | 0/0:0,66,255            | 0/0:0,75,255            | 0/0:0,60,255            | 0/1:83,0,174              | 0/1:151,0,205           |
| B_06   | 92568037 | .  | G   | T   | 125.092 | .      |   | GT:PL  | 0/0:0,160,255             | 0/0:0,86,255            | 0/0:0,75,255            | 0/0:0,105,255           | 0/0:0,66,255            | 0/1:16,0,201              | 0/1:155,0,255           |
| B_06   | 92568038 | .  | T   | C   |     999 | .      |   | GT:PL  | 0/0:0,110,255             | 0/1:115,0,255           | 0/0:0,72,255            | 0/0:0,105,255           | 0/0:0,66,255            | 0/1:201,0,29              | 0/1:255,0,255           |
| B_06   | 92568042 | .  | T   | G   | 199.474 | .      |   | GT:PL  | 0/0:0,70,255              | 0/0:0,16,255            | 0/0:0,61,255            | 0/0:0,38,255            | 0/0:0,32,255            | 0/1:102,0,175             | 0/1:144,0,255           |
| B_06   | 92568053 | .  | C   | A   | 154.911 | .      |   | GT:PL  | 0/0:0,116,255             | 0/0:0,15,255            | 0/0:0,75,255            | 0/0:0,108,255           | 0/0:0,69,255            | 0/0:0,3,200               | 0/1:193,0,255           |
| B_06   | 92568055 | .  | G   | A   | 101.436 | .      |   | GT:PL  | 0/0:0,172,255             | 0/0:0,99,255            | 0/0:0,72,255            | 0/0:0,102,255           | 0/0:0,69,255            | 0/1:101,0,91              | 0/1:47,0,255            |
| B_06   | 92568056 | .  | G   | A   | 252.652 | .      |   | GT:PL  | 0/0:0,118,255             | 0/1:123,0,255           | 0/0:0,72,255            | 0/0:0,102,255           | 0/0:0,72,255            | 0/0:3,0,155               | 0/1:174,0,255           |
| B_06   | 92568058 | .  | A   | G   | 275.668 | .      |   | GT:PL  | 0/0:0,123,255             | 0/1:40,0,255            | 0/0:0,56,255            | 0/0:0,43,255            | 0/0:0,56,255            | 0/1:115,0,83              | 0/1:172,0,255           |
| B_06   | 92568064 | .  | C   | T   | 280.667 | .      |   | GT:PL  | 0/0:0,120,255             | 0/1:44,0,255            | 0/0:0,47,255            | 0/0:0,43,255            | 0/0:0,63,255            | 0/1:106,0,72              | 0/1:182,0,255           |
| B_06   | 92568065 | .  | G   | A   | 16.6585 | .      |   | GT:PL  | 0/0:0,160,255             | 0/1:55,0,255            | 0/0:0,63,255            | 0/0:0,105,255           | 0/0:0,66,255            | 0/0:0,33,188              | 0/0:0,220,255           |
| B_06   | 92568078 | .  | A   | G   |     999 | .      |   | GT:PL  | 1/1:255,98,0              | 0/1:255,0,43            | 1/1:255,66,0            | 1/1:255,111,0           | 1/1:255,66,0            | 0/1:36,0,148              | 0/1:231,0,4             |
| B_06   | 92568106 | .  | A   | C,T |     999 | .      |   | GT:PL  | 0/1:255,0,255,255,255,255 | 1/1:255,45,0,255,45,255 | 1/1:255,72,0,255,72,255 | 1/1:255,93,0,255,93,255 | 1/1:255,75,0,255,75,255 | 1/1:125,30,0,125,30,125   | 1/1:255,121,7,210,0,192 |
| B_06   | 92568108 | .  | C   | G   | 57.4776 | .      |   | GT:PL  | 0/0:0,96,255              | 0/1:33,0,203            | 0/0:0,72,255            | 0/0:0,99,255            | 0/0:0,81,255            | 0/1:71,0,44               | 0/0:0,17,210            |
| B_06   | 92568109 | .  | G   | A   | 36.5753 | .      |   | GT:PL  | 0/0:0,163,255             | 0/0:0,36,255            | 0/0:0,69,255            | 0/0:0,99,255            | 0/0:0,78,255            | 0/0:0,18,136              | 0/1:75,0,179            |
| B_06   | 92568111 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/1:193,0,68            | 1/1:255,53,0            | 1/1:255,26,0            | 1/1:255,44,0            | 0/0:0,30,148              | 0/0:0,138,255           |
| B_06   | 92568125 | .  | G   | A   |     999 | .      |   | GT:PL  | 1/1:255,138,0             | 1/1:229,39,0            | 1/1:255,72,0            | 1/1:255,90,0            | 1/1:255,72,0            | 1/1:148,30,0              | 1/1:53,33,0             |
| B_06   | 92568134 | .  | G   | .   | 9.04328 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:0                   | 0/0:23                  | 0/0:7                   | 0/0:0                     | 0/0:0                   |
| B_06   | 92568141 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/0:0,151,255             | 0/1:205,0,70            | 1/1:255,75,0            | 1/1:255,84,0            | 1/1:255,45,0            | 0/0:0,30,142              | 0/0:0,27,60             |
| B_06   | 92568143 | .  | A   | C   | 181.534 | .      |   | GT:PL  | 0/0:0,111,255             | 0/1:59,0,222            | 0/0:0,73,255            | 0/0:0,33,255            | 0/0:0,37,255            | 1/1:131,18,0              | 1/1:56,9,0              |
| B_06   | 92568147 | .  | T   | G   | 184.533 | .      |   | GT:PL  | 0/0:0,111,255             | 0/1:54,0,237            | 0/0:0,70,255            | 0/0:0,36,255            | 0/0:0,42,255            | 1/1:135,18,0              | 1/1:60,9,0              |
| B_06   | 92568150 | .  | A   | G   | 171.518 | .      |   | GT:PL  | 0/0:0,108,255             | 0/1:49,0,244            | 0/0:0,81,255            | 0/0:0,73,255            | 0/0:0,53,255            | 1/1:126,12,0              | 1/1:60,9,0              |
| B_06   | 92568151 | .  | T   | A   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/1:234,0,78            | 1/1:255,81,0            | 1/1:255,76,0            | 1/1:255,50,0            | 0/0:0,33,145              | 0/0:0,36,69             |
| B_06   | 92568156 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/1:246,0,89            | 1/1:255,81,0            | 1/1:255,90,0            | 1/1:255,56,0            | 0/0:0,33,144              | 0/0:0,36,69             |
| B_06   | 92568162 | .  | A   | T   | 128.261 | .      |   | GT:PL  | 0/0:0,71,255              | 0/1:46,0,255            | 0/0:0,70,255            | 0/0:0,38,255            | 0/0:0,47,255            | 0/1:101,2,0               | 0/1:34,1,0              |
| B_06   | 92568164 | .  | A   | C   |     999 | .      |   | GT:PL  | 1/1:255,126,0             | 1/1:255,79,31           | 1/1:255,78,0            | 1/1:255,90,0            | 1/1:255,81,0            | 1/1:139,39,0              | 1/1:69,36,0             |
| B_06   | 92568188 | .  | T   | G   | 29.5685 | .      |   | GT:PL  | 0/0:0,129,255             | 0/0:0,51,255            | 0/0:0,84,255            | 0/0:0,99,255            | 0/0:0,99,255            | 0/1:68,0,99               | 0/0:0,36,69             |
| B_06   | 92568190 | .  | G   | T   |  3.0546 | .      |   | GT:PL  | 0/0:0,117,255             | 0/0:15,55,226           | 0/0:0,87,255            | 0/0:0,105,255           | 0/0:0,93,255            | 0/0:74,75,158             | 0/1:38,0,9              |
| B_06   | 92568191 | .  | G   | C   | 66.5297 | .      |   | GT:PL  | 0/0:0,62,255              | 0/1:38,0,227            | 0/0:0,87,255            | 0/0:0,105,255           | 0/0:0,93,255            | 0/1:75,0,94               | 0/0:0,12,64             |
| B_06   | 92568192 | .  | G   | T   | 42.1518 | .      |   | GT:PL  | 0/0:0,30,255              | 0/1:18,0,199            | 0/0:0,73,255            | 0/0:0,43,255            | 0/0:0,78,255            | 0/1:152,83,77             | 0/0:0,12,64             |
| B_06   | 92568193 | .  | T   | C   | 4.51773 | .      |   | GT:PL  | 0/0:0,132,255             | 0/0:0,45,255            | 0/0:0,84,255            | 0/0:0,102,255           | 0/0:0,90,255            | 0/0:0,19,189              | 0/1:41,0,5              |
| B_06   | 92568197 | .  | G   | T   | 45.7431 | .      |   | GT:PL  | 0/0:0,24,255              | 0/1:13,0,201            | 0/0:0,73,255            | 0/0:0,46,255            | 0/0:0,76,255            | 0/1:78,0,93               | 0/0:0,12,64             |
| B_06   | 92568199 | .  | C   | G   |  4.5232 | .      |   | GT:PL  | 0/0:0,135,255             | 0/0:0,48,255            | 0/0:0,81,255            | 0/0:0,90,255            | 0/0:0,90,255            | 0/0:0,16,179              | 0/1:41,0,5              |
| B_06   | 92568204 | .  | A   | G   | 8.07837 | .      |   | GT:PL  | 0/0:0,34,255              | 0/0:5,0,197             | 0/0:0,78,255            | 0/0:0,81,255            | 0/0:0,96,255            | 0/1:44,0,45               | 0/0:0,12,64             |
| B_06   | 92568206 | .  | A   | C   | 50.5659 | .      |   | GT:PL  | 0/0:0,22,255              | 0/1:18,0,189            | 0/0:0,78,255            | 0/0:0,87,255            | 0/0:0,93,255            | 0/1:80,2,0                | 0/0:0,9,53              |
| B_06   | 92568207 | .  | A   | T   | 142.162 | .      |   | GT:PL  | 0/0:0,51,255              | 0/1:30,0,165            | 0/0:0,78,255            | 0/0:0,73,255            | 0/0:0,69,255            | 1/1:124,33,0              | 1/1:53,9,0              |
| B_06   | 92568208 | .  | A   | C   | 139.185 | .      |   | GT:PL  | 0/0:0,24,255              | 0/1:27,0,159            | 0/0:0,75,255            | 0/0:0,70,255            | 0/0:0,69,255            | 1/1:124,33,0              | 1/1:53,9,0              |
| B_06   | 92568209 | .  | C   | G   | 142.215 | .      |   | GT:PL  | 0/0:0,19,255              | 0/1:30,0,163            | 0/0:0,75,255            | 0/0:0,70,255            | 0/0:0,72,255            | 1/1:124,33,0              | 1/1:53,9,0              |
| B_06   | 92568213 | .  | T   | A   | 56.2766 | .      |   | GT:PL  | 0/0:0,22,255              | 0/1:24,0,163            | 0/0:0,69,255            | 0/0:0,84,255            | 0/0:0,99,255            | 0/1:80,2,0                | 0/0:0,9,53              |
| B_06   | 92568226 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/0:0,51,255            | 0/0:0,81,255            | 0/0:0,84,255            | 0/0:0,114,255           | 0/0:0,54,238              | 0/0:0,9,53              |
| B_06   | 92568227 | .  | A   | T   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/1:160,0,82            | 1/1:255,61,0            | 1/1:255,16,0            | 1/1:255,67,0            | 0/0:0,54,238              | 0/0:0,9,53              |
| B_06   | 92568235 | .  | A   | T   |     999 | .      |   | GT:PL  | 0/1:74,0,255              | 0/1:47,0,169            | 0/0:0,58,255            | 0/0:0,27,255            | 0/0:0,87,255            | 1/1:220,42,0              | 1/1:25,6,0              |
| B_06   | 92568236 | .  | A   | C   | 53.1246 | .      |   | GT:PL  | 0/1:24,0,255              | 0/1:5,0,187             | 0/0:0,72,255            | 0/0:0,75,255            | 0/0:0,99,255            | 0/1:59,0,113              | 0/1:17,1,0              |
| B_06   | 92568237 | .  | A   | C   |     999 | .      |   | GT:PL  | 0/1:199,0,255             | 0/1:175,0,74            | 1/1:255,56,0            | 1/1:255,24,0            | 1/1:255,81,0            | 0/0:0,42,214              | 0/0:0,6,25              |
| B_06   | 92568241 | .  | T   | G   | 15.8562 | .      |   | GT:PL  | 0/0:0,82,255              | 0/0:0,17,231            | 0/0:0,72,255            | 0/0:0,63,255            | 0/0:0,99,255            | 0/1:54,0,128              | 0/0:0,6,31              |
| B_06   | 92568246 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 1/1:255,60,0            | 1/1:255,72,0            | 1/1:255,63,0            | 1/1:255,90,0            | 1/1:222,39,0              | 1/1:60,12,0             |
| B_06   | 92568257 | .  | G   | T   |     999 | .      |   | GT:PL  | 0/1:184,0,255             | 0/1:255,0,43            | 0/1:255,0,63            | 0/1:255,0,92            | 1/1:255,44,0            | 0/0:0,45,243              | 0/0:0,18,108            |
| B_06   | 92568266 | .  | C   | T   | 82.5069 | .      |   | GT:PL  | 0/0:0,148,255             | 0/0:0,75,255            | 0/0:0,96,255            | 0/0:0,87,255            | 0/0:0,120,255           | 0/1:51,0,167              | 0/1:78,0,7              |
| B_06   | 92568271 | .  | A   | T   | 101.438 | .      |   | GT:PL  | 0/0:0,53,255              | 0/0:0,59,255            | 0/1:63,0,255            | 0/1:85,0,255            | 0/0:0,80,255            | 0/0:0,45,237              | 0/0:0,27,157            |
| B_06   | 92568272 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:255,0,255             | 0/0:0,62,255            | 0/1:63,0,255            | 0/1:85,0,255            | 0/0:0,72,255            | 0/0:0,48,241              | 0/0:0,30,167            |
| B_06   | 92568275 | .  | G   | A   | 97.4357 | .      |   | GT:PL  | 0/0:0,47,255              | 0/0:0,65,255            | 0/1:61,0,255            | 0/1:83,0,255            | 0/0:0,72,255            | 0/0:0,45,221              | 0/0:0,36,167            |
| B_06   | 92568278 | .  | G   | A   | 99.4357 | .      |   | GT:PL  | 0/0:0,154,255             | 0/0:0,65,255            | 0/1:61,0,255            | 0/1:85,0,255            | 0/0:0,83,255            | 0/0:0,45,231              | 0/0:0,36,170            |
| B_06   | 92568290 | .  | G   | T   | 94.4357 | .      |   | GT:PL  | 0/0:0,50,255              | 0/0:0,65,255            | 0/1:61,0,255            | 0/1:80,0,255            | 0/0:0,77,255            | 0/0:0,42,218              | 0/0:0,36,175            |
| B_06   | 92568291 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:199,0,255             | 0/1:255,0,108           | 0/1:255,0,63            | 0/1:255,0,83            | 1/1:255,69,0            | 0/0:0,45,229              | 0/0:0,36,177            |
| B_06   | 92568296 | .  | A   | T   | 92.4358 | .      |   | GT:PL  | 0/0:0,138,255             | 0/0:0,62,255            | 0/1:63,0,255            | 0/1:76,0,255            | 0/0:0,77,255            | 0/0:0,39,215              | 0/0:0,36,181            |
| B_06   | 92568297 | .  | C   | G   |     999 | .      |   | GT:PL  | 0/1:10,0,255              | 0/1:69,0,255            | 0/0:72,139,255          | 0/0:76,151,255          | 0/0:0,108,255           | 1/1:213,19,0              | 1/1:185,36,0            |
| B_06   | 92568303 | .  | C   | .   | 7.74103 | .      |   | GT:PL  | 0/0:0                     | 0/0:0                   | 0/0:18                  | 0/0:21                  | 0/0:0                   | 0/0:0                     | 0/0:0                   |
| B_06   | 92568317 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/1:7,0,255               | 0/1:101,0,255           | 0/1:160,0,255           | 0/1:168,0,255           | 0/1:108,0,255           | 1/1:170,24,0              | 1/1:184,33,0            |
| B_06   | 92568320 | .  | C   | T   |     999 | .      |   | GT:PL  | 0/1:235,0,255             | 1/1:255,65,0            | 0/1:249,0,175           | 0/1:255,0,168           | 0/1:255,0,129           | 1/1:170,24,0              | 1/1:179,30,0            |
| B_06   | 92568321 | .  | A   | C   |     999 | .      |   | GT:PL  | 0/1:218,0,255             | 1/1:255,65,0            | 0/1:247,0,173           | 0/1:255,0,168           | 0/1:255,0,125           | 1/1:160,13,0              | 1/1:188,33,0            |
| B_06   | 92568326 | .  | A   | G   |     999 | .      |   | GT:PL  | 0/1:199,0,255             | 0/1:255,0,150           | 0/1:250,0,174           | 0/1:255,0,168           | 0/1:255,0,132           | 0/0:0,27,168              | 0/0:0,36,183            |
| B_06   | 92568330 | .  | T   | A   |     999 | .      |   | GT:PL  | 0/0:0,79,255              | 0/1:135,0,255           | 0/0:0,102,255           | 0/0:0,132,255           | 0/0:0,148,255           | 1/1:160,16,0              | 1/1:181,33,0            |
| B_06   | 92568331 | .  | G   | A   |     999 | .      |   | GT:PL  | 0/0:0,53,255              | 0/0:0,76,255            | 0/1:164,0,255           | 0/1:173,0,255           | 0/1:133,0,255           | 0/0:0,21,129              | 0/0:0,33,184            |
| B_06   | 92568333 | .  | T   | C   |     999 | .      |   | GT:PL  | 0/1:204,0,255             | 0/0:0,77,255            | 0/0:0,111,255           | 0/0:0,129,255           | 0/0:0,144,255           | 0/0:0,27,165              | 0/0:0,33,184            |



















|        | D3d                                          | D3d               | A2D1                 | A2D2                                          | A2                            |
|--------+----------------------------------------------+-------------------+----------------------+-----------------------------------------------+-------------------------------|
| no ref | GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG | GGGG$GGGGGGG$GGGG | TTTTttttGtGGttGTGtTG | g$GGGtttgGGGTttGGGGGGGGGGGGGGGGgttGGGtGGGGGgg | TTTTTTttttttTttTTtTTTTTTttttT |
| ref    | ............................................ | ....$.......$.... | TTTTtttt.t..tt.T.tT. | ,$...ttt,...Ttt................,tt...t.....,, | TTTTTTttttttTttTTtTTTTTTttttT |

|      |          |   |   |   |     |   |       | D3d           | D3d         | A2D1         | A2D2          | A2           |
| B_06 | 92567396 | . | G | T | 999 | . | GT:PL | 0/0:0,132,207 | 0/0:0,45,88 | 0/1:255,0,76 | 0/1:127,0,255 | 1/1:255,87,0 |


#+begin_example
b_06    92567426        43      aaaaaaa.-15tgggcgagagccagaa..-15tgggcgagagccaga.aaaa.-15tgggcgagagccagaaaa.-15tgggcgagagccagaaaaaaaaaa.-15tgggcgagagccagaaa.-15tgggcgagagccaga.-15tgggcgagagccaga.-15tgggcgagagccag
b_06    92567427        30      ..............................  2       ..      8       .,,,....        26      ,,...,..........,.....,,,,
b_06    92567428        26      ..........................      2       ..      5       .....   25      ,,.............,.....,,,,
b_06    92567429        27      ...........................     2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567430        27      ...........................     2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567431        26      ..........................      2       ..      3       ...     21      ,...t..tt.t,.tttt,,,,
b_06    92567432        28      ............................    2       ..      3       ...     22      aaaaa.......a.....aaaa
b_06    92567433        25      .........................       2       ..      3       ...     21      ,,..........,.....,,,
b_06    92567434        25      .........................       2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567435        23      ....................... 2       ..      3       ...     21      ,,..........,....,,,,
b_06    92567436        26      ..........................      2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567437        26      ..........................      2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567438        26      ..........................      2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567439        24      ........................        2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567440        24      ........................        2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567441        24      ........................        2       ..      3       ...     22      ,,..........,.....,,,,
b_06    92567442        23      ....................... 2       ..      8       .....,,,        24      ,,..........,.....,,,,.,
b_06    92567443        24      ........................        2       ..      8       .....,,,        25      ,,..........,.....,,,,..,
b_06    92567444        41      .........................................       2       ..      17      ,.,.,,...,....,,,       36      ,....,..........,,,...,......,,,,..,

#+end_example

** B06G223900

D3 doesn't align.
A2D3 show


#+begin_src sh

awk '!/^##/ {$8 = ""; c=0; for(i = 10; i < NF; i++) c += ($i !="0/0:0"); if(c) print}' OFS="\t" b-only/B06G223900.vcf
#+end_src

#+RESULTS:
| #CHROM |      POS | ID | REF | ALT | QUAL | FILTER |   | FORMAT | b-only/SRR6334601.bam | b-only/SRR6334602.bam | b-only/SRR8979965.bam | b-only/SRR8979944.bam | b-only/SRR8979925.bam | b-only/SRR6334584.bam | b-only/SRR8136261.bam |
| B_06   | 92741078 | .  | T   | G   |  999 | .      |   | GT:PL  | 1/1:255,60,0          | 1/1:22,3,0            | 1/1:75,6,0            | 1/1:150,12,0          | 1/1:82,6,0            | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741084 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/0:0,63,255          | 1/1:22,3,0            | 1/1:75,6,0            | 1/1:146,12,0          | 1/1:82,6,0            | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741085 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/0:0,63,255          | 1/1:37,3,0            | 1/1:75,6,0            | 1/1:150,12,0          | 1/1:82,6,0            | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741087 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/0:0,60,255          | 1/1:41,3,0            | 1/1:75,6,0            | 1/1:150,12,0          | 1/1:82,6,0            | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741090 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/0:0,57,255          | 1/1:37,3,0            | 1/1:88,9,0            | 1/1:150,12,0          | 1/1:82,6,0            | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741097 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/0:0,57,255          | 1/1:89,9,0            | 1/1:145,12,0          | 1/1:150,12,0          | 1/1:116,9,0           | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741102 | .  | C   | A   |  999 | .      |   | GT:PL  | 0/0:0,60,255          | 1/1:102,9,0           | 1/1:160,15,0          | 1/1:143,12,0          | 1/1:105,9,0           | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741106 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/0:0,60,255          | 1/1:96,9,0            | 1/1:164,15,0          | 1/1:150,12,0          | 1/1:112,9,0           | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741108 | .  | A   | G   |  999 | .      |   | GT:PL  | 0/0:0,57,255          | 1/1:93,9,0            | 1/1:164,15,0          | 1/1:150,12,0          | 1/1:116,9,0           | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741125 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:111,0,255         | 1/1:255,36,0          | 1/1:255,84,0          | 1/1:255,105,0         | 1/1:255,57,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741127 | .  | A   | G   |  999 | .      |   | GT:PL  | 0/1:174,0,255         | 1/1:255,45,0          | 1/1:255,93,0          | 1/1:255,108,0         | 1/1:255,57,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741142 | .  | T   | C   |  999 | .      |   | GT:PL  | 0/1:255,0,237         | 0/0:0,45,255          | 0/0:0,105,255         | 0/0:0,105,255         | 0/0:0,66,255          | 0/0:0,0,0             | 0/0:0,0,0             |
| B_06   | 92741168 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 0/0:0,39,255          | 0/0:0,111,255         | 0/0:0,132,255         | 0/0:0,63,255          | 0/0:0,0,0             | 0/0:0,0,0             |
| B_06   | 92741175 | .  | T   | C   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,33,0          | 1/1:255,75,0          | 1/1:255,117,0         | 1/1:255,45,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741178 | .  | T   | C   |  999 | .      |   | GT:PL  | 0/1:249,0,255         | 1/1:255,33,0          | 1/1:255,75,0          | 1/1:255,114,0         | 1/1:255,45,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741180 | .  | G   | T   |  999 | .      |   | GT:PL  | 0/1:253,0,255         | 1/1:255,36,0          | 1/1:255,75,0          | 1/1:255,120,0         | 1/1:255,45,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741186 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,39,0          | 1/1:255,75,0          | 1/1:255,123,0         | 1/1:255,45,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741194 | .  | T   | C   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,36,0          | 1/1:255,81,0          | 1/1:255,129,0         | 1/1:255,42,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741206 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,33,0          | 1/1:255,81,0          | 1/1:255,144,0         | 1/1:255,57,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741217 | .  | A   | T   |  999 | .      |   | GT:PL  | 0/1:218,0,255         | 1/1:255,30,0          | 1/1:255,78,0          | 1/1:255,135,0         | 1/1:255,57,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741218 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:223,0,255         | 1/1:255,30,0          | 1/1:255,75,0          | 1/1:255,132,0         | 1/1:255,60,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741226 | .  | A   | G   |  999 | .      |   | GT:PL  | 1/1:255,78,0          | 1/1:255,33,0          | 1/1:255,78,0          | 1/1:255,135,0         | 1/1:255,60,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741228 | .  | C   | G   |  999 | .      |   | GT:PL  | 0/1:225,0,255         | 0/0:0,33,255          | 0/0:0,72,255          | 0/0:0,135,255         | 0/0:0,60,255          | 0/0:0,3,41            | 0/0:0,0,0             |
| B_06   | 92741229 | .  | G   | T   |  999 | .      |   | GT:PL  | 0/1:225,0,255         | 1/1:255,33,0          | 1/1:255,78,0          | 1/1:255,135,0         | 1/1:255,57,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741239 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,33,0          | 1/1:255,72,0          | 1/1:255,126,0         | 1/1:255,66,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741251 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,45,0          | 1/1:255,78,0          | 1/1:255,126,0         | 1/1:255,63,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741266 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,48,0          | 1/1:255,75,0          | 1/1:255,129,0         | 1/1:255,72,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741271 | .  | A   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,42,0          | 1/1:255,78,0          | 1/1:255,129,0         | 1/1:255,75,0          | 1/1:37,3,0            | 1/1:0,0,0             |
| B_06   | 92741272 | .  | A   | G   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 0/0:0,39,255          | 0/0:0,78,255          | 0/0:0,129,255         | 0/0:0,75,255          | 0/0:0,3,41            | 0/0:0,0,0             |
| B_06   | 92741274 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,42,0          | 1/1:255,75,0          | 1/1:255,123,0         | 1/1:255,78,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741295 | .  | T   | C   |  999 | .      |   | GT:PL  | 0/1:255,0,169         | 1/1:255,39,0          | 1/1:255,72,0          | 1/1:255,102,0         | 1/1:255,60,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741299 | .  | A   | G   |  999 | .      |   | GT:PL  | 0/0:0,75,255          | 1/1:255,36,0          | 1/1:255,69,0          | 1/1:255,102,0         | 1/1:255,63,0          | 0/1:0,3,41            | 1/1:0,0,0             |
| B_06   | 92741301 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/1:255,0,219         | 1/1:255,33,0          | 1/1:255,69,0          | 1/1:255,102,0         | 1/1:255,60,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741307 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/1:255,0,215         | 1/1:255,33,0          | 1/1:255,66,0          | 1/1:255,102,0         | 1/1:255,63,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741313 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,192         | 1/1:255,33,0          | 1/1:255,63,0          | 1/1:255,93,0          | 1/1:255,60,0          | 1/1:41,3,0            | 1/1:0,0,0             |
| B_06   | 92741319 | .  | A   | G   |  999 | .      |   | GT:PL  | 0/1:251,0,227         | 1/1:255,36,0          | 1/1:255,51,0          | 1/1:255,90,0          | 1/1:255,63,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741322 | .  | G   | T   |  999 | .      |   | GT:PL  | 0/1:246,0,202         | 1/1:255,36,0          | 1/1:255,48,0          | 1/1:255,93,0          | 1/1:255,63,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741328 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/1:247,0,194         | 1/1:255,36,0          | 1/1:255,51,0          | 1/1:255,93,0          | 1/1:255,63,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741330 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,179         | 1/1:255,36,0          | 1/1:255,48,0          | 1/1:255,93,0          | 1/1:255,63,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741331 | .  | T   | A   |  999 | .      |   | GT:PL  | 0/1:255,0,193         | 1/1:255,36,0          | 1/1:255,48,0          | 1/1:255,90,0          | 1/1:255,63,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741341 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,229         | 1/1:255,48,0          | 1/1:255,93,0          | 1/1:255,123,0         | 1/1:255,78,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741365 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/0:0,108,255         | 1/1:255,14,0          | 1/1:255,93,0          | 1/1:255,111,0         | 1/1:255,60,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741370 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,51,0          | 1/1:255,90,0          | 1/1:255,111,0         | 1/1:255,60,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741371 | .  | A   | G   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,51,0          | 1/1:255,96,0          | 1/1:255,114,0         | 1/1:255,60,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741373 | .  | C   | T   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,45,0          | 1/1:255,87,0          | 1/1:255,111,0         | 1/1:255,57,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741388 | .  | G   | A   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,54,0          | 1/1:255,96,0          | 1/1:255,114,0         | 1/1:255,63,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741399 | .  | A   | C   |  999 | .      |   | GT:PL  | 0/1:255,0,255         | 1/1:255,42,0          | 1/1:255,93,0          | 1/1:255,117,0         | 1/1:255,54,0          | 1/1:0,0,0             | 1/1:0,0,0             |
| B_06   | 92741401 | .  | T   | A   |  999 | .      |   | GT:PL  | 1/1:255,120,0         | 1/1:255,42,0          | 1/1:255,90,0          | 1/1:255,117,0         | 1/1:255,54,0          | 1/1:0,0,0             | 1/1:0,0,0             |


4 SNPs suggest autapomorphy in 2A2D3 because everything 0/0 except a2d3 is 0/1
B_06 	92741142	 . 	 T 	 C 	999	 . 	 GT:PL 	 0/0:0,0,0  	 0/0:0,0,0 	 0/0:0,45,255 	 0/1:255,0,237 	 0/0:0,66,255
B_06 	92741168	 . 	 G 	 A 	999	 . 	 GT:PL 	 0/0:0,0,0  	 0/0:0,0,0 	 0/0:0,39,255 	 0/1:255,0,255 	 0/0:0,63,255
B_06 	92741228	 . 	 C 	 G 	999	 . 	 GT:PL 	 0/0:0,3,41 	 0/0:0,0,0 	 0/0:0,33,255 	 0/1:225,0,255 	 0/0:0,60,255
B_06 	92741272	 . 	 A 	 G 	999	 . 	 GT:PL 	 0/0:0,3,41 	 0/0:0,0,0 	 0/0:0,39,255 	 0/1:255,0,255 	 0/0:0,75,255


* plot gene coverage

#+NAME: plotdata
| SRA         | Species              | PI       |
|-------------+----------------------+----------|
| SRR6334584  | Gossypium davidsonii | Udall    |
| SRR8136261  | Gossypium davidsonii | Peterson |
| SRR6334602  | 2(A2D1)              | Udall    |
| SRR6334601  | 2(A2D3)              | Udall    |
| SRR3560153  | Gossypium anomalum   | Peterson |
| SRR3560155  | Gossypium anomalum   | Peterson |
| SRR12745560 | Gossypium anomalum   | CAAS     |
| SRR8979965  | Gossypium arboreum   | Udall    |
| SRR8979944  | Gossypium arboreum   | Udall    |
| SRR8979925  | Gossypium arboreum   | Udall    |
| SRR3560162  | Gossypium somalense  | E2       |
| SRR617704   | Gossypium longicalyx | F1       |

- Removed because lower coverage than the other three B1s
| SRR3560156  | Gossypium anomalum   | Peterson |



*** samplot
#+begin_src sh
wget -O samplot-1.2.0.sif  https://depot.galaxyproject.org/singularity/samplot:1.2.0--pyh5e36f6f_0
#+end_src

#+begin_src sh :tangle samplot.sh :var sra=plotdata[,0] :var titles=plotdata[,1]

ml singularity/3.5.2 htslib bedtools

# awk '$2 == "maker"' B1.gff3 |
#     sed 's/;$//' |
#     bedtools sort -i - |
#     bgzip -c > B1.gff3.gz
# tabix B1.gff3.gz

# cut -f 1,4,5 ../annotations/1-repeats/B1.rm.gff3 |
#     bedtools sort -i - |
#     bgzip -c > B1.repmask.bed.gz
# tabix B1.repmask.bed.gz

singularity exec samplot-1.2.0.sif samplot plot \
    -n  "${titles[@]}" \
    -b  $(printf "b-only/%s.bam " ${sra[@]}) \
    -o  b-only/B06G223600.png \
    -c B_06 -s 92567289 -e 92568440 \
    -T B1.gff3.gz --transcript_filename Gene

singularity exec samplot-1.2.0.sif samplot plot \
    -n  "${titles[@]}" \
    -b  $(printf "b-only/%s.bam " ${sra[@]}) \
    -o  b-only/B06G223900.png \
    -c B_06 -s 92740977 -e 92741503 \
    -T B1.gff3.gz --transcript_filename Gene

singularity exec samplot-1.2.0.sif samplot plot \
    -n  "${titles[@]}" \
    -b  $(printf "b-only/%s.bam " ${sra[@]}) \
    -o  b-only/introgressed-region.png \
    -c B_06 -s 92554341 -e 92841871 \
    -T B1.gff3.gz --transcript_filename Gene \
    -A B1.repmask.bed.gz --annotation_filenames RepeatMasker




#+end_src

#+RESULTS:

*** gnuplot

#+begin_src sh :tangle samplot.sh :var titles=plotdata[,0:1] :var sra=plotdata[,0]

ml singularity/3.5.2 samtools

for acc in "${sra[@]}"; do
    title="${titles[$acc]}"
   samtools depth -r B_06:92567100-92568600 b-only/$acc.bam |
       gnuplot -e "sra='$title'; outfile='$acc.png'" cov.plot

done

convert $(printf "%s.png " "${sra[@]}") -append -resize 50%  B06G223600.png

#+end_src

*** G3V

#+begin_src sh

ml singularity/3.5.2 bedtools samtools
PATH=`pwd`/G3V/bin/:$PATH


declare -A genes
genes["B06G223600"]="B_06:92567189-92568540"
genes["B06G223900"]="B_06:92740877-92741603"
genes["introgression"]="B_06:92554341-92841871"

for gene in "${!genes[@]}"; do
    region=${genes[$gene]};

for sra in  SRR6334584 SRR8136261 SRR6334602 SRR6334601 SRR3560153 SRR3560155 SRR12745560 SRR8979965 SRR8979944 SRR8979925 SRR3560162 SRR617704; do
    samtools view -bu b-only/$sra.bam $region |
        bedtools genomecov -bg -max 50 -ibam - > b-only/$sra.$region.bed
done



g3v init -i $region -t $gene |
    g3v anno -t Annotations -i B1.gff3 |
    g3v group -g "B1" -t "Gossypium anomalum" |
    g3v ruler |
    g3v hist -t "SRR3560153"  -i b-only/SRR3560153.$region.bed |
    g3v hist -t "SRR3560155"  -i b-only/SRR3560155.$region.bed |
    g3v hist -t "SRR12745560" -i b-only/SRR12745560.$region.bed |
    g3v group -g "group2" -t  "2(A2D1)" |
    g3v ruler |
    g3v hist -t "SRR6334602" -i b-only/SRR6334602.$region.bed |
    g3v group -g "group3" -t  "2(A2D3)" |
    g3v ruler |
    g3v hist -t "SRR6334601" -i b-only/SRR6334601.$region.bed |
    g3v group -g "D3" -t "Gossypium davidsonii" |
    g3v ruler |
    g3v hist -t "SRR6334584" -i b-only/SRR6334584.$region.bed |
    g3v hist -t "SRR8136261" -i b-only/SRR8136261.$region.bed |
    g3v group -g "A2" -t "Gossypium arboreum" |
    g3v ruler |
    g3v hist -t "SRR8979965" -i b-only/SRR8979965.$region.bed |
    g3v hist -t "SRR8979944" -i b-only/SRR8979944.$region.bed |
    g3v hist -t "SRR8979925" -i b-only/SRR8979925.$region.bed |
    g3v group -g "E2" -t "Gossypium somalense" |
    g3v ruler |
    g3v hist -t "SRR3560162" -i b-only/SRR3560162.$region.bed |
    g3v group -g "F1" -t "Gossypium longicalyx" |
    g3v ruler |
    g3v hist -t "SRR617704"  -i b-only/SRR617704.$region.bed |
    convert svg:- $gene.g3v.png

done
#+end_src


* Assemble high conf 2(A2D3) anomalum regions

#+begin_src sh

ml singularity/3.5.2 abyss

mkdir assemble

samtools fastq A2D3.B1.specific.bam > assemble/in.fastq

abyss-pe np=48 -C assemble k=121 in=in.fastq name=A2D3.B1.specific
#+end_src

#+RESULTS:
| name      |     n | n:500 |  L50 | min | N75 | N50 |  N25 | E-size |   max |     sum |
|-----------+-------+-------+------+-----+-----+-----+------+--------+-------+---------|
| unitigs   | 43065 |  5259 | 1829 | 500 | 627 | 807 | 1120 |    990 |  6889 | 4280548 |
| contigs   | 42619 |  5048 | 1609 | 500 | 632 | 832 | 1358 |   1192 | 13202 | 4342321 |
| scaffolds | 42612 |  5042 | 1603 | 500 | 632 | 832 | 1364 |   1203 | 13202 | 4342321 |
